
<!DOCTYPE HTML>
<html>
    <head>
        <meta charset="utf-8">
        <title>Nature, Keyword: diversity</title>
    </head>
    <body>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-79046-y" target="_blank" style="color:black; text-decoration:none">Reconstructing hotspots of genetic diversity from glacial refugia and subsequent dispersal in Italian common toads ( Bufo bufo )</a></h1>
            <p>Andrea Chiocchio, Jan. W. Arntzen, Iñigo Martínez-Solano, Wouter de Vries, Roberta Bisconti, Alice Pezzarossa, Luigi Maiorano, Daniele Canestrelli</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Genetic diversity feeds the evolutionary process and allows populations to adapt to environmental changes. However, we still lack a thorough understanding of why hotspots of genetic diversity are so 'hot'. Here, we analysed the relative contribution of bioclimatic stability and genetic admixture between divergent lineages in shaping spatial patterns of genetic diversity in the common toad Bufo bufo along the Italian peninsula. We combined population genetic, phylogeographic and species distribution modelling (SDM) approaches to map ancestral areas, glacial refugia, and secondary contact zones. We consistently identified three phylogeographic lineages, distributed in northern, central and southern Italy. These lineages expanded from their ancestral areas and established secondary contact zones, before the last interglacial. SDM identified widespread glacial refugia in peninsular Italy, sometimes located under the present-day sea-level. Generalized linear models indicated genetic admixture as the only significant predictor of the levels of population genetic diversity. Our results show that glacial refugia contributed to preserving both levels and patterns of genetic diversity across glacial-interglacial cycles, but not to their formation, and highlight a general principle emerging in Mediterranean species: higher levels of genetic diversity mark populations with substantial contributions from multiple genetic lineages, irrespective of the location of glacial refugia.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00862-1" target="_blank" style="color:black; text-decoration:none">Fungal sporocarps house diverse and host-specific communities of fungicolous fungi</a></h1>
            <p>Sundy Maurice, Gontran Arnault, Jenni Nordén, Synnøve Smebye Botnen, Otto Miettinen, Håvard Kauserud</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">Sporocarps (fruit bodies) are the sexual reproductive stage in the life cycle of many fungi. They are highly nutritious and consequently vulnerable to grazing by birds and small mammals, and invertebrates, and can be infected by microbial and fungal parasites and pathogens. The complexity of communities thriving inside sporocarps is largely unknown. In this study, we revealed the diversity, taxonomic composition and host preference of fungicolous fungi (i.e., fungi that feed on other fungi) in sporocarps. We carried out DNA metabarcoding of the ITS2 region from 176 sporocarps of 11 wood-decay fungal host species, all collected within a forest in northeast Finland. We assessed the influence of sporocarp traits, such as lifespan, morphology and size, on the fungicolous fungal community. The level of colonisation by fungicolous fungi, measured as the proportion of non-host ITS2 reads, varied between 2.8–39.8% across the 11 host species and was largely dominated by Ascomycota. Host species was the major determinant of the community composition and diversity of fungicolous fungi, suggesting that host adaptation is important for many fungicolous fungi. Furthermore, the alpha diversity was consistently higher in short-lived and resupinate sporocarps compared to long-lived and pileate ones, perhaps due to a more hostile environment for fungal growth in the latter too. The fungicolous fungi represented numerous lineages in the fungal tree of life, among which a significant portion was poorly represented with reference sequences in databases.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00868-9" target="_blank" style="color:black; text-decoration:none">Cervicovaginal bacterial communities in reproductive-aged Tanzanian women with Schistosoma mansoni , Schistosoma haematobium , or without schistosome infection</a></h1>
            <p>Brooke W. Bullington, Myung Hee Lee, Jane Mlingi, Ndalloh Paul, Christine Aristide, Emily Fontana, Eric R. Littmann, Crispin Mukerebe, Peter Shigella, Philibert Kashangaki, Samuel E. Kalluvya, Claudia J. de Dood, Govert J. van Dam, Paul L.A.M. Corstjens, Daniel W. Fitzgerald, Eric G. Pamer, Jennifer A. Downs</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2021-01-06</pre>
            <h2>Abstract</h2>
            <font size="4">Schistosome infection is recognized as a potentially modifiable risk factor for HIV in women by the World Health Organization. Alterations in cervicovaginal bacteria have been associated with HIV acquisition and have not been studied in schistosome infection. We collected cervical swabs from Tanzanian women with and without S. mansoni and S. haematobium to determine effects on cervicovaginal microbiota. Infected women were treated, and follow-up swabs were collected after 3 months. 16S rRNA sequencing was performed on DNA extracted from swabs. We compared 39 women with S. mansoni with 52 uninfected controls, and 16 with S. haematobium with 27 controls. S. mansoni-infected women had increased abundance of Peptostreptococcus (p = 0.026) and presence of Prevotella timonesis (p = 0.048) compared to controls. High-intensity S. haematobium infection was associated with more diverse cervicovaginal bacterial communities than uninfected controls (p = 0.0159). High-intensity S. mansoni infection showed a similar trend (p = 0.154). At follow-up, we observed increased alpha diversity in S. mansoni (2.53 vs. 1.72, p = 0.022) and S. haematobium (2.05 vs. 1.12, p = 0.066) infection groups compared to controls. Modifications in cervicovaginal microbiota, particularly increased diversity and abundance of taxa associated with bacterial vaginosis and HIV (Peptostreptococcus, Prevotella), were associated with schistosome infection.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-20240-x" target="_blank" style="color:black; text-decoration:none">Longitudinal dynamics of gut bacteriome, mycobiome and virome after fecal microbiota transplantation in graft-versus-host disease</a></h1>
            <p>Fen Zhang, Tao Zuo, Yun Kit Yeoh, Frankie W. T. Cheng, Qin Liu, Whitney Tang, Kitty C. Y. Cheung, Keli Yang, Chun Pan Cheung, Chow Chung Mo, Mamie Hui, Francis K. L. Chan, Chi-Kong Li, Paul K. S. Chan, Siew C. Ng</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Fecal microbiota transplant (FMT) has emerged as a potential treatment for severe colitis associated with graft-versus-host disease (GvHD) following hematopoietic stem cell transplant. Bacterial engraftment from FMT donor to recipient has been reported, however the fate of fungi and viruses after FMT remains unclear. Here we report longitudinal dynamics of the gut bacteriome, mycobiome and virome in a teenager with GvHD after receiving four doses of FMT at weekly interval. After serial FMTs, the gut bacteriome, mycobiome and virome of the patient differ from compositions before FMT with variable temporal dynamics. Diversity of the gut bacterial community increases after each FMT. Gut fungal community initially shows expansion of several species followed by a decrease in diversity after multiple FMTs. In contrast, gut virome community varies substantially over time with a stable rise in diversity. The bacterium, Corynebacterium jeikeium, and Torque teno viruses, decrease after FMTs in parallel with an increase in the relative abundance of Caudovirales bacteriophages. Collectively, FMT may simultaneously impact on the various components of the gut microbiome with distinct effects. Fecal microbiota transplant (FMT) is emerging as a potential treatment for graft-versus-host disease (GvHD). Here, the authors examine temporal dynamics of the bacteriome, mycobiome, and virome of a patient with GvHD who received multiple FMTs.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80077-8" target="_blank" style="color:black; text-decoration:none">Repeated surveying over 6 years reveals that fine-scale habitat variables are key to tropical mountain ant assemblage composition and functional diversity</a></h1>
            <p>Mulalo M. Muluvhahothe, Grant S. Joseph, Colleen L. Seymour, Thinandavha C. Munyai, Stefan H. Foord</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">High-altitude-adapted ectotherms can escape competition from dominant species by tolerating low temperatures at cooler elevations, but climate change is eroding such advantages. Studies evaluating broad-scale impacts of global change for high-altitude organisms often overlook the mitigating role of biotic factors. Yet, at fine spatial-scales, vegetation-associated microclimates provide refuges from climatic extremes. Using one of the largest standardised data sets collected to date, we tested how ant species composition and functional diversity (i.e., the range and value of species traits found within assemblages) respond to large-scale abiotic factors (altitude, aspect), and fine-scale factors (vegetation, soil structure) along an elevational gradient in tropical Africa. Altitude emerged as the principal factor explaining species composition. Analysis of nestedness and turnover components of beta diversity indicated that ant assemblages are specific to each elevation, so species are not filtered out but replaced with new species as elevation increases. Similarity of assemblages over time (assessed using beta decay) did not change significantly at low and mid elevations but declined at the highest elevations. Assemblages also differed between northern and southern mountain aspects, although at highest elevations, composition was restricted to a set of species found on both aspects. Functional diversity was not explained by large scale variables like elevation, but by factors associated with elevation that operate at fine scales (i.e., temperature and habitat structure). Our findings highlight the significance of fine-scale variables in predicting organisms’ responses to changing temperature, offering management possibilities that might dilute climate change impacts, and caution when predicting assemblage responses using climate models, alone.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78910-1" target="_blank" style="color:black; text-decoration:none">Gene diversity explains variation in biological features of insect killing fungus, Beauveria bassiana</a></h1>
            <p>Laila Gasmi, Sehyeon Baek, Jong Cheol Kim, Sihyeon Kim, Mi Rong Lee, So Eun Park, Tae Young Shin, Se Jin Lee, Bruce L. Parker, Jae Su Kim</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Beauveria bassiana is a species complex whose isolates show considerable natural genetic variability. However, little is known about how this genetic diversity affects the fungus performance. Herein, we characterized the diversity of genes involved in various mechanisms of the infective cycle of 42 isolates that have different growth rates, thermotolerance and virulence. The analysed genes showed general genetic diversity measured as non-synonymous changes (NSC) and copy number variation (CNV), with most of them being subjected to positive episodic diversifying selection. Correlation analyses between NSC or CNV and the isolate virulence, thermotolerance and growth rate revealed that various genes shaped the biological features of the fungus. Lectin-like, mucin signalling, Biotrophy associated and chitinase genes NSCs correlated with the three biological features of B. bassiana. In addition, other genes (i.e. DNA photolyase and cyclophilin B) that had relatively conserved sequences, had variable CNs across the isolates which were correlated with the variability of either virulence or thermotolerance of B. bassiana isolates. The data obtained is important for a better understanding of population structure, ecological and potential impact when isolates are used as mycoinsecticides and can justify industrialization of new isolates.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00874-x" target="_blank" style="color:black; text-decoration:none">Specificity of assemblage, not fungal partner species, explains mycorrhizal partnerships of mycoheterotrophic Burmannia plants</a></h1>
            <p>Zhongtao Zhao, Xiaojuan Li, Ming Fai Liu, Vincent S. F. T. Merckx, Richard M. K. Saunders, Dianxiang Zhang</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2021-01-06</pre>
            <h2>Abstract</h2>
            <font size="4">Mycoheterotrophic plants (MHPs) growing on arbuscular mycorrhizal fungi (AMF) usually maintain specialized mycorrhizal associations. The level of specificity varies between MHPs, although it remains largely unknown whether interactions with mycorrhizal fungi differ by plant lineage, species, and/or by population. Here, we investigate the mycorrhizal interactions among Burmannia species (Burmanniaceae) with different trophic modes using high-throughput DNA sequencing. We characterized the inter- and intraspecific dynamics of the fungal communities by assessing the composition and diversity of fungi among sites. We found that fully mycoheterotrophic species are more specialized in their fungal associations than chlorophyllous species, and that this specialization possibly results from the gradual loss of some fungal groups. In particular, although many fungal species were shared by different Burmannia species, fully MHP species typically host species-specific fungal assemblages, suggesting that they have a preference for the selected fungi. Although no apparent cophylogenetic relationship was detected between fungi and plants, we observe that evolutionarily closely related plants tend to have a greater proportion of shared or closely related fungal partners. Our findings suggest a host preference and specialization toward fungal assemblages in Burmannia, improving understanding of interactions between MHPs and fungi.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-79874-y" target="_blank" style="color:black; text-decoration:none">Metagenomic analysis of formalin-fixed paraffin-embedded tumor and normal mucosa reveals differences in the microbiome of colorectal cancer patients</a></h1>
            <p>Gabriela Debesa-Tur, Vicente Pérez-Brocal, Susana Ruiz-Ruiz, Adela Castillejo, Amparo Latorre, José Luis Soto, Andrés Moya</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">An increased risk of developing colorectal cancer (CRC) and other types of tumor is associated to Lynch syndrome (LS), an inherited condition caused by germline mutations in mismatch repair genes. We selected a cohort of LS patients that had developed CRC and had undergone surgical resection. Formalin-fixed paraffin embedded (FFPE) tissue blocks from matched colorectal and normal mucosa were used for genomic DNA extraction with a commercial kit and sequenced by high-throughput sequencing. A metagenomic approach enabled the taxonomic and functional identification of the microbial community and associated genes detected in the specimens. Slightly lower taxonomic diversity was observed in the tumor compared to the non-tumor tissue. Furthermore, the most remarkable differences between tumors and healthy tissue was the significant increase in the genus Fusobacterium in the former, in particular the species F. nucleatum, as well as Camplylobacter or Bacteroides fragilis, in accordance with previous studies of CRC. However, unlike prior studies, the present work is not based on directed detection by qPCR but instead uses a metagenomic approach to retrieve the whole bacterial community, and addresses the additional difficulty of using long-term stored FFPE samples.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-79712-1" target="_blank" style="color:black; text-decoration:none">Genetic and demographic history define a conservation strategy for earth’s most endangered pinniped, the Mediterranean monk seal Monachus monachus</a></h1>
            <p>Alexandros A. Karamanlidis, Tomaž Skrbinšek, George Amato, Panagiotis Dendrinos, Stephen Gaughran, Panagiotis Kasapidis, Alexander Kopatz, Astrid Vik Stronen</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">The Mediterranean monk seal (Monachus monachus) is a flagship species for marine conservation, but important aspects of its life history remain unknown. Concerns over imminent extinction motivated a nuclear DNA study of the species in its largest continuous subpopulation in the eastern Mediterranean Sea. Despite recent evidence of partial subpopulation recovery, we demonstrate that there is no reason for complacency, as the species still shares several traits that are characteristic of a critically endangered species: Mediterranean monk seals in the eastern Mediterranean survive in three isolated and genetically depauperate population clusters, with small effective population sizes and high levels of inbreeding. Our results indicated male philopatry over short distances, which is unexpected for a polygynous mammal. Such a pattern may be explained by the species’ unique breeding behavior, in which males defend aquatic territories near breeding sites, while females are often forced to search for new pupping areas. Immediate action is necessary to reverse the downward spiral of population decline, inbreeding accumulation and loss of genetic diversity. We propose concrete conservation measures for the Mediterranean monk seal focusing on reducing anthropogenic threats, increasing the population size and genetic diversity, and thus improving the long-term prospects of survival.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80236-x" target="_blank" style="color:black; text-decoration:none">Metabarcoding profiling of microbial diversity associated with trout fish farming</a></h1>
            <p>Mohamed A. A. Mahmoud, Mahmoud Magdy</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">Earthy and musty off-flavors are routinely observed in farmed trout worldwide. The microbial association to the production of those off-flavors was previously reported. The current manuscript aimed to catalog the microbial enrichment (eukaryotes and prokaryotes) in semi-intensive aquaculture freshwater sources that might influence the trout aquaculture quality production. The 16S rRNA and ITS metabarcoding analyses were applied on the inflow- and pond-water samples from trout farms previously recorded a malodor fish products and located alongside Moosach and Sempt Rivers in Bavaria province, Germany. The results showed that more than 99% of the detected prokaryotic OTUs (Operational Taxonomic Unit identification) were bacteria as of ~ 75.57% were Proteobacteria, and ~ 14.4% were Bacteroidetes. Meanwhile, 118 out of 233 of the eukaryotic OTUs were known species. Of these, ~ 45% were plant pathogens, and ~ 28% were mushroom/yeasts. Based on the comparative analysis between inflow- and pond-water samples, several pro- and eukaryotic microorganisms that affect the trout aquaculture water quality and industry have been detected, including the malodor-producing microorganisms, e.g., Cyanobacteria and Actinobacteria, along with fish infectious microorganisms, e.g., Chilodonella cyprinid, Metschnikowia bicuspidate. Additionally, the effect of the human- and industrial-related activities around the sampling area on the microbiota of the investigated farms were highlighted.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-76917-2" target="_blank" style="color:black; text-decoration:none">Different response of the taxonomic, phylogenetic and functional diversity of birds to forest fragmentation</a></h1>
            <p>Michał Bełcik, Magdalena Lenda, Tatsuya Amano, Piotr Skórka</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-23</pre>
            <h2>Abstract</h2>
            <font size="4">Habitat fragmentation is considered as major threat to biodiversity worldwide. Biodiversity can be described as taxonomic, functional and phylogenetic diversity. However, the effect of forest fragmentation on taxonomic, phylogenetic and functional diversity is barely understood. We compare the response of taxonomic (species richness), phylogenetic and functional diversity of birds to forest fragmentation. We hypothesised that with increasing forest patch isolation and/or decreasing patch size the diversity of birds decreases but only if certain thresholds of fragmentation metrics are reached. Specifically, we hypothesized that out of the three diversity components the taxonomic diversity is the most sensitive to forest fragmentation, which means that it starts declining at larger patch size and higher connectivity values than phylogenetic and functional diversity do. We compared the three biodiversity metrics of central European bird species in a large set of forest patches located in an agricultural landscape. General additive modeling and segmented regression were used in analyses. Habitat fragmentation differentially affected studied biodiversity metrics. Bird taxonomic diversity was the most responsive towards changes in fragmentation. We observed an increase in taxonomic diversity with increasing patch area, which then stabilized after reaching certain patch size. Functional diversity turned out to be the least responsive to the fragmentation metrics and forest stand characteristics. It decreased linearly with the decreasing isolation of forest patches. Apart from the habitat fragmentation, bird taxonomic diversity but not phylogenetic diversity was positively associated with forest stand age. The lower share of dominant tree species, the highest taxonomic diversity was. While preserving a whole spectrum of forests (in terms of age, fragmentation and size) is important from the biodiversity perspective, forest bird species might need large, intact, old-growth forests. Since the large and intact forest becomes scarcer, our study underscore their importance for the preservation of forest specialist species.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-19228-4" target="_blank" style="color:black; text-decoration:none">Local community assembly mechanisms shape soil bacterial β diversity patterns along a latitudinal gradient</a></h1>
            <p>Xiao Zhang, Shirong Liu, Jingxin Wang, Yongtao Huang, Zachary Freedman, Shenglei Fu, Kuan Liu, Hui Wang, Xiangzhen Li, Minjie Yao, Xiaojing Liu, Jamie Schuler</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-10-27</pre>
            <h2>Abstract</h2>
            <font size="4">Biodiversity patterns across geographical gradients could result from regional species pool and local community assembly mechanisms. However, little has been done to separate the effects of local ecological mechanisms from variation in the regional species pools on bacterial diversity patterns. In this study, we compare assembly mechanisms of soil bacterial communities in 660 plots from 11 regions along a latitudinal gradient in eastern China with highly divergent species pools. Our results show that β diversity does not co-vary with γ diversity, and local community assembly mechanisms appear to explain variation in β diversity patterns after correcting for variation in regional species pools. The variation in environmental conditions along the latitudinal gradient accounts for the variation in β diversity through mediating the strength of heterogeneous selection. In conclusion, our study clearly illustrates the importance of local community assembly processes in shaping geographical patterns of soil bacterial β diversity. The relative importance of regional species pool and local assembly processes as drivers of beta diversity is unclear. Here the authors investigate soil bacterial diversity patterns along a 3700-km latitudinal gradient in Chinese forests, finding that community assembly processes differ based on environmental heterogeneity.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-20442-3" target="_blank" style="color:black; text-decoration:none">HIV-1 diversity considerations in the application of the Intact Proviral DNA Assay (IPDA)</a></h1>
            <p>Natalie N. Kinloch, Yanqin Ren, Winiffer D. Conce Alberto, Winnie Dong, Pragya Khadka, Szu Han Huang, Talia M. Mota, Andrew Wilson, Aniqa Shahid, Don Kirkby, Marianne Harris, Colin Kovacs, Erika Benko, Mario A. Ostrowski, Perla M. Del Rio Estrada, Avery Wimpelberg, Christopher Cannon, W. David Hardy, Lynsay MacLaren, Harris Goldstein, Chanson J. Brumme, Guinevere Q. Lee, Rebecca M. Lynch, Zabrina L. Brumme, R. Brad Jones</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">The Intact Proviral DNA Assay (IPDA) was developed to address the critical need for a scalable method for intact HIV-1 reservoir quantification. This droplet digital PCR-based assay simultaneously targets two HIV-1 regions to distinguish genomically intact proviruses against a large background of defective ones, and its application has yielded insights into HIV-1 persistence. Reports of assay failures however, attributed to HIV-1 polymorphism, have recently emerged. Here, we describe a diverse North American cohort of people with HIV-1 subtype B, where the IPDA yielded a failure rate of 28% due to viral polymorphism. We further demonstrate that within-host HIV-1 diversity can lead the IPDA to underestimate intact reservoir size, and provide examples of how this phenomenon could lead to erroneous interpretation of clinical trial data. While the IPDA represents a major methodological advance, HIV-1 diversity should be addressed before its widespread adoption as a principal readout in HIV-1 remission trials. The intact proviral DNA assay quantifies the genomically intact HIV reservoir, but assay failure due to HIV-1 polymorphism has been observed. Here, the authors report a 28% failure rate in a cohort of people with HIV-1, and note within-host HIV-1 diversity as a further challenge to IPDA accuracy.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80387-x" target="_blank" style="color:black; text-decoration:none">Impact of improved small-scale livestock farming on human nutrition</a></h1>
            <p>Md. Emran Hossain, Md. Ahasanul Hoque, Emanuele Giorgi, Guillaume Fournié, Goutam Buddha Das, Joerg Henning</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Micronutrient deficiencies and stunting rates are high in many low-income countries. Increasing and diversifying food intake are often challenging for small-scale farmers in lowland areas as flooding often results in crop losses and drowning of livestock. A cluster-randomised controlled trial was conducted over 12-months in Bangladesh, involving 150 small-scale duck rearing households, including 50 control, and 50 households each in two intervention arms. Interventions focussing on improving duck health and duck nutrition were applied on a village level. Data analysis focussed on assessing the impact of interventions on duck mortality, sales and consumption, and on dietary diversity of household members. Improved duck rearing increased the consumption and the sales of ducks. Household selling more ducks were more likely to purchase and consume milk products, contributing to an improved households’ dietary diversity. Our results suggest that improving duck rearing can provide a suitable and sustainable alternative to maintain and improve dietary diversity of households in flood-prone areas.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78736-x" target="_blank" style="color:black; text-decoration:none">Urban fragmentation leads to lower floral diversity, with knock-on impacts on bee biodiversity</a></h1>
            <p>Panagiotis Theodorou, Sarah-Christine Herbst, Belinda Kahnt, Patricia Landaverde-González, Lucie M. Baltz, Julia Osterman, Robert J. Paxton</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-10</pre>
            <h2>Abstract</h2>
            <font size="4">Bees and flowering plants are two closely interacting groups of organisms. Habitat loss and fragmentation associated with urbanisation are major threats to both partners. Yet how and why bee and floral richness and diversity co-vary within the urban landscape remain unclear. Here, we sampled bees and flowering plants in urban green spaces to investigate how bee and flowering plant species richness, their phylogenetic diversity and pollination-relevant functional trait diversity influence each other in response to urban fragmentation. As expected, bee abundance and richness were positively related to flowering plant richness, with bee body size (but not bee richness and diversity) increasing with nectar-holder depth of flowering plants. Causal modelling indicated that bottom-up effects dictated patterns of bee-flower relationships, with urban fragmentation diminishing flowering plants richness and thereby indirectly reducing bee species richness and abundance. The close relationship between bees and flowering plants highlights the risks of their parallel declines in response to land-use change within the urban landscape.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41576-020-00306-8" target="_blank" style="color:black; text-decoration:none">African genetic diversity and adaptation inform a precision medicine agenda</a></h1>
            <p>Luisa Pereira, Leon Mutesa, Paulina Tindana, Michèle Ramsay</p>
            <pre style="font-family:timesnewroman">Journal: Nature Reviews Genetics    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">The deep evolutionary history of African populations, since the emergence of modern humans more than 300,000 years ago, has resulted in high genetic diversity and considerable population structure. Selected genetic variants have increased in frequency due to environmental adaptation, but recent exposures to novel pathogens and changes in lifestyle render some of them with properties leading to present health liabilities. The unique discoverability potential from African genomic studies promises invaluable contributions to understanding the genomic and molecular basis of health and disease. Globally, African populations are understudied, and precision medicine approaches are largely based on data from European and Asian-ancestry populations, which limits the transferability of findings to the continent of Africa. Africa needs innovative precision medicine solutions based on African data that use knowledge and implementation strategies aligned to its climatic, cultural, economic and genomic diversity. Africa is a continent with deep evolutionary history, which has implications for the genetic underpinnings of disease. In this Review, the authors discuss how genetic features of African populations provide both challenges and opportunities for understanding disease genetics in Africa. They describe how this genetic knowledge — combined with initiatives including capacity-building, data sharing and increased representation of African genomes in genetic variation databases — can be leveraged towards achieving precision medicine approaches in African healthcare.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00855-0" target="_blank" style="color:black; text-decoration:none">Coinfection and infection duration shape how pathogens affect the African buffalo gut microbiota</a></h1>
            <p>Kate A. Sabey, Se Jin Song, Anna Jolles, Rob Knight, Vanessa O. Ezenwa</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2020-12-17</pre>
            <h2>Abstract</h2>
            <font size="4">Changes in the gut microbiota during pathogen infection are often predicted to influence disease outcomes. However, studies exploring whether pathogens induce microbiota shifts have yielded inconsistent results. This suggests that variation in infection, rather than the presence of infection alone, might shape pathogen–microbiota relationships. For example, most hosts are coinfected with multiple pathogens simultaneously, and hosts vary in how long they are infected, which may amplify or diminish microbial shifts expected in response to a focal pathogen. We used a longitudinal anthelmintic treatment study of free-ranging African buffalo (Syncerus caffer) to examine whether (i) coinfection with bovine tuberculosis (Mycobacterium bovis, TB) and gastrointestinal nematodes, and (ii) the duration of TB infection, modified effects of single pathogens on the gut microbiota. By accounting for the interaction between TB and nematodes, we found that coinfection affected changes in microbial abundance associated with single infections. Furthermore, the duration of TB infection predicted more microbiota variation than the presence of TB. Importantly, coinfection and infection duration had nearly as much influence on microbial patterns as demographic and environmental factors commonly examined in microbiota research. These findings demonstrate that acknowledging infection heterogeneities may be crucial to understanding relationships between pathogens and the gut microbiota.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41564-020-00831-6" target="_blank" style="color:black; text-decoration:none">Bifidobacterium bifidum strains synergize with immune checkpoint inhibitors to reduce tumour burden in mice</a></h1>
            <p>Se-Hoon Lee, Sung-Yup Cho, Youngmin Yoon, Changho Park, Jinyoung Sohn, Jin-Ju Jeong, Bu-Nam Jeon, Mongjoo Jang, Choa An, Suro Lee, Yun Yeon Kim, Gihyeon Kim, Sujeong Kim, Yunjae Kim, Gwang Bin Lee, Eun Ju Lee, Sang Gyun Kim, Hong Sook Kim, Yeongmin Kim, Hyun Kim, Hyun-Suk Yang, Sarang Kim, Seonggon Kim, Hayung Chung, Myeong Hee Moon, Myung Hee Nam, Jee Young Kwon, Sungho Won, Joon-Suk Park, George M. Weinstock, Charles Lee, Kyoung Wan Yoon, Hansoo Park</p>
            <pre style="font-family:timesnewroman">Journal: Nature Microbiology    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">The gut microbiome can influence the development of tumours and the efficacy of cancer therapeutics1–5; however, the multi-omics characteristics of antitumour bacterial strains have not been fully elucidated. In this study, we integrated metagenomics, genomics and transcriptomics of bacteria, and analyses of mouse intestinal transcriptome and serum metabolome data to reveal an additional mechanism by which bacteria determine the efficacy of cancer therapeutics. In gut microbiome analyses of 96 samples from patients with non-small-cell lung cancer, Bifidobacterium bifidum was abundant in patients responsive to therapy. However, when we treated syngeneic mouse tumours with commercial strains of B. bifidum to establish relevance for potential therapeutic uses, only specific B. bifidum strains reduced tumour burden synergistically with PD-1 blockade or oxaliplatin treatment by eliciting an antitumour host immune response. In mice, these strains induced tuning of the immunological background by potentiating the production of interferon-γ, probably through the enhanced biosynthesis of immune-stimulating molecules and metabolites. An analysis of gut microbiomes of patients with non-small-cell lung cancer reveals an association between Bifidobacterium bifidum abundance and response to cancer therapy. In murine models of syngeneic tumours, administration of commercial B. bifidum strains synergizes with immune checkpoint blockade to reduce tumour burden, but the therapeutic potential of B. bifidum is affected by strain-level variation.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41591-020-01183-8" target="_blank" style="color:black; text-decoration:none">Microbiome connections with host metabolism and habitual diet from 1,098 deeply phenotyped individuals</a></h1>
            <p>Francesco Asnicar, Sarah E. Berry, Ana M. Valdes, Long H. Nguyen, Gianmarco Piccinno, David A. Drew, Emily Leeming, Rachel Gibson, Caroline Le Roy, Haya Al Khatib, Lucy Francis, Mohsen Mazidi, Olatz Mompeo, Mireia Valles-Colomer, Adrian Tett, Francesco Beghini, Léonard Dubois, Davide Bazzani, Andrew Maltez Thomas, Chloe Mirzayi, Asya Khleborodova, Sehyun Oh, Rachel Hine, Christopher Bonnett, Joan Capdevila, Serge Danzanvilliers, Francesca Giordano, Ludwig Geistlinger, Levi Waldron, Richard Davies, George Hadjigeorgiou, Jonathan Wolf, José M. Ordovás, Christopher Gardner, Paul W. Franks, Andrew T. Chan, Curtis Huttenhower, Tim D. Spector, Nicola Segata</p>
            <pre style="font-family:timesnewroman">Journal: Nature Medicine    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">The gut microbiome is shaped by diet and influences host metabolism; however, these links are complex and can be unique to each individual. We performed deep metagenomic sequencing of 1,203 gut microbiomes from 1,098 individuals enrolled in the Personalised Responses to Dietary Composition Trial (PREDICT 1) study, whose detailed long-term diet information, as well as hundreds of fasting and same-meal postprandial cardiometabolic blood marker measurements were available. We found many significant associations between microbes and specific nutrients, foods, food groups and general dietary indices, which were driven especially by the presence and diversity of healthy and plant-based foods. Microbial biomarkers of obesity were reproducible across external publicly available cohorts and in agreement with circulating blood metabolites that are indicators of cardiovascular disease risk. While some microbes, such as Prevotella copri and Blastocystis spp., were indicators of favorable postprandial glucose metabolism, overall microbiome composition was predictive for a large panel of cardiometabolic blood markers including fasting and postprandial glycemic, lipemic and inflammatory indices. The panel of intestinal species associated with healthy dietary habits overlapped with those associated with favorable cardiometabolic and postprandial markers, indicating that our large-scale resource can potentially stratify the gut microbiome into generalizable health levels in individuals without clinically manifest disease. Analyses from the gut microbiome of over 1,000 individuals from the PREDICT 1 study, for which detailed long-term diet information as well as hundreds of fasting and same-meal postprandial cardiometabolic blood marker measurements are available, unveil new associations between specific gut microbes, dietary habits and cardiometabolic health.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-79320-z" target="_blank" style="color:black; text-decoration:none">Bioactive diterpenoids impact the composition of the root-associated microbiome in maize ( Zea mays )</a></h1>
            <p>Katherine M. Murphy, Joseph Edwards, Katherine B. Louie, Benjamin P. Bowen, Venkatesan Sundaresan, Trent R. Northen, Philipp Zerbe</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">Plants deploy both primary and species-specific, specialized metabolites to communicate with other organisms and adapt to environmental challenges, including interactions with soil-dwelling microbial communities. However, the role of specialized metabolites in modulating plant-microbiome interactions often remains elusive. In this study, we report that maize (Zea mays) diterpenoid metabolites with known antifungal bioactivities also influence rhizosphere bacterial communities. Metabolite profiling showed that dolabralexins, antibiotic diterpenoids that are highly abundant in roots of some maize varieties, can be exuded from the roots. Comparative 16S rRNA gene sequencing determined the bacterial community composition of the maize mutant Zman2 (anther ear 2), which is deficient in dolabralexins and closely related bioactive kauralexin diterpenoids. The Zman2 rhizosphere microbiome differed significantly from the wild-type sibling with the most significant changes observed for Alphaproteobacteria of the order Sphingomonadales. Metabolomics analyses support that these differences are attributed to the diterpenoid deficiency of the Zman2 mutant, rather than other large-scale metabolome alterations. Together, these findings support physiological functions of maize diterpenoids beyond known chemical defenses, including the assembly of the rhizosphere microbiome.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-76658-2" target="_blank" style="color:black; text-decoration:none">Diversity begets diversity in mammal species and human cultures</a></h1>
            <p>Marcus J. Hamilton, Robert S. Walker, Christopher P. Kempes</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-12</pre>
            <h2>Abstract</h2>
            <font size="4">Across the planet the biogeographic distribution of human cultural diversity tends to correlate positively with biodiversity. In this paper we focus on the biogeographic distribution of mammal species and human cultural diversity. We show that not only are these forms of diversity similarly distributed in space, but they both scale superlinearly with environmental production. We develop theory that explains that as environmental productivity increases the ecological kinetics of diversity increases faster than expected because more complex environments are also more interactive. Using biogeographic databases of the global distributions of mammal species and human cultures we test a series of hypotheses derived from this theory and find support for each. For both mammals and cultures, we show that (1) both forms of diversity increase exponentially with ecological kinetics; (2) the kinetics of diversity is faster than the kinetics of productivity; (3) diversity scales superlinearly with environmental productivity; and (4) the kinetics of diversity is faster in increasingly productive environments. This biogeographic convergence is particularly striking because while the dynamics of biological and cultural evolution may be similar in principle the underlying mechanisms and time scales are very different. However, a common currency underlying all forms of diversity is ecological kinetics; the temperature-dependent fluxes of energy and biotic interactions that sustain all forms of life at all levels of organization. Diversity begets diversity in mammal species and human cultures because ecological kinetics drives superlinear scaling with environmental productivity.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00842-5" target="_blank" style="color:black; text-decoration:none">Distinct ecotypes within a natural haloarchaeal population enable adaptation to changing environmental conditions without causing population sweeps</a></h1>
            <p>Tomeu Viver, Roth E. Conrad, Luis H. Orellana, Mercedes Urdiain, José E. González-Pastor, Janet K. Hatt, Rudolf Amann, Josefa Antón, Konstantinos T. Konstantinidis, Ramon Rosselló-Móra</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2020-12-20</pre>
            <h2>Abstract</h2>
            <font size="4">Microbial communities thriving in hypersaline brines of solar salterns are highly resistant and resilient to environmental changes, and salinity is a major factor that deterministically influences community structure. Here, we demonstrate that this resilience occurs even after rapid osmotic shocks caused by a threefold change in salinity (a reduction from 34 to 12% salts) leading to massive amounts of archaeal cell lysis. Specifically, our temporal metagenomic datasets identified two co-occurring ecotypes within the most dominant archaeal population of the brines Haloquadratum walsbyi that exhibited different salt concentration preferences. The dominant ecotype was generally more abundant and occurred in high-salt conditions (34%); the low abundance ecotype always co-occurred but was enriched at salinities around 20% or lower and carried unique gene content related to solute transport and gene regulation. Despite their apparent distinct ecological preferences, the ecotypes did not outcompete each other presumably due to weak functional differentiation between them. Further, the osmotic shock selected for a temporal increase in taxonomic and functional diversity at both the Hqr. walsbyi population and whole-community levels supporting the specialization-disturbance hypothesis, that is, the expectation that disturbance favors generalists. Altogether, our results provide new insights into how intraspecies diversity is maintained in light of substantial gene-content differences and major environmental perturbations.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80543-3" target="_blank" style="color:black; text-decoration:none">First reported quantitative microbiota in different livestock manures used as organic fertilizers in the Northeast of Thailand</a></h1>
            <p>Lampet Wongsaroj, Ratmanee Chanabun, Naruemon Tunsakul, Pinidphon Prombutara, Somsak Panha, Naraporn Somboonna</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Northeastern Thailand relies on agriculture as a major economic activity, and has used high levels of agrochemicals due to low facility, and salty sandy soil. To support soil recovery and sustainable agriculture, local farmers have used organic fertilizers from farmed animal feces. However, knowledge about these animal fecal manures remains minimal restricting their optimal use. Specifically, while bacteria are important for soil and plant growth, an abundance and a diversity of bacterial composition in these animal fecal manures have not been reported to allow selection and adjustment for a more effective organic fertilizer. This study thereby utilized metagenomics combined with 16S rRNA gene quantitative PCR (qPCR) and sequencing to analyze quantitative microbiota profiles in association with nutrients (N, P, K), organic matters, and the other physiochemical properties, of the commonly used earthworm manure and other manures from livestock animals (including breed and feeding diet variations) in the region. Unlike the other manures, the earthworm manure demonstrated more favorable nutrient profiles and physiochemical properties for forming fertile soil. Despite low total microbial biomass, the microbiota were enriched with maximal OTUs and Chao richness, and no plant pathogenic bacteria were found based on the VFDB database. The microbial metabolic potentials supported functions to promote crop growth, such as C, N and P cyclings, xenobiotic degradation, and synthesis of bioactive compounds. Pearson’s correlation analyses indicated that the quantitative microbiota of the earthworm manure were clustered in the same direction as N, and conductivity, salinity, and water content were essential to control the microbiota of animal manures.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80633-2" target="_blank" style="color:black; text-decoration:none">Potential use of molecular and structural characterization of the gut bacterial community for postmortem interval estimation in Sprague Dawley rats</a></h1>
            <p>Huan Li, Siruo Zhang, Ruina Liu, Lu Yuan, Di Wu, E. Yang, Han Yang, Shakir Ullah, Hafiz Muhammad Ishaq, Hailong Liu, Zhenyuan Wang, Jiru Xu</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Once the body dies, the indigenous microbes of the host begin to break down the body from the inside and play a key role thereafter. This study aimed to investigate the probable shift in the composition of the rectal microbiota at different time intervals up to 15 days after death and to explore bacterial taxa important for estimating the time since death. At the phylum level, Proteobacteria and Firmicutes showed major shifts when checked at 11 different intervals and emerged at most of the postmortem intervals. At the species level, Enterococcus faecalis and Proteus mirabilis showed a downward and upward trend, respectively, after day 5 postmortem. The phylum-, family-, genus-, and species-taxon richness decreased initially and then increased considerably. The turning point occurred on day 9, when the genus, rather than the phylum, family, or species, provided the most information for estimating the time since death. We constructed a prediction model using genus-level data from high-throughput sequencing, and seven bacterial taxa, namely, Enterococcus, Proteus, Lactobacillus, unidentified Clostridiales, Vagococcus, unidentified Corynebacteriaceae, and unidentified Enterobacteriaceae, were included in this model. The abovementioned bacteria showed potential for estimating the shortest time since death.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-79478-6" target="_blank" style="color:black; text-decoration:none">The fecal mycobiome in patients with Irritable Bowel Syndrome</a></h1>
            <p>A. Das, E. O’Herlihy, F. Shanahan, P. W. O’Toole, I. B. Jeffery</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Alterations of the gut microbiota have been reported in various gastrointestinal disorders, but knowledge of the mycobiome is limited. We investigated the gut mycobiome of 80 patients with Irritable Bowel Syndrome (IBS) in comparison with 64 control subjects. The fungal-specific internal transcribed spacer 1 (ITS-1) amplicon was sequenced, and mycobiome zero-radius operational taxonomic units (zOTUs) were defined representing known and unknown species and strains. The fungal community was sparse and individual-specific in all (both IBS and control) subjects. Although beta-diversity differed significantly between IBS and controls, no difference was found among clinical subtypes of IBS or in comparison with the mycobiome of subjects with bile acid malabsorption (BAM), a condition which may overlap with IBS with diarrhoea. The mycobiome alterations co-varied significantly with the bacteriome and metabolome but were not linked with dietary habits. As a putative biomarker of IBS, the predictive power of the fecal mycobiome in machine learning models was significantly better than random but insufficient for clinical diagnosis. The mycobiome presents limited therapeutic and diagnostic potential for IBS, despite co-variation with bacterial components which do offer such potential.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-19989-y" target="_blank" style="color:black; text-decoration:none">Using metacommunity ecology to understand environmental metabolomes</a></h1>
            <p>Robert E. Danczak, Rosalie K. Chu, Sarah J. Fansler, Amy E. Goldman, Emily B. Graham, Malak M. Tfaily, Jason Toyoda, James C. Stegen</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-12-11</pre>
            <h2>Abstract</h2>
            <font size="4">Environmental metabolomes are fundamentally coupled to microbially-linked biogeochemical processes within ecosystems. However, significant gaps exist in our understanding of their spatiotemporal organization, limiting our ability to uncover transferrable principles and predict ecosystem function. We propose that a theoretical paradigm, which integrates concepts from metacommunity ecology, is necessary to reveal underlying mechanisms governing metabolomes. We call this synthesis between ecology and metabolomics ‘meta-metabolome ecology’ and demonstrate its utility using a mass spectrometry dataset. We developed three relational metabolite dendrograms using molecular properties and putative biochemical transformations and performed ecological null modeling. Based upon null modeling results, we show that stochastic processes drove molecular properties while biochemical transformations were structured deterministically. We further suggest that potentially biochemically active metabolites were more deterministically assembled than less active metabolites. Understanding variation in the influences of stochasticity and determinism provides a way to focus attention on which meta-metabolomes and which parts of meta-metabolomes are most likely to be important to consider in mechanistic models. We propose that this paradigm will allow researchers to study the connections between ecological systems and their molecular processes in previously inaccessible detail. Despite growing interest in environmental metabolomics, we lack conceptual frameworks for considering how metabolites vary across space and time in ecological systems. Here, the authors apply (species) community assembly concepts to metabolomics data, offering a way forward in understanding the assembly of metabolite assemblages.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41558-020-00959-7" target="_blank" style="color:black; text-decoration:none">Genomic evidence of past and future climate-linked loss in a migratory Arctic fish</a></h1>
            <p>K. K. S. Layton, P. V. R. Snelgrove, J. B. Dempson, T. Kess, S. J. Lehnert, P. Bentzen, S. J. Duffy, A. M. Messmer, R. R. E. Stanley, C. DiBacco, S. J. Salisbury, D. E. Ruzzante, C. M. Nugent, M. M. Ferguson, J. S. Leong, B. F. Koop, I. R. Bradbury</p>
            <pre style="font-family:timesnewroman">Journal: Nature Climate Change    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">Despite widespread biodiversity losses, an understanding of how most taxa will respond to future climate change is lacking. Here we integrate genomics and environmental modelling to assess climate change responses in an ecologically and economically important Arctic species. Environmentally associated genomic diversity and machine learning are used to identify highly vulnerable populations of anadromous (migratory) Arctic charr, and we reconstruct estimates of effective population size spanning the twentieth century to identify past climate-associated declines. We uncover past region-wide declines in effective population size that correspond to decreases in temperature and community biomass in the Northwest Atlantic. We find vulnerable populations near the southern range limit, indicating northward shifts and a possible loss of commercially important life-history variation in response to climate change. The genomic approach used here to investigate climate change response identifies past and future declines that impact species persistence, ecosystem stability and food security in the Arctic. Genomics and environmental modelling are integrated to assess past and future changes in Arctic charr populations in response to changing climate. Southern population vulnerability suggests climate change may lead to northward shifts and the loss of important life-history variation.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78300-7" target="_blank" style="color:black; text-decoration:none">Phytoplankton alpha diversity indices response the trophic state variation in hydrologically connected aquatic habitats in the Harbin Section of the Songhua River</a></h1>
            <p>Fanhua Meng, Zhenxiang Li, Lei Li, Feng Lu, Yan Liu, Xinxin Lu, Yawen Fan</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-07</pre>
            <h2>Abstract</h2>
            <font size="4">The relationship between biodiversity and ecological functioning is a central issue in freshwater ecology, but how this relationship is influenced by hydrological connectivity stress is still unknown. In this study we analyzed the dynamic of the phytoplankton alpha diversity indices and their relationships with trophic state in two hydrologically connected aquatic habitats (Jinhewan Wetland and Harbin Section of the Songhua River) in the Songhua River Basin in northeast China. We hypothesized that the phytoplankton alpha-diversity indices have the potential to provide a signal linking trophic state variation in hydrologically connected aquatic habitats. Our results showed the Cyanophyta and Bacillariophyta were abundant at most stations. T-test showed that phytoplankton alpha diversity indices varied significantly between rainy season and dry season. Trophic State Index recorded that a meso-trophic to eutrophic states of two connected habits during study period. Multivariate statistical analysis revealed that the dynamic of phytoplankton alpha diversity index was closely associated with trophic states change. Our result indicated that hydrological connectivity is a key factor influenced phytoplankton community assembly. In addition, it is beneficial to develop an integrated approach to appropriately describe and measure the trophic state variations of hydrologically connected aquatic habits in freshwater ecosystem.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00867-w" target="_blank" style="color:black; text-decoration:none">Negative frequency-dependent selection and asymmetrical transformation stabilise multi-strain bacterial population structures</a></h1>
            <p>Gabrielle L. Harrow, John A. Lees, William P. Hanage, Marc Lipsitch, Jukka Corander, Caroline Colijn, Nicholas J. Croucher</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2021-01-06</pre>
            <h2>Abstract</h2>
            <font size="4">Streptococcus pneumoniae can be divided into many strains, each a distinct set of isolates sharing similar core and accessory genomes, which co-circulate within the same hosts. Previous analyses suggested the short-term vaccine-associated dynamics of S. pneumoniae strains may be mediated through multi-locus negative frequency-dependent selection (NFDS), which maintains accessory loci at equilibrium frequencies. Long-term simulations demonstrated NFDS stabilised clonally-evolving multi-strain populations through preventing the loss of variation through drift, based on polymorphism frequencies, pairwise genetic distances and phylogenies. However, allowing symmetrical recombination between isolates evolving under multi-locus NFDS generated unstructured populations of diverse genotypes. Replication of the observed data improved when multi-locus NFDS was combined with recombination that was instead asymmetrical, favouring deletion of accessory loci over insertion. This combination separated populations into strains through outbreeding depression, resulting from recombinants with reduced accessory genomes having lower fitness than their parental genotypes. Although simplistic modelling of recombination likely limited these simulations’ ability to maintain some properties of genomic data as accurately as those lacking recombination, the combination of asymmetrical recombination and multi-locus NFDS could restore multi-strain population structures from randomised initial populations. As many bacteria inhibit insertions into their chromosomes, this combination may commonly underlie the co-existence of strains within a niche.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-20399-3" target="_blank" style="color:black; text-decoration:none">Optimizing respiratory virus surveillance networks using uncertainty propagation</a></h1>
            <p>Sen Pei, Xian Teng, Paul Lewis, Jeffrey Shaman</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">Infectious disease prevention, control and forecasting rely on sentinel observations; however, many locations lack the capacity for routine surveillance. Here we show that, by using data from multiple sites collectively, accurate estimation and forecasting of respiratory diseases for locations without surveillance is feasible. We develop a framework to optimize surveillance sites that suppresses uncertainty propagation in a networked disease transmission model. Using influenza outbreaks from 35 US states, the optimized system generates better near-term predictions than alternate systems designed using population and human mobility. We also find that monitoring regional population centers serves as a reasonable proxy for the optimized network and could direct surveillance for diseases with limited records. The  proxy method is validated using model simulations for 3,108 US counties and historical data for two other respiratory pathogens – human metapneumovirus and seasonal coronavirus – from 35 US states and can be used to guide systemic allocation of surveillance efforts. Lack of a widespread surveillance network hampers accurate infectious disease forecasting. Here the authors provide a framework to optimize the selection of surveillance site locations and show that accurate forecasting of respiratory diseases for locations without surveillance is feasible.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-19793-8" target="_blank" style="color:black; text-decoration:none">Vitamin D metabolites and the gut microbiome in older men</a></h1>
            <p>Robert L. Thomas, Lingjing Jiang, John S. Adams, Zhenjiang Zech Xu, Jian Shen, Stefan Janssen, Gail Ackermann, Dirk Vanderschueren, Steven Pauwels, Rob Knight, Eric S. Orwoll, Deborah M. Kado</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-11-26</pre>
            <h2>Abstract</h2>
            <font size="4">The vitamin D receptor is highly expressed in the gastrointestinal tract where it transacts gene expression. With current limited understanding of the interactions between the gut microbiome and vitamin D, we conduct a cross-sectional analysis of 567 older men quantifying serum vitamin D metabolites using LC-MSMS and defining stool sub-Operational Taxonomic Units from16S ribosomal RNA gene sequencing data. Faith’s Phylogenetic Diversity and non-redundant covariate analyses reveal that the serum 1,25(OH)2D level explains 5% of variance in α-diversity. In β-diversity analyses using unweighted UniFrac, 1,25(OH)2D is the strongest factor assessed, explaining 2% of variance. Random forest analyses identify 12 taxa, 11 in the phylum Firmicutes, eight of which are positively associated with either 1,25(OH)2D and/or the hormone-to-prohormone [1,25(OH)2D/25(OH)D] “activation ratio.” Men with higher levels of 1,25(OH)2D and higher activation ratios, but not 25(OH)D itself, are more likely to possess butyrate producing bacteria that are associated with better gut microbial health. Here, the authors investigate associations of vitamin D metabolites with gut microbiome in a cross-sectional analysis of 567 elderly men enrolled in the Osteoporotic Fractures in Men (MrOS) Study and find larger alpha-diversity correlates with high 1,25(OH)2D and high 24,25(OH)2D and higher ratios of activation and catabolism.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s42003-020-01551-7" target="_blank" style="color:black; text-decoration:none">Skin T cells maintain their diversity and functionality in the elderly</a></h1>
            <p>Hanako Koguchi-Yoshioka, Elena Hoffer, Stanley Cheuk, Yutaka Matsumura, Sa Vo, Petra Kjellman, Lucian Grema, Yosuke Ishitsuka, Yoshiyuki Nakamura, Naoko Okiyama, Yasuhiro Fujisawa, Manabu Fujimoto, Liv Eidsmo, Rachael A. Clark, Rei Watanabe</p>
            <pre style="font-family:timesnewroman">Journal: Communications Biology    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Recent studies have highlighted that human resident memory T cells (TRM) are functionally distinct from circulating T cells. Thus, it can be postulated that skin T cells age differently from blood-circulating T cells. We assessed T-cell density, diversity, and function in individuals of various ages to study the immunologic effects of aging on human skin from two different countries. No decline in the density of T cells was noted with advancing age, and the frequency of epidermal CD49a+ CD8 TRM was increased in elderly individuals regardless of ethnicity. T-cell diversity and antipathogen responses were maintained in the skin of elderly individuals but declined in the blood. Our findings demonstrate that in elderly individuals, skin T cells maintain their density, diversity, and protective cytokine production despite the reduced T-cell diversity and function in blood. Skin resident T cells may represent a long-lived, highly protective reservoir of immunity in elderly people. Koguchi-Yoshioka et al. demonstrate that functional activities and diversity of T cells are highly maintained in the skin of elderly people when compared with those in the blood regardless of the ethnicity of the participants or the experimental procedures. They also find the accumulation of epidermal T cells with maintained cytokine production in the elderly participants.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41586-020-2965-6" target="_blank" style="color:black; text-decoration:none">Crop asynchrony stabilizes food production</a></h1>
            <p>Lukas Egli, Matthias Schröter, Christoph Scherber, Teja Tscharntke, Ralf Seppelt</p>
            <pre style="font-family:timesnewroman">Journal: Nature    Published: 2020-12-09</pre>
            <h2>Abstract</h2>
            <font size="4"></font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-79130-3" target="_blank" style="color:black; text-decoration:none">Skin microbiome correlates with bioclimate and Batrachochytrium dendrobatidis infection intensity in Brazil’s Atlantic Forest treefrogs</a></h1>
            <p>Katharina Ruthsatz, Mariana L. Lyra, Carolina Lambertini, Anat M. Belasen, Thomas S. Jenkinson, Domingos da Silva Leite, C. Guilherme Becker, Célio F. B. Haddad, Timothy Y. James, Kelly R. Zamudio, Luís Felipe Toledo, Miguel Vences</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-18</pre>
            <h2>Abstract</h2>
            <font size="4">In Brazil’s Atlantic Forest (AF) biodiversity conservation is of key importance since the fungal pathogen Batrachochytrium dendrobatidis (Bd) has led to the rapid loss of amphibian populations here and worldwide. The impact of Bd on amphibians is determined by the host's immune system, of which the skin microbiome is a critical component. The richness and diversity of such cutaneous bacterial communities are known to be shaped by abiotic factors which thus may indirectly modulate host susceptibility to Bd. This study aimed to contribute to understanding the environment-host–pathogen interaction determining skin bacterial communities in 819 treefrogs (Anura: Hylidae and Phyllomedusidae) from 71 species sampled across the AF. We investigated whether abiotic factors influence the bacterial community richness and structure on the amphibian skin. We further tested for an association between skin bacterial community structure and Bd co-occurrence. Our data revealed that temperature, precipitation, and elevation consistently correlate with richness and diversity of the skin microbiome and also predict Bd infection status. Surprisingly, our data suggest a weak but significant positive correlation of Bd infection intensity and bacterial richness. We highlight the prospect of future experimental studies on the impact of changing environmental conditions associated with global change on environment-host–pathogen interactions in the AF.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80138-y" target="_blank" style="color:black; text-decoration:none">Gut microbiota dynamics in carnivorous European seabass ( Dicentrarchus labrax ) fed plant-based diets</a></h1>
            <p>Cláudia R. Serra, Aires Oliva-Teles, Paula Enes, Fernando Tavares</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">A healthy gastrointestinal microbiota is essential for host fitness, and strongly modulated by host diet. In aquaculture, a current challenge is to feed carnivorous fish with plant-feedstuffs in substitution of fish meal, an unsustainable commodity. Plants have a limited nutritive value due to the presence of non-starch polysaccharides (NSP) which are not metabolized by fish. In this work we assessed the effects of NSP-enriched diets on European seabass gut microbiota and evaluate the selective pressure of plant feedstuffs towards gut microbes with NSP-hydrolytic potential, i.e. capable to convert indigestible dietary constituents in fish metabolites. Triplicate groups of European seabass juveniles were fed a fish meal-based diet (control) or three plant-based diets (SBM, soybean meal; RSM, rapeseed meal; SFM, sunflower meal) for 6 weeks, before recovering intestinal samples for microbiota analysis, using the Illumina’s MiSeq platform. Plant-based diets impacted differently digesta and mucosal microbiota. A decrease (p = 0.020) on species richness, accompanied by a decline on the relative abundance of specific phyla such as Acidobacteria (p = 0.030), was observed in digesta samples of SBM and RSM experimental fish, but no effects were seen in mucosa-associated microbiota. Plant-based diets favored the Firmicutes (p = 0.01), in particular the Bacillaceae (p = 0.017) and Clostridiaceae (p = 0.007), two bacterial families known to harbor carbohydrate active enzymes and thus putatively more prone to grow in high NSP environments. Overall, bacterial gut communities of European seabass respond to plant-feedstuffs with adjustments in the presence of transient microorganisms (allochthonous) with carbohydrolytic potential, while maintaining a balanced core (autochthonous) microbiota.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41579-020-00482-8" target="_blank" style="color:black; text-decoration:none">Population bottlenecks and founder effects: implications for mosquito-borne arboviral emergence</a></h1>
            <p>Scott C. Weaver, Naomi L. Forrester, Jianying Liu, Nikos Vasilakis</p>
            <pre style="font-family:timesnewroman">Journal: Nature Reviews Microbiology    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">Transmission of arthropod-borne viruses (arboviruses) involves infection and replication in both arthropod vectors and vertebrate hosts. Nearly all arboviruses are RNA viruses with high mutation frequencies, which leaves them vulnerable to genetic drift and fitness losses owing to population bottlenecks during vector infection, dissemination from the midgut to the salivary glands and transmission to the vertebrate host. However, despite these bottlenecks, they seem to avoid fitness declines that can result from Muller’s ratchet. In addition, founder effects that occur during the geographic introductions of human-amplified arboviruses, including chikungunya virus and Zika virus, can affect epidemic and endemic circulation, as well as virulence. In this Review, we discuss the role of genetic drift following population bottlenecks and founder effects in arboviral evolution and spread, and the emergence of human disease. In this Review, Weaver and colleagues discuss the role of genetic drift following population bottlenecks and founder effects in mosquito-borne arboviral evolution and spread, and the emergence of human disease, focusing on chikungunya virus and Zika virus.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41398-020-01130-8" target="_blank" style="color:black; text-decoration:none">CD36 deficiency affects depressive-like behaviors possibly by modifying gut microbiota and the inflammasome pathway in mice</a></h1>
            <p>Shunjie Bai, Wei Wang, Ting Wang, Juan Li, Shuxiao Zhang, Zhi Chen, Xunzhong Qi, Jianjun Chen, Ke Cheng, Peng Xie</p>
            <pre style="font-family:timesnewroman">Journal: Translational Psychiatry    Published: 2021-01-05</pre>
            <h2>Abstract</h2>
            <font size="4">Both inflammatory processes and gut microbiota have been implicated in the pathophysiology of depressive disorders. The class B scavenger receptor CD36 is involved in the cytotoxicity associated with inflammation. However, its role in depression has not yet been examined. In this study, we investigated whether CD36 affects depression by modulating the microbiota-gut-inflammasome-brain axis. We used CD36−/− (knockout) mice subjected to chronic social defeat stress, and measured the expression of CD36 in these depressed mice and in patients with depression. The hippocampus of CD36−/− mice was used to investigate changes in the NLRP3 inflammasome signaling pathway. The 16S rRNA gene sequence-based approach was used to compare the cecal microbial communities in CD36−/− and WT mice. The CD36 deficiency in CD36−/− mice alleviated chronic stress-induced depression-like behaviors. CD36 was upregulated in depressed mice as well as in depressed patients. Furthermore, the NLRP3 inflammasome signaling pathway was downregulated in the hippocampus of CD36−/− mice. The Simpson Diversity Index revealed increased cecal bacterial alpha-diversity in the CD36−/− mice. Among genera, Bacteroides, Rikenella, and Alloprevotella were significantly more abundant in the CD36−/− mice, whereas Allobaculum was less abundant, consistent with the attenuated inflammation in the hippocampus of CD36−/− mice. Our findings suggest that CD36 deficiency changes the gut microbiota composition, which in turn may impact depressive-like behaviors by affecting the inflammasome pathway.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78891-1" target="_blank" style="color:black; text-decoration:none">An altered microbiome in urban coyotes mediates relationships between anthropogenic diet and poor health</a></h1>
            <p>Scott Sugden, Dana Sanderson, Kyra Ford, Lisa Y. Stein, Colleen Cassady St. Clair</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-17</pre>
            <h2>Abstract</h2>
            <font size="4">Generalist species able to exploit anthropogenic food sources are becoming increasingly common in urban environments. Coyotes (Canis latrans) are one such urban generalist that now resides in cities across North America, where diseased or unhealthy coyotes are frequently reported in cases of human-wildlife conflict. Coyote health and fitness may be related to habitat use and diet via the gut microbiome, which has far-reaching effects on animal nutrition and physiology. In this study, we used stomach contents, stable isotope analysis, 16S rRNA gene amplicon sequencing, and measures of body condition to identify relationships among habitat use, diet, fecal microbiome composition, and health in urban and rural coyotes. Three distinct relationships emerged: (1) Urban coyotes consumed more anthropogenic food, which was associated with increased microbiome diversity, higher abundances of Streptococcus and Enterococcus, and poorer average body condition. (2) Conversely, rural coyotes harbored microbiomes rich in Fusobacteria, Sutterella, and Anaerobiospirillum, which were associated with protein-rich diets and improved body condition. (3) Diets rich in anthropogenic food were associated with increased abundances of Erysipelotrichiaceae, Lachnospiraceae, and Coriobacteriaceae, which correlated with larger spleens in urban coyotes. Urban coyotes also had an increased prevalence of the zoonotic parasite Echinococcus multilocularis, but there were no detectable connections between parasite infection and microbiome composition. Our results demonstrate how the consumption of carbohydrate-rich anthropogenic food by urban coyotes alters the microbiome to negatively affect body condition, with potential relationships to parasite susceptibility and conflict-prone behavior.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-79363-2" target="_blank" style="color:black; text-decoration:none">T-cell receptor repertoire of cytomegalovirus-specific cytotoxic T-cells after allogeneic stem cell transplantation</a></h1>
            <p>Takashi Toya, Ayumi Taguchi, Kazutaka Kitaura, Fumi Misumi, Yujiro Nakajima, Yuki Otsuka, Ryosuke Konuma, Hiroto Adachi, Atsushi Wada, Yuya Kishida, Tatsuya Konishi, Akihito Nagata, Yuta Yamada, Atsushi Marumo, Yuma Noguchi, Kota Yoshifuji, Junichi Mukae, Kyoko Inamoto, Aiko Igarashi, Yuho Najima, Takeshi Kobayashi, Kazuhiko Kakihana, Kazuteru Ohashi, Ryuji Suzuki, Takeshi Nagamatsu, Noriko Doki</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-17</pre>
            <h2>Abstract</h2>
            <font size="4">Cytomegalovirus (CMV) infection is a major complication during allogeneic stem cell transplantation (allo-SCT). However, mechanisms of adaptive immunity that drive this remain unclear. To define early immunological responses to CMV after transplantation, we using next-generation sequencing to examine the repertoire of T-cell receptors in CD8+/CMV pp65 tetramer+ cells (CMV-CTLs) in peripheral blood samples obtained from 16 allo-SCT recipients with HLA-A*24:02 at the time of CMV reactivation. In most patients, TCR beta repertoire of CMV-CTLs was highly skewed (median Inverse Simpson’s index: 1.595) and, 15 of 16 patients shared at least one TCR-beta clonotype with ≥ 2 patients. The shared TCRs were dominant in 12 patients and, two clonotypes were shared by about half of the patients. Similarity analysis showed that CDR3 sequences of shared TCRs were more similar than unshared TCRs. TCR beta repertoires of CMV-CTLs in 12 patients were also analyzed after 2–4 weeks to characterize the short-term dynamics of TCR repertoires. In ten patients, we observed persistence of prevailing clones. In the other two patients, TCR repertoires became more diverse, major clones declined, and new private clones subsequently emerged. These results provided the substantive clue to understand the immunological behavior against CMV reactivation after allo-SCT.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41559-020-01364-1" target="_blank" style="color:black; text-decoration:none">Host diversity slows bacteriophage adaptation by selecting generalists over specialists</a></h1>
            <p>Duhita G. Sant, Laura C. Woods, Jeremy J. Barr, Michael J. McDonald</p>
            <pre style="font-family:timesnewroman">Journal: Nature Ecology & Evolution    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">Most viruses can infect multiple hosts, yet the selective mechanisms that maintain multi-host generalists over single-host specialists remain an open question. Here we propagate populations of the newly identified bacteriophage øJB01 in coculture with many host genotypes and find that while phage can adapt to infect any of the new hosts, increasing the number of hosts slows the rate of adaptation. We quantify trade-offs in the capacity for individual phage to infect different hosts and find that phage from evolved populations with more hosts are more likely to be generalists. Sequencing of evolved phage reveals strong selection and the genetic basis of adaptation, supporting a model that shows how the addition of more potential hosts to a community can select for low-fitness generalists over high-fitness specialists. Our results show how evolution with multiple hosts alters the rate of viral adaptation and provides empirical support for an evolutionary mechanism that promotes generalists over specialists. Propagating bacteriophage in cocultures of multiple host strains, the authors show that increasing host strain diversity decreases the rate of adaptation and selects for lower fitness generalists over higher fitness specialists.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78090-y" target="_blank" style="color:black; text-decoration:none">Linear infrastructure habitats increase landscape-scale diversity of plants but not of flower-visiting insects</a></h1>
            <p>Juliana Dániel-Ferreira, Riccardo Bommarco, Jörgen Wissman, Erik Öckinger</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-07</pre>
            <h2>Abstract</h2>
            <font size="4">Habitats along linear infrastructure, such as roads and electrical transmission lines, can have high local biodiversity. To determine whether these habitats also contribute to landscape-scale biodiversity, we estimated species richness, evenness and phylogenetic diversity of plant, butterfly and bumblebee communities in 32 km2 landscapes with or without power line corridors, and with contrasting areas of road verges. Landscapes with power line corridors had on average six more plant species than landscapes without power lines, but there was no such effect for butterflies and bumblebees. Plant communities displayed considerable evenness in species abundances both in landscapes with and without power lines and high and low road verge densities. We hypothesize that the higher number of plant species in landscapes with power line corridors is due to these landscapes having a higher extinction debt than the landscapes without power line corridors, such that plant diversity is declining slower in landscapes with power lines. This calls for targeted conservation actions in semi-natural grasslands within landscapes with power line corridors to maintain biodiversity and prevent imminent population extinctions.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41437-020-00387-5" target="_blank" style="color:black; text-decoration:none">Weak founder effects but significant spatial genetic imprint of recent contraction and expansion of European beech populations</a></h1>
            <p>Tonya A. Lander, Etienne K. Klein, Anne Roig, Sylvie Oddou-Muratorio</p>
            <pre style="font-family:timesnewroman">Journal: Heredity    Published: 2020-11-23</pre>
            <h2>Abstract</h2>
            <font size="4">Understanding the ecological and evolutionary processes occurring during species range shifts is important in the current context of global change. Here, we investigate the interplay between recent expansion, gene flow and genetic drift, and their consequences for genetic diversity and structure at landscape and local scales in European beech (Fagus sylvatica L.) On Mont Ventoux, South-Eastern France, we located beech forest refugia at the time of the most recent population minimum, ~150 years ago, and sampled 71 populations (2042 trees) in both refugia and expanding populations over an area of 15,000 ha. We inferred patterns of gene flow and genetic structure using 12 microsatellite markers. We identified six plots as originating from planting, rather than natural establishment, mostly from local genetic material. Comparing genetic diversity and structure in refugia versus recent populations did not support the existence of founder effects: heterozygosity (He = 0.667) and allelic richness (Ar = 4.298) were similar, and FST was low (0.031 overall). Still, significant spatial evidence of colonization was detected, with He increasing along the expansion front, while genetic differentiation from the entire pool (βWT) decreased. Isolation by distance was found in refugia but not in recently expanding populations. Our study indicates that beech capacities for colonization and gene flow were sufficient to preserve genetic diversity despite recent forest contraction and expansion. Because beech has long distance pollen and seed dispersal, these results illustrate a ‘best case scenario’ for the maintenance of high genetic diversity and adaptive potential under climate-change-related range change.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77382-7" target="_blank" style="color:black; text-decoration:none">Warming and leaf litter functional diversity, not litter quality, drive decomposition in a freshwater ecosystem</a></h1>
            <p>Gustavo H. Migliorini, Gustavo Q. Romero</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-23</pre>
            <h2>Abstract</h2>
            <font size="4">Environment, litter composition and decomposer community are known to be the main drivers of litter decomposition in aquatic ecosystems. However, it remains unclear whether litter quality or functional diversity prevails under warming conditions. Using tank bromeliad ecosystems, we evaluated the combined effects of warming, litter quality and litter functional diversity on the decomposition process. We also assessed the contribution of macroinvertebrates and microorganisms in explaining litter decomposition patterns using litter bags made with different mesh sizes. Our results showed that litter decomposition was driven by litter functional diversity and was increasingly higher under warming, in both mesh sizes. Decomposition was explained by increasing litter dissimilarities in C and N. Our results highlight the importance of considering different aspects of litter characteristics (e.g., quality and functional diversity) in order to predict the decomposition process in freshwater ecosystems. Considering the joint effect of warming and litter traits aspects allow a more refined understanding of the underlying mechanisms of climate change and biodiversity shifts effects on ecosystem functioning.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80696-1" target="_blank" style="color:black; text-decoration:none">Genetic structure analysis of cultivated and wild chestnut populations reveals gene flow from cultivars to natural stands</a></h1>
            <p>Sogo Nishio, Norio Takada, Shingo Terakami, Yukie Takeuchi, Megumi K. Kimura, Keiya Isoda, Toshihiro Saito, Hiroyuki Iketani</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Japanese chestnut (Castanea crenata Sieb. et Zucc.), the only fruit tree species domesticated in Japan, has been cultivated alongside natural stands since prehistorical times. Understanding the genetic diversity of this species and the relationships between cultivated and wild chestnut is important for clarifying its breeding history and determining conservation strategies. We assessed 3 chestnut cultivar populations and 29 wild chestnut populations (618 accessions). Genetic distance analysis revealed that wild populations in the Kyushu region are genetically distant from other populations, whereas other wild and cultivar populations are comparatively similar. Assignment tests suggested that cultivars were relatively similar to populations from central to western Honshu. Bayesian structure analyses showed that wild individuals were roughly classified according to geographical distribution along the Japanese archipelago, except that some wild individuals carried the genetic cluster prevalent in cultivars. Parentage analyses between cultivars and wild individuals identified 26 wild individuals presumed to have a parent–offspring relationship with a cultivar. These results suggested that the genetic structure of some wild individuals in natural stands was influenced by gene flow from cultivars. To conserve wild individuals carrying true “wild” genetic clusters, these individuals should be collected and preserved by ex situ conservation programs.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80631-4" target="_blank" style="color:black; text-decoration:none">Response of bacterial and fungal communities to high petroleum pollution in different soils</a></h1>
            <p>Polina Galitskaya, Liliya Biktasheva, Sergey Blagodatsky, Svetlana Selivanovskaya</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Petroleum pollution of soils is a major environmental problem. Soil microorganisms can decompose a significant fraction of petroleum hydrocarbons in soil at low concentrations (1–5%). This characteristic can be used for soil remediation after oil pollution. Microbial community dynamics and functions are well studied in cases of moderate petroleum pollution, while cases with heavy soil pollution have received much less attention. We studied bacterial and fungal successions in three different soils with high petroleum contents (6 and 25%) in a laboratory experiment. The proportion of aliphatic and aromatic compounds decreased by 4–7% in samples with 6% pollution after 120 days of incubation but remained unchanged in samples with 25% hydrocarbons. The composition of the microbial community changed significantly in all cases. Oil pollution led to an increase in the relative abundance of bacteria such as Actinobacteria and the candidate TM7 phylum (Saccaribacteria) and to a decrease in that of Bacteroidetes. The gene abundance (number of OTUs) of oil-degrading bacteria (Rhodococcus sp., candidate class TM7-3 representative) became dominant in all soil samples, irrespective of the petroleum pollution level and soil type. The fungal communities in unpolluted soil samples differed more significantly than the bacterial communities. Nonmetric multidimensional scaling revealed that in the polluted soil, successions of fungal communities differed between soils, in contrast to bacterial communities. However, these successions showed similar trends: fungi capable of lignin and cellulose decomposition, e.g., from the genera Fusarium and Mortierella, were dominant during the incubation period.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80636-z" target="_blank" style="color:black; text-decoration:none">Shift in polar benthic community structure in a fast retreating glacial area of Marian Cove, West Antarctica</a></h1>
            <p>Hanna Bae, In-Young Ahn, Jinsoon Park, Sung Joon Song, Junsung Noh, Hosang Kim, Jong Seong Khim</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Glacier retreat is a major long-standing global issue; however, the ecological impacts of such retreats on marine organisms remain unanswered. Here, we examined changes to the polar benthic community structure of “diatoms” under current global warming in a recently retreated glacial area of Marian Cove, Antarctica. The environments and spatiotemporal assemblages of benthic diatoms surveyed in 2018–2019 significantly varied between the intertidal (tidal height of 2.5 m) and subtidal zone (10 and 30 m). A distinct floral distribution along the cove (~ 4.5 km) was characterized by the adaptive strategy of species present, with chain-forming species predominating near the glacier. The predominant chain-forming diatoms, such as Fragilaria striatula and Paralia sp., are widely distributed in the innermost cove over years, indicating sensitive responses of benthic species to the fast-evolving polar environment. The site-specific and substrate-dependent distributions of certain indicator species (e.g., F. striatula, Navicula glaciei, Cocconeis cf. pinnata) generally reflected such shifts in the benthic community. Our review revealed that the inner glacier region reflected trophic association, featured with higher diversity, abundance, and biomass of benthic diatoms and macrofauna. Overall, the polar benthic community shift observed along the cove generally represented changing environmental conditions, (in)directly linked to ice-melting due to the recent glacier retreat.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00856-z" target="_blank" style="color:black; text-decoration:none">Morphological complexity affects the diversity of marine microbiomes</a></h1>
            <p>Matthew A. Lemay, Melissa Y. Chen, Florent Mazel, Katharine R. Hind, Samuel Starko, Patrick J. Keeling, Patrick T. Martone, Laura Wegener Parfrey</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2020-12-21</pre>
            <h2>Abstract</h2>
            <font size="4">Large eukaryotes support diverse communities of microbes on their surface—epibiota—that profoundly influence their biology. Alternate factors known to structure complex patterns of microbial diversity—host evolutionary history and ecology, environmental conditions and stochasticity—do not act independently and it is challenging to disentangle their relative effects. Here, we surveyed the epibiota from 38 sympatric seaweed species that span diverse clades and have convergent morphology, which strongly influences seaweed ecology. Host identity explains most of the variation in epibiont communities and deeper host phylogenetic relationships (e.g., genus level) explain a small but significant portion of epibiont community variation. Strikingly, epibiota community composition is significantly influenced by host morphology and epibiota richness increases with morphological complexity of the seaweed host. This effect is robust after controlling for phylogenetic non-independence and is strongest for crustose seaweeds. We experimentally validated the effect of host morphology by quantifying bacterial community assembly on latex sheets cut to resemble three seaweed morphologies. The patterns match those observed in our field survey. Thus, biodiversity increases with habitat complexity in host-associated microbial communities, mirroring patterns observed in animal communities. We suggest that host morphology and structural complexity are underexplored mechanisms structuring microbial communities.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41586-020-3003-4" target="_blank" style="color:black; text-decoration:none">Impacts of speciation and extinction measured by an evolutionary decay clock</a></h1>
            <p>Jennifer F. Hoyal Cuthill, Nicholas Guttenberg, Graham E. Budd</p>
            <pre style="font-family:timesnewroman">Journal: Nature    Published: 2020-12-09</pre>
            <h2>Abstract</h2>
            <font size="4">The hypothesis that destructive mass extinctions enable creative evolutionary radiations (creative destruction) is central to classic concepts of macroevolution1,2. However, the relative impacts of extinction and radiation on the co-occurrence of species have not been directly quantitatively compared across the Phanerozoic eon. Here we apply machine learning to generate a spatial embedding (multidimensional ordination) of the temporal co-occurrence structure of the Phanerozoic fossil record, covering 1,273,254 occurrences in the Paleobiology Database for 171,231 embedded species. This facilitates the simultaneous comparison of macroevolutionary disruptions, using measures independent of secular diversity trends. Among the 5% most significant periods of disruption, we identify the ‘big five’ mass extinction events2, seven additional mass extinctions, two combined mass extinction–radiation events and 15 mass radiations. In contrast to narratives that emphasize post-extinction radiations1,3, we find that the proportionally most comparable mass radiations and extinctions (such as the Cambrian explosion and the end-Permian mass extinction) are typically decoupled in time, refuting any direct causal relationship between them. Moreover, in addition to extinctions4, evolutionary radiations themselves cause evolutionary decay (modelled co-occurrence probability and shared fraction of species between times approaching zero), a concept that we describe as destructive creation. A direct test of the time to over-threshold macroevolutionary decay4 (shared fraction of species between two times ≤ 0.1), counted by the decay clock, reveals saw-toothed fluctuations around a Phanerozoic mean of 18.6 million years. As the Quaternary period began at a below-average decay-clock time of 11 million years, modern extinctions further increase life’s decay-clock debt. Analysis of data on species co-occurrence in the Paleobiology Database using a new machine learning algorithm reveals that mass extinctions and mass radiations are not coupled in evolutionary history.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77256-y" target="_blank" style="color:black; text-decoration:none">Diet induces parallel changes to the gut microbiota and problem solving performance in a wild bird</a></h1>
            <p>Gabrielle L. Davidson, Niamh Wiley, Amy C. Cooke, Crystal N. Johnson, Fiona Fouhy, Michael S. Reichert, Iván de la Hera, Jodie M. S. Crane, Ipek G. Kulahci, R. Paul Ross, Catherine Stanton, John L. Quinn</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-27</pre>
            <h2>Abstract</h2>
            <font size="4">The microbial community in the gut is influenced by environmental factors, especially diet, which can moderate host behaviour through the microbiome-gut-brain axis. However, the ecological relevance of microbiome-mediated behavioural plasticity in wild animals is unknown. We presented wild-caught great tits (Parus major) with a problem-solving task and showed that performance was weakly associated with variation in the gut microbiome. We then manipulated the gut microbiome by feeding birds one of two diets that differed in their relative levels of fat, protein and fibre content: an insect diet (low content), or a seed diet (high content). Microbial communities were less diverse among individuals given the insect compared to those on the seed diet. Individuals were less likely to problem-solve after being given the insect diet, and the same microbiota metrics that were altered as a consequence of diet were also those that correlated with variation in problem solving performance. Although the effect on problem-solving behaviour could have been caused by motivational or nutritional differences between our treatments, our results nevertheless raise the possibility that dietary induced changes in the gut microbiota could be an important mechanism underlying individual behavioural plasticity in wild populations.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41576-020-00305-9" target="_blank" style="color:black; text-decoration:none">The influence of evolutionary history on human health and disease</a></h1>
            <p>Mary Lauren Benton, Abin Abraham, Abigail L. LaBella, Patrick Abbot, Antonis Rokas, John A. Capra</p>
            <pre style="font-family:timesnewroman">Journal: Nature Reviews Genetics    Published: 2021-01-06</pre>
            <h2>Abstract</h2>
            <font size="4">Nearly all genetic variants that influence disease risk have human-specific origins; however, the systems they influence have ancient roots that often trace back to evolutionary events long before the origin of humans. Here, we review how advances in our understanding of the genetic architectures of diseases, recent human evolution and deep evolutionary history can help explain how and why humans in modern environments become ill. Human populations exhibit differences in the prevalence of many common and rare genetic diseases. These differences are largely the result of the diverse environmental, cultural, demographic and genetic histories of modern human populations. Synthesizing our growing knowledge of evolutionary history with genetic medicine, while accounting for environmental and social factors, will help to achieve the promise of personalized genomics and realize the potential hidden in an individual’s DNA sequence to guide clinical decisions. In short, precision medicine is fundamentally evolutionary medicine, and integration of evolutionary perspectives into the clinic will support the realization of its full potential. Our evolutionary history has resulted in highly complex and sophisticated human physiology. Yet evolutionary footprints have also left us prone to diseases. In this Review, the authors discuss how events from the earliest history of life on Earth through to modern human evolution influence many disease traits and outcomes. They describe how an understanding and application of evolutionary frameworks can inform precision medicine initiatives.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78413-z" target="_blank" style="color:black; text-decoration:none">Metagenomic analysis reveals rapid development of soil biota on fresh volcanic ash</a></h1>
            <p>Hokyung Song, Dorsaf Kerfahi, Koichi Takahashi, Sophie L. Nixon, Binu M. Tripathi, Hyoki Kim, Ryunosuke Tateno, Jonathan Adams</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-08</pre>
            <h2>Abstract</h2>
            <font size="4">Little is known of the earliest stages of soil biota development of volcanic ash, and how rapidly it can proceed. We investigated the potential for soil biota development during the first 3 years, using outdoor mesocosms of sterile, freshly fallen volcanic ash from the Sakurajima volcano, Japan. Mesocosms were positioned in a range of climates across Japan and compared over 3 years, against the developed soils of surrounding natural ecosystems. DNA was extracted from mesocosms and community composition assessed using 16S rRNA gene sequences. Metagenome sequences were obtained using shotgun metagenome sequencing. While at 12 months there was insufficient DNA for sequencing, by 24 months and 36 months, the ash-soil metagenomes already showed a similar diversity of functional genes to the developed soils, with a similar range of functions. In a surprising contrast with our hypotheses, we found that the developing ash-soil community already showed a similar gene function diversity, phylum diversity and overall relative abundances of kingdoms of life when compared to developed forest soils. The ash mesocosms also did not show any increased relative abundance of genes associated with autotrophy (rbc, coxL), nor increased relative abundance of genes that are associated with acquisition of nutrients from abiotic sources (nifH). Although gene identities and taxonomic affinities in the developing ash-soils are to some extent distinct from the natural vegetation soils, it is surprising that so many of the key components of a soil community develop already by the 24-month stage. In this system, however, rapid development may be facilitated by the relatively moderate pH of the Sakurajima ash, proximity of our mesocosms to propagule sources, and the rapid establishment of a productive bryophyte and lichen layer on the surface. Ash from other volcanoes richer in acids or more distant from propagule sources could show a different pattern and slower soil biota development.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41591-020-1125-8" target="_blank" style="color:black; text-decoration:none">Single-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma</a></h1>
            <p>Ruiping Wang, Minghao Dang, Kazuto Harada, Guangchun Han, Fang Wang, Melissa Pool Pizzi, Meina Zhao, Ghia Tatlonghari, Shaojun Zhang, Dapeng Hao, Yang Lu, Shuangtao Zhao, Brian D. Badgwell, Mariela Blum Murphy, Namita Shanbhag, Jeannelyn S. Estrella, Sinchita Roy-Chowdhuri, Ahmed Adel Fouad Abdelhakeem, Yuanxin Wang, Guang Peng, Samir Hanash, George A. Calin, Xingzhi Song, Yanshuo Chu, Jianhua Zhang, Mingyao Li, Ken Chen, Alexander J. Lazar, Andrew Futreal, Shumei Song, Jaffer A. Ajani, Linghua Wang</p>
            <pre style="font-family:timesnewroman">Journal: Nature Medicine    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Intratumoral heterogeneity (ITH) is a fundamental property of cancer; however, the origins of ITH remain poorly understood. We performed single-cell transcriptome profiling of peritoneal carcinomatosis (PC) from 15 patients with gastric adenocarcinoma (GAC), constructed a map of 45,048 PC cells, profiled the transcriptome states of tumor cell populations, incisively explored ITH of malignant PC cells and identified significant correlates with patient survival. The links between tumor cell lineage/state compositions and ITH were illustrated at transcriptomic, genotypic, molecular and phenotypic levels. We uncovered the diversity in tumor cell lineage/state compositions in PC specimens and defined it as a key contributor to ITH. Single-cell analysis of ITH classified PC specimens into two subtypes that were prognostically independent of clinical variables, and a 12-gene prognostic signature was derived and validated in multiple large-scale GAC cohorts. The prognostic signature appears fundamental to GAC carcinogenesis and progression and could be practical for patient stratification. Single-cell analysis of gastric cancer samples tracks the cell of origin of metastatic lesions and identifies an independent prognostic signature of the clinical outcome.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-20337-3" target="_blank" style="color:black; text-decoration:none">The patterns of deleterious mutations during the domestication of soybean</a></h1>
            <p>Myung-Shin Kim, Roberto Lozano, Ji Hong Kim, Dong Nyuk Bae, Sang-Tae Kim, Jung-Ho Park, Man Soo Choi, Jaehyun Kim, Hyun-Choong Ok, Soo-Kwon Park, Michael A. Gore, Jung-Kyung Moon, Soon-Chun Jeong</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Globally, soybean is a major protein and oil crop. Enhancing our understanding of the soybean domestication and improvement process helps boost genomics-assisted breeding efforts. Here we present a genome-wide variation map of 10.6 million single-nucleotide polymorphisms and 1.4 million indels for 781 soybean individuals which includes 418 domesticated (Glycine max), 345 wild (Glycine soja), and 18 natural hybrid (G. max/G. soja) accessions. We describe the enhanced detection of 183 domestication-selective sweeps and the patterns of putative deleterious mutations during domestication and improvement. This predominantly selfing species shows 7.1% reduction of overall deleterious mutations in domesticated soybean relative to wild soybean and a further 1.4% reduction from landrace to improved accessions. The detected domestication-selective sweeps also show reduced levels of deleterious alleles. Importantly, genotype imputation with this resource increases the mapping resolution of genome-wide association studies for seed protein and oil traits in a soybean diversity panel. The accumulation of recombination events in selfing species may lead to a rapid fixation of both beneficial and deleterious mutations. Here, the authors resequence 781 soybean accessions, show purging of deleterious mutation during domestication, and report genome-wide associations for seed protein and oil traits.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77480-6" target="_blank" style="color:black; text-decoration:none">Microsatellite and mitochondrial DNA analyses unveil the genetic structure of native sheep breeds from three major agro-ecological regions of India</a></h1>
            <p>Rekha Sharma, Sonika Ahlawat, Himani Sharma, Priyanka Sharma, Poonam Panchal, Reena Arora, M S Tantia</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-24</pre>
            <h2>Abstract</h2>
            <font size="4">Sheep farming has been fundamental to many civilizations in the world and is practiced in India since antiquity. Several thousand years of adaptation to local environmental conditions and selective breeding have evolved 44 sheep breeds in India. They are paramount in terms of economic, scientific, and cultural heritage. Genetic characterization information is imperative for sustainable utilization and conservation of ovine heritage. In this study, the genetic diversity, differentiation, and structure of 11 indigenous sheep breeds from three different agro-ecological zones of India were explored with genomic microsatellite loci and mitochondrial DNA (D loop). The estimated diversity parameters indicated that populations retained high levels of genetic diversity (Na = 8.27 ± 0.17; Ho = 0.65 ± 0.01), which provides an optimistic viewpoint for their survival. However, significant inbreeding was also observed in the nine populations. Moderate genetic differentiation existed among the groups (FST = 0.129 ± 0.012), and most likely clusters existing in the dataset are seven. Phylogenetic clustering was in line with the geographical locations of sheep populations. Mitochondrial sequences revealed high haplotype diversity with the existence of maternal haplogroups A, B, and C, and signals of population expansion. Decreased genetic diversity and unique maternal lineage (C) in endangered Tibetan and Bonpala sheep breed, warrant their immediate scientific management. Overall, the quantitative data reported here on the extant variability, and genetic relationships among the Indian sheep breeds, provide critically important inputs that will be valuable for the decision-making process on their management, both for the conservation of endangered breeds, and formulation of breeding programs to check genetic erosion.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77313-6" target="_blank" style="color:black; text-decoration:none">Feed additives for the control of post-weaning Streptococcus suis disease and the effect on the faecal and nasal microbiota</a></h1>
            <p>Florencia Correa-Fiz, Carlos Neila-Ibáñez, Sergio López-Soria, Sebastian Napp, Blanca Martinez, Laia Sobrevia, Simon Tibble, Virginia Aragon, Lourdes Migura-Garcia</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-23</pre>
            <h2>Abstract</h2>
            <font size="4">Medicated feed is a common strategy to control the occurrence of Streptococcus suis disease in swine production, but feed additives may constitute an alternative to metaphylaxis. In a farm with post-weaning S. suis disease, the following additives were tested: lysozyme (Lys), medium chain fatty acids plus lysozyme (FA + Lys), FA plus a natural anti-inflammatory (FA + antiinf) and amoxicillin (Amox). During the course of the study, FA + antiinf and Amox groups showed lower prevalence of clinical signs compatible with S. suis disease than the rest of the groups. Piglets from the FA + antiinf group showed high diversity and richness in their nasal and faecal microbiota. Diet supplements did not have major effects on the faecal microbiota, where the genus Mitsuokella was the only differentially present in the FA + Lys group. In the nasal microbiota, piglets from FA + antiinf presented higher differential abundance of a sequence variant from Ruminococcaceae and lower abundance of an unclassified genus from Weeksellaceae. In general, we detected more significant changes in the nasal than in the feacal microbiota, and found that parity of the dams affected the microbiota composition of their offspring, with piglets born to gilts exhibiting lower richness and diversity. Our results suggest that additives could be useful to control post-weaning disease when removing antimicrobials in farms.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41564-020-00811-w" target="_blank" style="color:black; text-decoration:none">Trace gas oxidizers are widespread and active members of soil microbial communities</a></h1>
            <p>Sean K. Bay, Xiyang Dong, James A. Bradley, Pok Man Leung, Rhys Grinter, Thanavit Jirapanjawat, Stefan K. Arndt, Perran L. M. Cook, Douglas E. LaRowe, Philipp A. Nauer, Eleonora Chiri, Chris Greening</p>
            <pre style="font-family:timesnewroman">Journal: Nature Microbiology    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Soil microorganisms globally are thought to be sustained primarily by organic carbon sources. Certain bacteria also consume inorganic energy sources such as trace gases, but they are presumed to be rare community members, except within some oligotrophic soils. Here we combined metagenomic, biogeochemical and modelling approaches to determine how soil microbial communities meet energy and carbon needs. Analysis of 40 metagenomes and 757 derived genomes indicated that over 70% of soil bacterial taxa encode enzymes to consume inorganic energy sources. Bacteria from 19 phyla encoded enzymes to use the trace gases hydrogen and carbon monoxide as supplemental electron donors for aerobic respiration. In addition, we identified a fourth phylum (Gemmatimonadota) potentially capable of aerobic methanotrophy. Consistent with the metagenomic profiling, communities within soil profiles from diverse habitats rapidly oxidized hydrogen, carbon monoxide and to a lesser extent methane below atmospheric concentrations. Thermodynamic modelling indicated that the power generated by oxidation of these three gases is sufficient to meet the maintenance needs of the bacterial cells capable of consuming them. Diverse bacteria also encode enzymes to use trace gases as electron donors to support carbon fixation. Altogether, these findings indicate that trace gas oxidation confers a major selective advantage in soil ecosystems, where availability of preferred organic substrates limits microbial growth. The observation that inorganic energy sources may sustain most soil bacteria also has broad implications for understanding atmospheric chemistry and microbial biodiversity in a changing world. A combination of metagenomic analyses, thermodynamic modelling and in situ measurements of gas fluxes shows that a large fraction of soil bacteria can use inorganic energy sources, such as the trace gases hydrogen and carbon monoxide, for growth and persistence.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41522-020-00171-7" target="_blank" style="color:black; text-decoration:none">The human respiratory tract microbial community structures in healthy and cystic fibrosis infants</a></h1>
            <p>Marie-Madlen Pust, Lutz Wiehlmann, Colin Davenport, Isa Rudolf, Anna-Maria Dittrich, Burkhard Tümmler</p>
            <pre style="font-family:timesnewroman">Journal: npj Biofilms and Microbiomes    Published: 2020-12-15</pre>
            <h2>Abstract</h2>
            <font size="4">The metagenome development of the human respiratory tract was investigated by shotgun metagenome metagenomic sequencing of cough swabs from healthy children and children with cystic fibrosis (CF) between 3 weeks and 6 years of age. A healthy microbial community signature was associated with increased absolute abundances in terms of bacterial–human cell ratios of core and rare species across all age groups, with a higher diversity of rare species and a tightly interconnected species co-occurrence network, in which individual members were found in close proximity to each other and negative correlations were absent. Even without typical CF pathogens, the CF infant co-occurrence network was found to be less stable and prone to fragmentation due to fewer connections between species, a higher number of bridging species and the presence of negative species correlations. Detection of low-abundant DNA of the CF hallmark pathogen Pseudomonas aeruginosa was neither disease- nor age-associated in our cohort. Healthy and CF children come into contact with P. aeruginosa on a regular basis and from early on.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41893-020-00637-8" target="_blank" style="color:black; text-decoration:none">Impact of local and landscape complexity on the stability of field-level pest control</a></h1>
            <p>Ashley E. Larsen, Frederik Noack</p>
            <pre style="font-family:timesnewroman">Journal: Nature Sustainability    Published: 2020-11-09</pre>
            <h2>Abstract</h2>
            <font size="4">Agricultural production has increased dramatically in the past 50 years, supported, in part, by the simplification of agricultural landscapes. While the benefits of increased food production are difficult to dispute, simplification, at both the local and landscape level, has fuelled declines in biodiversity and ecosystem services. In addition to the concerns that this loss of complexity necessitates higher levels of pesticide use in general, local and landscape simplification may also increase pest outbreaks and, consequently, infrequent but particularly high pesticide use with potentially damaging consequences for the environment and human health. We find that increasing cropland in the landscape—and larger fields generally—increase the level and variability of pesticide use while crop diversity has the opposite effect, as predicted by ecological theory. In all cases, accounting for non-random planting decisions and farmer-specific behaviour strongly influences the magnitude of the estimated statistical relationships. This suggests that, while complexity increases stability and reduces high deviations in insecticide use, accounting for crop and farmer-specific characteristics is crucial for statistical inference and sound scientific understanding. Agricultural landscapes have become more simplified as crop production has risen. This study finds that pesticide use rises with crop area and field size but falls with crop diversity.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-79376-x" target="_blank" style="color:black; text-decoration:none">Reconstruction of forest dynamics in the Western Palaearctic based on phylogeographic analysis of the ringlet butterfly Erebia aethiops</a></h1>
            <p>Martin Wendt, Martin Husemann, Katja Kramp, Thomas Schmitt</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Glacial refugia are centers of high biodiversity. Therefore, knowledge on their locations and reactions of associated populations and landscapes to climatic changes is crucial for conservation management. We here investigated the biogeography of a butterfly species linked to open forest habitats. Using mitochondrial and nuclear markers in combination with Bayesian simulations, we analyzed the location and age of potential glacial refugia of the species. We identified five putative refugia in Europe. Considering the ecological needs of our study species, tree density within these refugial areas, in contrast to earlier assumptions, must have exceeded the level of individually scattered trees. Our results also provide evidence that especially the refuge areas in the Carpathians were previously underestimated regarding their age: the refugia in the Southern Carpathians presented suitable conditions throughout several glacial cycles, probably since the Mindel or Riss cycles. Additionally, our analyses provided support for a forest refugium near the Tatra Mountains persisting the last glacial maximum. Our results underline the usefulness of this and probably other butterfly species as indicators of forest refugia.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80642-1" target="_blank" style="color:black; text-decoration:none">Particulate matter emission sources and meteorological parameters combine to shape the airborne bacteria communities in the Ligurian coast, Italy</a></h1>
            <p>Giorgia Palladino, Pietro Morozzi, Elena Biagi, Erika Brattich, Silvia Turroni, Simone Rampelli, Laura Tositti, Marco Candela</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Aim of the present study is to explore how the chemical composition of particulate matter (PM) and meteorological conditions combine in shaping the air microbiome in Savona (Italy), a medium-size, heavily inhabited urban settlement, hosting a wide range of industrial activities. In particular, the air microbiome and PM10 were monitored over six months in 2012. During that time, the air microbiome was highly dynamic, fluctuating between different compositional states, likely resulting from the aerosolization of different microbiomes emission sources. According to our findings, this dynamic process depends on the combination of local meteorological parameters and particle emission sources, which may affect the prevalent aerosolized microbiomes, thus representing further fundamental tools for source apportionment in a holistic approach encompassing chemical as well as microbiological pollution. In particular, we showed that, in the investigated area, industrial emissions and winds blowing from the inlands combine with an airborne microbiome which include faecal microbiomes components, suggesting multiple citizens’ exposure to both chemicals and microorganisms of faecal origin, as related to landscape exploitation and population density. In conclusion, our findings support the need to include monitoring of the air microbiome compositional structure as a relevant factor for the final assessment of local air quality.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80586-6" target="_blank" style="color:black; text-decoration:none">Geological and Pleistocene glaciations explain the demography and disjunct distribution of red panda ( A. fulgens ) in eastern Himalayas</a></h1>
            <p>Supriyo Dalui, Sujeet Kumar Singh, Bheem Dutt Joshi, Avijit Ghosh, Shambadeb Basu, Hiren Khatri, Lalit Kumar Sharma, Kailash Chandra, Mukesh Thakur</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">Pleistocene glaciations facilitated climatic oscillations that caused for enormous heterogeneity in landscapes, and consequently affected demography and distribution patterns of the mountain endemic species. In this context, we investigated demographic history and population genetic structure of red panda, distributed along the geographical proximity in the southern edge of the Qinghai-Tibetan Plateau. Bayesian based phylogeny demonstrated that red panda diverged about 0.30 million years ago (CI 0.23–0.39) into two phylogenetic (sub) species, that correspond to the middle-late Pleistocene transition. The observed intraspecific clades with respect to Himalayan and Chinese red panda indicated restricted gene flow resulting from the Pleistocene glaciations in the eastern and southern Tibetan Plateau. We found Himalayan red panda population at least in KL-India declined abruptly in last 5–10 thousand years after being under demographic equilibrium. We suggest revisiting the ongoing conservation activities through cross border collaboration by developing multi-nationals, and multi-lateral species-oriented conservation action plans to support the red panda populations in transboundary landscapes.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-20422-7" target="_blank" style="color:black; text-decoration:none">Gut microbiota impact on the peripheral immune response in non-alcoholic fatty liver disease related hepatocellular carcinoma</a></h1>
            <p>Jason Behary, Nadia Amorim, Xiao-Tao Jiang, Anita Raposo, Lan Gong, Emily McGovern, Ragy Ibrahim, Francis Chu, Carlie Stephens, Hazem Jebeili, Vincenzo Fragomeli, Yen Chin Koay, Miriam Jackson, John O’Sullivan, Martin Weltman, Geoffrey McCaughan, Emad El-Omar, Amany Zekry</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">The gut microbiota is reported to modulate the immune response in hepatocellular carcinoma (HCC). Here, we employ metagenomic and metabolomic studies to characterise gut microbiota in patients with non-alcoholic fatty liver disease (NAFLD) related cirrhosis, with or without HCC, and evaluate its effect on the peripheral immune response in an ex vivo model. We find that dysbiosis characterises the microbiota of patients with NAFLD-cirrhosis, with compositional and functional shifts occurring with HCC development. Gene function of the microbiota in NAFLD-HCC supports short chain fatty acid production, and this is confirmed by metabolomic studies. Ex vivo studies show that bacterial extracts from the NAFLD-HCC microbiota, but not from the control groups, elicit a T cell immunosuppressive phenotype, characterised by expansion of regulatory T cells and attenuation of CD8 + T cells. Our study suggest that the gut microbiota in NAFLD-HCC is characterised by a distinctive microbiome/metabolomic profile, and can modulate the peripheral immune response. Disease-specific gut microbiome signatures have been previously defined for patients with liver cirrhosis or hepatocellular carcinoma (HCC). Here the authors examine the composition of the gut microbiota in cirrhotic patients with non-alcoholic fatty liver disease with or without HCC and evaluate how dysbiosis influences peripheral immune responses.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77878-2" target="_blank" style="color:black; text-decoration:none">Recent land use and management changes decouple the adaptation of livestock diversity to the environment</a></h1>
            <p>Elena Velado-Alonso, Ignacio Morales-Castilla, Antonio Gómez-Sal</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-03</pre>
            <h2>Abstract</h2>
            <font size="4">Native livestock breeds, i.e. those autochthonous to a specific region, are locally adapted domesticated animals that conserve genetic resources, guaranty food security and provide agroecosystem services. Native breeds are largely threatened worldwide by agricultural intensification and rural areas abandonment processes related to recent changes in production schemes and planning. Yet, our gap of knowledge regarding livestock breed-environment relationships may prevent the design of successful conservation measures. In this work, we analyse the links between livestock diversity -i.e. richness of native breeds- and a selection of environmental factors that express at broad scales, with a temporal perspective. We compare native breeds distributional patterns before and after the agricultural intensification, in the context of land-use change in mainland Spain. Our results confirm the existence of strong associations between the distribution of native livestock breeds and environmental factors. These links, however, weaken for contemporary distributions. In fact, changes in breed distribution reflect a shift towards more productive environments. Finally, we found that the areas having higher breed richness are undergoing land abandonment processes. Succeeding in the conservation of threatened native breeds will require going beyond merely genetic and production-oriented views. Ecological and sociocultural perspectives should also be accounted for as global change processes are determinant for livestock agrobiodiversity.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00755-3" target="_blank" style="color:black; text-decoration:none">Evolution of diversity explains the impact of pre-adaptation of a focal species on the structure of a natural microbial community</a></h1>
            <p>Daniel Padfield, Alex Vujakovic, Steve Paterson, Rob Griffiths, Angus Buckling, Elze Hesse</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2020-09-03</pre>
            <h2>Abstract</h2>
            <font size="4">Rapid within-species evolution can alter community structure, yet the mechanisms underpinning this effect remain unknown. Populations that rapidly evolve large amounts of phenotypic diversity are likely to interact with more species and have the largest impact on community structure. However, the evolution of phenotypic diversity is, in turn, influenced by the presence of other species. Here, we investigate how microbial community structure changes as a consequence of rapidly evolved within-species diversity using Pseudomonas fluorescens as a focal species. Evolved P. fluorescens populations showed substantial phenotypic diversification in resource-use (and correlated genomic change) irrespective of whether they were pre-adapted in isolation or in a community context. Manipulating diversity revealed that more diverse P. fluorescens populations had the greatest impact on community structure, by suppressing some bacterial taxa, but facilitating others. These findings suggest that conditions that promote the evolution of high within-population diversity should result in a larger impact on community structure.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s42003-020-01582-0" target="_blank" style="color:black; text-decoration:none">Curbing gastrointestinal infections by defensin fragment modifications without harming commensal microbiota</a></h1>
            <p>Louis Koeninger, Lisa Osbelt, Anne Berscheid, Judith Wendler, Jürgen Berger, Katharina Hipp, Till R. Lesker, Marina C. Pils, Nisar P. Malek, Benjamin A. H. Jensen, Heike Brötz-Oesterhelt, Till Strowig, Jan Wehkamp</p>
            <pre style="font-family:timesnewroman">Journal: Communications Biology    Published: 2021-01-08</pre>
            <h2>Abstract</h2>
            <font size="4">The occurrence and spread of multidrug-resistant pathogens, especially bacteria from the ESKAPE panel, increases the risk to succumb to untreatable infections. We developed a novel antimicrobial peptide, Pam-3, with antibacterial and antibiofilm properties to counter this threat. The peptide is based on an eight-amino acid carboxyl-terminal fragment of human β-defensin 1. Pam-3 exhibited prominent antimicrobial activity against multidrug-resistant ESKAPE pathogens and additionally eradicated already established biofilms in vitro, primarily by disrupting membrane integrity of its target cell. Importantly, prolonged exposure did not result in drug-resistance to Pam-3. In mouse models, Pam-3 selectively reduced acute intestinal Salmonella and established Citrobacter infections, without compromising the core microbiota, hence displaying an added benefit to traditional broad-spectrum antibiotics. In conclusion, our data support the development of defensin-derived antimicrobial agents as a novel approach to fight multidrug-resistant bacteria, where Pam-3 appears as a particularly promising microbiota-preserving candidate. Here, the authors designed a lipopeptide, Pam-3, based on an eight-amino acid carboxyl-terminal fragment of human β-defensin 1 with prominent antimicrobial activity against multidrug-resistant ESKAPE pathogens and antibiofilm properties. They show in mouse models, that Pam-3 selectively reduced acute intestinal Salmonella and established Citrobacter infections, without compromising the core microbiota.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s42003-020-01561-5" target="_blank" style="color:black; text-decoration:none">Environmental drivers of body size evolution in crocodile-line archosaurs</a></h1>
            <p>Maximilian T. Stockdale, Michael J. Benton</p>
            <pre style="font-family:timesnewroman">Journal: Communications Biology    Published: 2021-01-07</pre>
            <h2>Abstract</h2>
            <font size="4">Ever since Darwin, biologists have debated the relative roles of external and internal drivers of large-scale evolution. The distributions and ecology of living crocodilians are controlled by environmental factors such as temperature. Crocodilians have a rich history, including amphibious, marine and terrestrial forms spanning the past 247 Myr. It is uncertain whether their evolution has been driven by extrinsic factors, such as climate change and mass extinctions, or intrinsic factors like sexual selection and competition. Using a new phylogeny of crocodilians and their relatives, we model evolutionary rates using phylogenetic comparative methods. We find that body size evolution follows a punctuated, variable rate model of evolution, consistent with environmental drivers of evolution, with periods of stability interrupted by periods of change. Regression analyses show warmer environmental temperatures are associated with high evolutionary rates and large body sizes. We confirm that environmental factors played a significant role in the evolution of crocodiles. Stockdale and Benton use a new phylogeny of living and extinct crocodilians and their closest relatives to model evolutionary rates and identify the relative influences of extrinsic and intrinsic drivers of evolution in this order. Their results show that environmental factors played a significant role in the evolution of crocodiles.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41390-020-01287-4" target="_blank" style="color:black; text-decoration:none">Valuing and achieving diversity in academic medicine</a></h1>
            <p>Marva Moxey-Mims, Robin Steinhorn, Elena Fuentes-Afflick</p>
            <pre style="font-family:timesnewroman">Journal: Pediatric Research    Published: 2020-11-19</pre>
            <h2>Abstract</h2>
            <font size="4"></font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-18871-1" target="_blank" style="color:black; text-decoration:none">Health and disease markers correlate with gut microbiome composition across thousands of people</a></h1>
            <p>Ohad Manor, Chengzhen L. Dai, Sergey A. Kornilov, Brett Smith, Nathan D. Price, Jennifer C. Lovejoy, Sean M. Gibbons, Andrew T. Magis</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-10-15</pre>
            <h2>Abstract</h2>
            <font size="4">Variation in the human gut microbiome can reflect host lifestyle and behaviors and influence disease biomarker levels in the blood. Understanding the relationships between gut microbes and host phenotypes are critical for understanding wellness and disease. Here, we examine associations between the gut microbiota and ~150 host phenotypic features across ~3,400 individuals. We identify major axes of taxonomic variance in the gut and a putative diversity maximum along the Firmicutes-to-Bacteroidetes axis. Our analyses reveal both known and unknown associations between microbiome composition and host clinical markers and lifestyle factors, including host-microbe associations that are composition-specific. These results suggest potential opportunities for targeted interventions that alter the composition of the microbiome to improve host health. By uncovering the interrelationships between host diet and lifestyle factors, clinical blood markers, and the human gut microbiome at the population-scale, our results serve as a roadmap for future studies on host-microbe interactions and interventions. Variation in the gut microbiome can reflect host lifestyle, behaviour, and influence blood-based biomarkers. Here the authors examine associations between the microbiota and 150 host phenotypic features in a large cohort of >3,000 individuals.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41559-020-01329-4" target="_blank" style="color:black; text-decoration:none">Remote spectral detection of biodiversity effects on forest biomass</a></h1>
            <p>Laura J. Williams, Jeannine Cavender-Bares, Philip A. Townsend, John J. Couture, Zhihui Wang, Artur Stefanski, Christian Messier, Peter B. Reich</p>
            <pre style="font-family:timesnewroman">Journal: Nature Ecology & Evolution    Published: 2020-11-02</pre>
            <h2>Abstract</h2>
            <font size="4">Quantifying how biodiversity affects ecosystem functions through time over large spatial extents is needed for meeting global biodiversity goals yet is infeasible with field-based approaches alone. Imaging spectroscopy is a tool with potential to help address this challenge. Here, we demonstrate a spectral approach to assess biodiversity effects in young forests that provides insight into its underlying drivers. Using airborne imaging of a tree-diversity experiment, spectral differences among stands enabled us to quantify net biodiversity effects on stem biomass and canopy nitrogen. By subsequently partitioning these effects, we reveal how distinct processes contribute to diversity-induced differences in stand-level spectra, chemistry and biomass. Across stands, biomass overyielding was best explained by species with greater leaf nitrogen dominating upper canopies in mixtures, rather than intraspecific shifts in canopy structure or chemistry. Remote imaging spectroscopy may help to detect the form and drivers of biodiversity–ecosystem function relationships across space and time, advancing the capacity to monitor and manage Earth’s ecosystems. Based on a tree-diversity experiment, the authors introduce and test an approach to quantify net biodiversity effects on stem biomass in young forests using airborne spectroscopic imaging.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77673-z" target="_blank" style="color:black; text-decoration:none">Emotional well-being and gut microbiome profiles by enterotype</a></h1>
            <p>Sung-Ha Lee, Seok-Hwan Yoon, Yeonjae Jung, Namil Kim, Uigi Min, Jongsik Chun, Incheol Choi</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-26</pre>
            <h2>Abstract</h2>
            <font size="4">With increasing attention being paid to improving emotional well-being, recent evidence points to gut microbiota as a key player in regulating mental and physical health via bidirectional communication between the brain and gut. Here, we examine the association between emotional well-being and gut microbiome profiles (i.e., gut microbiome composition, diversity, and the moderating role of the enterotypes) among healthy Korean adults (n = 83, mean age = 48.9, SD = 13.2). The research was performed using high-throughput 16S rRNA gene sequencing to obtain gut microbiome profiles, as well as a self-report survey that included the Positive Affect Negative Affect Schedule (PANAS). The cluster-based analysis identified two enterotypes dominated by the genera Bacteroides (n = 49) and Prevotella (n = 34). Generalized linear regression analysis reveals significant associations between positive emotion and gut microbiome diversity (Shannon Index) among participants in the Prevotella dominant group, whereas no such relationship emerged among participants in the Bacteroides group. Moreover, a novel genus from the family Lachnospiraceae is associated with emotional well-being scores, both positive and negative. Together, the current findings highlight the enterotype-specific links between the gut microbiota community and emotion in healthy adults and suggest the possible roles of the gut microbiome in promoting mental health.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80444-5" target="_blank" style="color:black; text-decoration:none">A glimpse of antimicrobial resistance gene diversity in kefir and yoghurt</a></h1>
            <p>Adrienn Gréta Tóth, István Csabai, Gergely Maróti, Ákos Jerzsele, Attila Dubecz, Árpád V. Patai, Maura Fiona Judge, Sára Ágnes Nagy, László Makrai, Krisztián Bányai, Géza Szita, Norbert Solymosi</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-31</pre>
            <h2>Abstract</h2>
            <font size="4">Antimicrobial resistance (AMR) is a global threat gaining more and more practical significance every year. The main determinants of AMR are the antimicrobial resistance genes (ARGs). Since bacteria can share genetic components via horizontal gene transfer, even non-pathogenic bacteria may provide ARG to any pathogens which they become physically close to (e.g. in the human gut). In addition, fermented food naturally contains bacteria in high amounts. In this study, we examined the diversity of ARG content in various kefir and yoghurt samples (products, grains, bacterial strains) using a unified metagenomic approach. We found numerous ARGs of commonly used fermenting bacteria. Even with the strictest filter restrictions, we identified ARGs undermining the efficacy of aminocoumarins, aminoglycosides, carbapenems, cephalosporins, cephamycins, diaminopyrimidines, elfamycins, fluoroquinolones, fosfomycins, glycylcyclines, lincosamides, macrolides, monobactams, nitrofurans, nitroimidazoles, penams, penems, peptides, phenicols, rifamycins, tetracyclines and triclosan. In the case of gene lmrD, we detected genetic environment providing mobility of this ARG. Our findings support the theory that during the fermentation process, the ARG content of foods can grow due to bacterial multiplication. The results presented suggest that the starting culture strains of fermented foods should be monitored and selected in order to decrease the intake of ARGs via foods.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41591-020-01145-0" target="_blank" style="color:black; text-decoration:none">Modeling human adaptive immune responses with tonsil organoids</a></h1>
            <p>Lisa E. Wagar, Ameen Salahudeen, Christian M. Constantz, Ben S. Wendel, Michael M. Lyons, Vamsee Mallajosyula, Lauren P. Jatt, Julia Z. Adamska, Lisa K. Blum, Neha Gupta, Katherine J. L. Jackson, Fan Yang, Katharina Röltgen, Krishna M. Roskin, Kelly M. Blaine, Kara D. Meister, Iram N. Ahmad, Mario Cortese, Emery G. Dora, Sean N. Tucker, Anne I. Sperling, Aarti Jain, D. Huw Davies, Philip L. Felgner, Gregory B. Hammer, Peter S. Kim, William H. Robinson, Scott D. Boyd, Calvin J. Kuo, Mark M. Davis</p>
            <pre style="font-family:timesnewroman">Journal: Nature Medicine    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">Most of what we know about adaptive immunity has come from inbred mouse studies, using methods that are often difficult or impossible to confirm in humans. In addition, vaccine responses in mice are often poorly predictive of responses to those same vaccines in humans. Here we use human tonsils, readily available lymphoid organs, to develop a functional organotypic system that recapitulates key germinal center features in vitro, including the production of antigen-specific antibodies, somatic hypermutation and affinity maturation, plasmablast differentiation and class-switch recombination. We use this system to define the essential cellular components necessary to produce an influenza vaccine response. We also show that it can be used to evaluate humoral immune responses to two priming antigens, rabies vaccine and an adenovirus-based severe acute respiratory syndrome coronavirus 2 vaccine, and to assess the effects of different adjuvants. This system should prove useful for studying critical mechanisms underlying adaptive immunity in much greater depth than previously possible and to rapidly test vaccine candidates and adjuvants in an entirely human system. An in vitro human tonsil tissue-based system captures key features of a functional germinal center and can be used to study humoral immune responses to vaccines, new antigens and adjuvants.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41589-020-00708-z" target="_blank" style="color:black; text-decoration:none">Hierarchical chemical determination of amyloid polymorphs in neurodegenerative disease</a></h1>
            <p>Dan Li, Cong Liu</p>
            <pre style="font-family:timesnewroman">Journal: Nature Chemical Biology    Published: 2021-01-11</pre>
            <h2>Abstract</h2>
            <font size="4">Amyloid aggregation, which disrupts protein homeostasis, is a common pathological event occurring in human neurodegenerative diseases (NDs). Numerous evidences have shown that the structural diversity, so-called polymorphism, is decisive to the amyloid pathology and is closely associated with the onset, progression, and phenotype of ND. But how could one protein form so many stable structures? Recently, atomic structural evidence has been rapidly mounting to depict the involvement of chemical modifications in the amyloid fibril formation. In this Perspective, we aim to present a hierarchical regulation of chemical modifications including covalent post-translational modifications (PTMs) and noncovalent cofactor binding in governing the polymorphic amyloid formation, based mainly on the latest α-synuclein and Tau fibril structures. We hope to emphasize the determinant role of chemical modifications in amyloid assembly and pathology and to evoke chemical biological approaches to lead the fundamental and therapeutic research on protein amyloid state and the associated NDs. 
              
                
                  
                
              
             This Perspective summarizes the recent advances in the structural diversity of amyloid fibrils and discusses the roles of post-translational modifications and chemical cofactors in the conformational determination of fibril polymorphs.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41522-020-00173-5" target="_blank" style="color:black; text-decoration:none">The upper respiratory tract microbiome of indigenous Orang Asli in north-eastern Peninsular Malaysia</a></h1>
            <p>David W. Cleary, Denise E. Morris, Rebecca A. Anderson, Jessica Jones, Ahmed Ghazi Alattraqchi, Nor Iza A. Rahman, Salwani Ismail, Mohd Sayuti Razali, Rahmah Mohd Amin, Aniza Abd Aziz, Nor Kamaruzaman Esa, Salman Amiruddin, Ching Hoong Chew, Hafis Simin, Ramle Abdullah, Chew Chieng Yeo, Stuart C. Clarke</p>
            <pre style="font-family:timesnewroman">Journal: npj Biofilms and Microbiomes    Published: 2021-01-05</pre>
            <h2>Abstract</h2>
            <font size="4">Much microbiome research has focused on populations that are predominantly of European descent, and from narrow demographics that do not capture the socio-economic and lifestyle differences which impact human health. Here we examined the airway microbiomes of the Orang Asli, the indigenous peoples of Malaysia. A total of 130 participants were recruited from two sites in the north-eastern state of Terengganu in Peninsular Malaysia. Using 16S rRNA sequencing, the nasal microbiome was significantly more diverse in those aged 5–17 years compared to 50+ years (p = 0.023) and clustered by age (PERMANOVA analysis of the Bray–Curtis distance, p = 0.001). Hierarchical clustering of Bray–Curtis dissimilarity scores revealed six microbiome clusters. The largest cluster (n = 28; 35.4%) had a marked abundance of Corynebacterium. In the oral microbiomes Streptococcus, Neisseria and Haemophilus were dominant. Using conventional microbiology, high levels of Staphylococcus aureus carriage were observed, particularly in the 18–65 age group (n = 17/36; 47.2% 95% CI: 30.9–63.5). The highest carriage of pneumococci was in the <5 and 5 to 17 year olds, with 57.1% (4/7) and 49.2% (30/61), respectively. Sixteen pneumococcal serotypes were identified, the most common being the nonvaccine-type 23A (14.6%) and the vaccine-type 6B (9.8%). The prevalence of pneumococcal serotypes covered by pneumococcal conjugate vaccines support introduction into a Malaysian national immunisation schedule. In addition, the dominance of Corynebacterium in the airway microbiomes is intriguing given their role as a potentially protective commensal with respect to acute infection and respiratory health.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41590-020-00830-z" target="_blank" style="color:black; text-decoration:none">Broadly effective metabolic and immune recovery with C5 inhibition in CHAPLE disease</a></h1>
            <p>Ahmet Ozen, Nurhan Kasap, Ivan Vujkovic-Cvijin, Richard Apps, Foo Cheung, Elif Karakoc-Aydiner, Bilge Akkelle, Sinan Sari, Engin Tutar, Figen Ozcay, Dilara Kocacik Uygun, Ali Islek, Gamze Akgun, Merve Selcuk, Oya Balci Sezer, Yu Zhang, Gunsel Kutluk, Erdem Topal, Ersin Sayar, Cigdem Celikel, Roderick H. J. Houwen, Aysen Bingol, Ismail Ogulur, Sevgi Bilgic Eltan, Andrew L. Snow, Camille Lake, Giovanna Fantoni, Camille Alba, Brian Sellers, Samuel D. Chauvin, Clifton L. Dalgard, Olivier Harari, Yan G. Ni, Ming-Dauh Wang, Kishor Devalaraja-Narashimha, Poorani Subramanian, Rabia Ergelen, Reha Artan, Sukru Nail Guner, Buket Dalgic, John Tsang, Yasmine Belkaid, Deniz Ertem, Safa Baris, Michael J. Lenardo</p>
            <pre style="font-family:timesnewroman">Journal: Nature Immunology    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Complement hyperactivation, angiopathic thrombosis and protein-losing enteropathy (CHAPLE disease) is a lethal disease caused by genetic loss of the complement regulatory protein CD55, leading to overactivation of complement and innate immunity together with immunodeficiency due to immunoglobulin wasting in the intestine. We report in vivo human data accumulated using the complement C5 inhibitor eculizumab for the medical treatment of patients with CHAPLE disease. We observed cessation of gastrointestinal pathology together with restoration of normal immunity and metabolism. We found that patients rapidly renormalized immunoglobulin concentrations and other serum proteins as revealed by aptamer profiling, re-established a healthy gut microbiome, discontinued immunoglobulin replacement and other treatments and exhibited catch-up growth. Thus, we show that blockade of C5 by eculizumab effectively re-establishes regulation of the innate immune complement system to substantially reduce the pathophysiological manifestations of CD55 deficiency in humans. CHAPLE disease is a lethal syndrome caused by genetic loss of the complement regulatory protein CD55. Lenardo, Ozen and their colleagues report that blockade of C5 complement activation in a small cohort of pediatric patients with CHAPLE disease reduced gastrointestinal pathology and restored their immunity and growth.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78934-7" target="_blank" style="color:black; text-decoration:none">DNA metabarcoding uncovers fungal diversity in soils of protected and non-protected areas on Deception Island, Antarctica</a></h1>
            <p>Luiz Henrique Rosa, Thamar Holanda da Silva, Mayara Baptistucci Ogaki, Otávio Henrique Bezerra Pinto, Michael Stech, Peter Convey, Micheline Carvalho-Silva, Carlos Augusto Rosa, Paulo E. A. S. Câmara</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-15</pre>
            <h2>Abstract</h2>
            <font size="4">We assessed soil fungal diversity at two sites on Deception Island, South Shetland Islands, Antarctica using DNA metabarcoding analysis. The first site was a relatively undisturbed area, and the second was much more heavily impacted by research and tourism. We detected 346 fungal amplicon sequence variants dominated by the phyla Ascomycota, Basidiomycota, Mortierellomycota and Chytridiomycota. We also detected taxa belonging to the rare phyla Mucoromycota and Rozellomycota, which have been difficult to detect in Antarctica by traditional isolation methods. Cladosporium sp., Pseudogymnoascus roseus, Leotiomycetes sp. 2, Penicillium sp., Mortierella sp. 1, Mortierella sp. 2, Pseudogymnoascus appendiculatus and Pseudogymnoascus sp. were the most dominant fungi. In addition, 440,153 of the total of 1,214,875 reads detected could be classified only at the level of Fungi. In both sampling areas the DNA of opportunistic, phytopathogenic and symbiotic fungi were detected, which might have been introduced by human activities, transported by birds or wind, and/or represent resident fungi not previously reported from Antarctica. Further long-term studies are required to elucidate how biological colonization in the island may be affected by climatic changes and/or other anthropogenic influences.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78642-2" target="_blank" style="color:black; text-decoration:none">Impact of outdoor nature-related activities on gut microbiota, fecal serotonin, and perceived stress in preschool children: the Play&Grow randomized controlled trial</a></h1>
            <p>Tanja Sobko, Suisha Liang, Will H. G. Cheng, Hein M. Tun</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-15</pre>
            <h2>Abstract</h2>
            <font size="4">Due to rapid urbanization, children today have fewer opportunities to interact with nature and this may result in a greater risk for developing stress and depression. Outdoor nature-related activities can enhance general well-being. However, the underlying mechanisms are not fully delineated. Here we recruited 54 preschool children to participate in a 10-week structured nature-related “Play&Grow” program. Following the intervention, children were assessed for connectedness to nature and perceived stress levels using validated questionnaires. Moreover, fecal serotonin level and gut microbiota profiles were measured by ELISA and 16S rDNA amplicon sequencing, respectively. Children were significantly more connected to nature after the intervention. Their gut microbiota altered, especially by modulating the abundance of Roseburia and the fecal-serotonin level. Moreover, we also observed a reduction in the overall perceived stress, particularly in the frequency of anger among these children. This study is the first to demonstrate the impact of nature-related activities on gut microbiota, fecal serotonin and psychosocial behaviour of preschool children. However, further mechanistic studies are needed to confirm the functional role of gut microbiota in the association between connectedness to nature and improved psychosocial behavior.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-19929-w" target="_blank" style="color:black; text-decoration:none">A collection of bacterial isolates from the pig intestine reveals functional and taxonomic diversity</a></h1>
            <p>David Wylensek, Thomas C. A. Hitch, Thomas Riedel, Afrizal Afrizal, Neeraj Kumar, Esther Wortmann, Tianzhe Liu, Saravanan Devendran, Till R. Lesker, Sara B. Hernández, Viktoria Heine, Eva M. Buhl, Paul M. D’Agostino, Fabio Cumbo, Thomas Fischöder, Marzena Wyschkon, Torey Looft, Valeria R. Parreira, Birte Abt, Heidi L. Doden, Lindsey Ly, João M. P. Alves, Markus Reichlin, Krzysztof Flisikowski, Laura Navarro Suarez, Anthony P. Neumann, Garret Suen, Tomas de Wouters, Sascha Rohn, Ilias Lagkouvardos, Emma Allen-Vercoe, Cathrin Spröer, Boyke Bunk, Anja J. Taverne-Thiele, Marcel Giesbers, Jerry M. Wells, Klaus Neuhaus, Angelika Schnieke, Felipe Cava, Nicola Segata, Lothar Elling, Till Strowig, Jason M. Ridlon, Tobias A. M. Gulder, Jörg Overmann, Thomas Clavel</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-12-15</pre>
            <h2>Abstract</h2>
            <font size="4">Our knowledge about the gut microbiota of pigs is still scarce, despite the importance of these animals for biomedical research and agriculture. Here, we present a collection of cultured bacteria from the pig gut, including 110 species across 40 families and nine phyla. We provide taxonomic descriptions for 22 novel species and 16 genera. Meta-analysis of 16S rRNA amplicon sequence data and metagenome-assembled genomes reveal prevalent and pig-specific species within Lactobacillus, Streptococcus, Clostridium, Desulfovibrio, Enterococcus, Fusobacterium, and several new genera described in this study. Potentially interesting functions discovered in these organisms include a fucosyltransferase encoded in the genome of the novel species Clostridium porci, and prevalent gene clusters for biosynthesis of sactipeptide-like peptides. Many strains deconjugate primary bile acids in in vitro assays, and a Clostridium scindens strain produces secondary bile acids via dehydroxylation. In addition, cells of the novel species Bullifex porci are coccoidal or spherical under the culture conditions tested, in contrast with the usual helical shape of other members of the family Spirochaetaceae. The strain collection, called ‘Pig intestinal bacterial collection’ (PiBAC), is publicly available at 
                www.dsmz.de/pibac
                
               and opens new avenues for functional studies of the pig gut microbiota. The authors present a public collection of 117 bacterial isolates from the pig gut, including the description of 38 novel taxa. Interesting functions discovered in these organisms include a new fucosyltransferease and sactipeptide-like molecules encoded by biosynthetic gene clusters.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41588-020-00722-w" target="_blank" style="color:black; text-decoration:none">Triticum population sequencing provides insights into wheat adaptation</a></h1>
            <p>Yao Zhou, Xuebo Zhao, Yiwen Li, Jun Xu, Aoyue Bi, Lipeng Kang, Daxing Xu, Haofeng Chen, Ying Wang, Yuan-ge Wang, Sanyang Liu, Chengzhi Jiao, Hongfeng Lu, Jing Wang, Changbin Yin, Yuling Jiao, Fei Lu</p>
            <pre style="font-family:timesnewroman">Journal: Nature Genetics    Published: 2020-10-26</pre>
            <h2>Abstract</h2>
            <font size="4">Bread wheat expanded its habitat from a core area of the Fertile Crescent to global environments within ~10,000 years. The genetic mechanisms of this remarkable evolutionary success are not well understood. By whole-genome sequencing of populations from 25 subspecies within the genera Triticum and Aegilops, we identified composite introgression from wild populations contributing to a substantial portion (4–32%) of the bread wheat genome, which increased the genetic diversity of bread wheat and allowed its divergent adaptation. Meanwhile, convergent adaptation to human selection showed 2- to 16-fold enrichment relative to random expectation—a certain set of genes were repeatedly selected in Triticum species despite their drastic differences in ploidy levels and growing zones, indicating the important role of evolutionary constraints in shaping the adaptive landscape of bread wheat. These results showed the genetic necessities of wheat as a global crop and provided new perspectives on transferring adaptive success across species for crop improvement. Whole-genome sequencing of wheat populations from 25 subspecies within the genera Triticum and Aegilops provides insights into the role of evolutionary constraints in shaping the adaptive landscape of bread wheat.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78923-w" target="_blank" style="color:black; text-decoration:none">Population genetics of the Manila clam ( Ruditapes philippinarum) in East Asia</a></h1>
            <p>Yue Tan, Lei Fang, Ming Qiu, Zhongming Huo, Xiwu Yan</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-14</pre>
            <h2>Abstract</h2>
            <font size="4">The Manila clam Ruditapes philippinarum is the world’s second most important bivalve mollusk commercially farmed, whose indigenous populations are mainly distributed in the coastal areas of East Asia. However, with the development of commercialization, mixture of populations and loss of local germplasm have become prominent problems. Here, genetic differentiation of seven Manila clam populations from East Asia was investigated through analyzing the polymorphism of the mitochondrial cytochrome C oxidase subunit I (COI) gene as well as 20 simple sequence repeat (SSR) molecular loci. In total, 40 haplotypes were identified, among which 31 were unique. Moreover, two main haplotypes were detected with several radiating derived haplotypes. Populations in Japan-North Korea shared haplotype Hap_31, and populations in China shared haplotype Hap_7, suggesting that the natural geographical isolation of the Yangtze River and the Yalu River might have divided the East Asian indigenous populations into three groups, which were located in South China, North China, and Japan-North Korea, respectively. The Aquaculture breeding activities from South to North in China might have promoted gene exchange among Manila clam populations. Population in Laizhou had the highest genetic diversity and therefore could be an excellent germplasm source.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-76656-4" target="_blank" style="color:black; text-decoration:none">Effects of two measures of riparian plant biodiversity on litter decomposition and associated processes in stream microcosms</a></h1>
            <p>Naiara López-Rojo, Javier Pérez, Ana Basaguren, Jesús Pozo, Juan Rubio-Ríos, J. Jesús Casas, Luz Boyero</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-12</pre>
            <h2>Abstract</h2>
            <font size="4">Plant litter decomposition is a key ecosystem process that can be altered by global changes such as biodiversity loss. These effects can be particularly important in detritus-based ecosystems, such as headwater streams, which are mainly fuelled by allochthonous plant litter inputs. However, experiments examining effects of plant diversity on litter decomposition in streams have not reached consensus about which measures of biodiversity are more relevant. We explored the influence of two of these measures, plant species richness (SR; monocultures vs. 3-species mixtures) and phylogenetic distance (PD; species belonging to the same family vs. different families), on leaf litter decomposition and associated processes and variables (nutrient dynamics, fungal biomass and detritivore growth), in a stream microcosm experiment using litter from 9 tree species belonging to 3 families. We found a negative effect of SR on decomposition (which contradicted the results of previous experiments) but a positive effect on fungal biomass. While PD did not affect decomposition, both SR and PD altered nutrient dynamics: there was greater litter and detritivore N loss in low-PD mixtures, and greater litter P loss and detritivore P gain in monocultures. This suggested that the number of species in mixtures and the similarity of their traits both modulated nutrient availability and utilization by detritivores. Moreover, the greater fungal biomass with higher SR could imply positive effects on detritivores in the longer term. Our results provide new insights of the functional repercussions of biodiversity loss by going beyond the often-explored relationship between SR and decomposition, and reveal an influence of plant species phylogenetic relatedness on nutrient cycling that merits further investigation.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78940-9" target="_blank" style="color:black; text-decoration:none">The largest known cowrie and the iterative evolution of giant cypraeid gastropods</a></h1>
            <p>Stefano Dominici, Mariagabriella Fornasiero, Luca Giusberti</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-14</pre>
            <h2>Abstract</h2>
            <font size="4">Based on the fossil record, we explore the macroevolutionary relationship between species richness and gigantism in cowries (Cypraeidae), the best-studied family of gastropods, with a global diversity distribution that parallels that of tropical corals, mangroves and seagrasses. We introduce Vicetia bizzottoi sp. nov. based on a Priabonian fossil found in northeastern Italy, the largest documented cowrie found so far and the youngest of a lineage of Eocene Gisortiinae species. The Gisortiinae stratigraphic record in western Europe indicates that species selection favoured large size and armouring of the shell. Palaeoecology and per-stage species richness suggest that gigantism occurred in peripheral habitats with respect to diversity hotspots, where smaller species were favoured. The Eocene–Oligocene boundary was marked by a turnover and the Chattian global warming favoured small-sized species of derived clades. Species selection leading to gigantism is further documented in Miocene lineages of Zoila and Umbilia, in the southern hemisphere, two extant genera distributed at the periphery of modern diversity hotspots, suggesting that the negative relationship between size and diversity is a recurring pattern in the evolutionary history of cowries. This palaeontological evidence is projected onto the existing hypotheses that explain analogous biogeographic patterns in various other taxa. Likewise, body size-species richness negative relationship was possibly driven in cowries by physiological, ecological and life history constraints.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78057-z" target="_blank" style="color:black; text-decoration:none">Bacterial diversity and functional analysis of severe early childhood caries and recurrence in India</a></h1>
            <p>Balakrishnan Kalpana, Puniethaa Prabhu, Ashaq Hussain Bhat, Arunsaikiran Senthilkumar, Raj Pranap Arun, Sharath Asokan, Sachin S. Gunthe, Rama S. Verma</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-04</pre>
            <h2>Abstract</h2>
            <font size="4">Dental caries is the most prevalent oral disease affecting nearly 70% of children in India and elsewhere. Micro-ecological niche based acidification due to dysbiosis in oral microbiome are crucial for caries onset and progression. Here we report the tooth bacteriome diversity compared in Indian children with caries free (CF), severe early childhood caries (SC) and recurrent caries (RC). High quality V3–V4 amplicon sequencing revealed that SC exhibited high bacterial diversity with unique combination and interrelationship. Gracillibacteria_GN02 and TM7 were unique in CF and SC respectively, while Bacteroidetes, Fusobacteria were significantly high in RC. Interestingly, we found Streptococcus oralis subsp. tigurinus clade 071 in all groups with significant abundance in SC and RC. Positive correlation between low and high abundant bacteria as well as with TCS, PTS and ABC transporters were seen from co-occurrence network analysis. This could lead to persistence of SC niche resulting in RC. Comparative in vitro assessment of biofilm formation showed that the standard culture of S. oralis and its phylogenetically similar clinical isolates showed profound biofilm formation and augmented the growth and enhanced biofilm formation in S. mutans in both dual and multispecies cultures.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-62362-8" target="_blank" style="color:black; text-decoration:none">Defining Individual-Level Genetic Diversity and Similarity Profiles</a></h1>
            <p>Zhanshan (Sam) Ma, Lianwei Li, Ya-Ping Zhang</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-04-02</pre>
            <h2>Abstract</h2>
            <font size="4">Classic concepts of genetic (gene) diversity (heterozygosity) such as Nei & Li’s nucleotide diversity were defined within a population context. Although variations are often measured in population context, the basic carriers of variation are individuals. Hence, measuring variations such as SNP of an individual against a reference genome, which has been ignored previously, is certainly in its own right. Indeed, similar practice has been a tradition in community ecology, where the basic unit of diversity measure is individual community sample. We propose to use Renyi’s-entropy-based Hill numbers to define individual-level genetic diversity and similarity and demonstrate the definitions with the SNP (single nucleotide polymorphism) datasets from the 1000-Genomes Project. Hill numbers, derived from Renyi’s entropy (of which Shannon’s entropy is a special case), have found widely applications including measuring the quantum information entanglement and ecological diversity. The demonstrated individual-level SNP diversity not only complements the existing population-level genetic diversity concepts, but also offers building blocks for comparative genetic analysis at higher levels. The concept of individual covers, but is not limited to, individual chromosome, region of chromosome, gene cluster(s), or whole genome. Similarly, the SNP can be replaced by other structural variants or mutation types such as indels.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-19252-4" target="_blank" style="color:black; text-decoration:none">General destabilizing effects of eutrophication on grassland productivity at multiple spatial scales</a></h1>
            <p>Yann Hautier, Pengfei Zhang, Michel Loreau, Kevin R. Wilcox, Eric W. Seabloom, Elizabeth T. Borer, Jarrett E. K. Byrnes, Sally E. Koerner, Kimberly J. Komatsu, Jonathan S. Lefcheck, Andy Hector, Peter B. Adler, Juan Alberti, Carlos A. Arnillas, Jonathan D. Bakker, Lars A. Brudvig, Miguel N. Bugalho, Marc Cadotte, Maria C. Caldeira, Oliver Carroll, Mick Crawley, Scott L. Collins, Pedro Daleo, Laura E. Dee, Nico Eisenhauer, Anu Eskelinen, Philip A. Fay, Benjamin Gilbert, Amandine Hansar, Forest Isbell, Johannes M. H. Knops, Andrew S. MacDougall, Rebecca L. McCulley, Joslin L. Moore, John W. Morgan, Akira S. Mori, Pablo L. Peri, Edwin T. Pos, Sally A. Power, Jodi N. Price, Peter B. Reich, Anita C. Risch, Christiane Roscher, Mahesh Sankaran, Martin Schütz, Melinda Smith, Carly Stevens, Pedro M. Tognetti, Risto Virtanen, Glenda M. Wardle, Peter A. Wilfahrt, Shaopeng Wang</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-10-23</pre>
            <h2>Abstract</h2>
            <font size="4">Eutrophication is a widespread environmental change that usually reduces the stabilizing effect of plant diversity on productivity in local communities. Whether this effect is scale dependent remains to be elucidated. Here, we determine the relationship between plant diversity and temporal stability of productivity for 243 plant communities from 42 grasslands across the globe and quantify the effect of chronic fertilization on these relationships. Unfertilized local communities with more plant species exhibit greater asynchronous dynamics among species in response to natural environmental fluctuations, resulting in greater local stability (alpha stability). Moreover, neighborhood communities that have greater spatial variation in plant species composition within sites (higher beta diversity) have greater spatial asynchrony of productivity among communities, resulting in greater stability at the larger scale (gamma stability). Importantly, fertilization consistently weakens the contribution of plant diversity to both of these stabilizing mechanisms, thus diminishing the positive effect of biodiversity on stability at differing spatial scales. Our findings suggest that preserving grassland functional stability requires conservation of plant diversity within and among ecological communities. Eutrophication has been shown to weaken diversity-stability relationships in grasslands, but it is unclear whether the effect depends on scale. Analysing a globally distributed network of grassland sites, the authors show a positive role of beta diversity and spatial asynchrony as drivers of stability but find that nitrogen enrichment weakens the diversity-stability relationships at different spatial scales.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77919-w" target="_blank" style="color:black; text-decoration:none">Glioma and temozolomide induced alterations in gut microbiome</a></h1>
            <p>Anthony Patrizz, Antonio Dono, Soheil Zorofchian, Gabriella Hines, Takeshi Takayasu, Nuruddin Husein, Yoshihiro Otani, Octavio Arevalo, H. Alex Choi, Jude Savarraj, Nitin Tandon, Bhanu P. Ganesh, Balveen Kaur, Louise D. McCullough, Leomar Y. Ballester, Yoshua Esquenazi</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-03</pre>
            <h2>Abstract</h2>
            <font size="4">The gut microbiome is fundamental in neurogenesis processes. Alterations in microbial constituents promote inflammation and immunosuppression. Recently, in immune-oncology, specific microbial taxa have been described to enhance the effects of therapeutic modalities. However, the effects of microbial dysbiosis on glioma are still unknown. The aim of this study was to explore the effects of glioma development and Temozolomide (TMZ) on fecal microbiome in mice and humans. C57BL/6 mice were implanted with GL261/Sham and given TMZ/Saline. Fecal samples were collected longitudinally and analyzed by 16S rRNA sequencing. Fecal samples were collected from healthy controls as well as glioma patients at diagnosis, before and after chemoradiation. Compared to healthy controls, mice and glioma patients demonstrated significant differences in beta diversity, Firmicutes/Bacteroides (F/B) ratio, and increase of Verrucomicrobia phylum and Akkermansia genus. These changes were not observed following TMZ in mice. TMZ treatment in the non-tumor bearing mouse-model diminished the F/B ratio, increase Muribaculaceae family and decrease Ruminococcaceae family. Nevertheless, there were no changes in Verrucomicrobia/Akkermansia. Glioma development leads to gut dysbiosis in a mouse-model, which was not observed in the setting of TMZ. These findings seem translational to humans and warrant further study.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41437-020-00367-9" target="_blank" style="color:black; text-decoration:none">Riverscape genetics in brook lamprey: genetic diversity is less influenced by river fragmentation than by gene flow with the anadromous ecotype</a></h1>
            <p>Quentin Rougemont, Victoria Dolo, Adrien Oger, Anne-Laure Besnard, Dominique Huteau, Marie-Agnès Coutellec, Charles Perrier, Sophie Launey, Guillaume Evanno</p>
            <pre style="font-family:timesnewroman">Journal: Heredity    Published: 2020-09-28</pre>
            <h2>Abstract</h2>
            <font size="4">Understanding the effect of human-induced landscape fragmentation on gene flow and evolutionary potential of wild populations has become a major concern. Here, we investigated the effect of riverscape fragmentation on patterns of genetic diversity in the freshwater resident European brook lamprey (Lampetra planeri) that has a low ability to pass obstacles to migration. We tested the hypotheses of (i) asymmetric gene flow following water current and (ii) an effect of gene flow with the closely related anadromous river lamprey (L. fluviatilis) ecotype on L. planeri genetic diversity. We genotyped 2472 individuals, including 225 L. fluviatilis, sampled from 81 sites upstream and downstream barriers to migration, in 29 western European rivers. Linear modelling revealed a strong positive relationship between genetic diversity and the distance from the river source, consistent with expected patterns of decreased gene flow into upstream populations. However, the presence of anthropogenic barriers had a moderate effect on spatial genetic structure. Accordingly, we found evidence for downstream-directed gene flow, supporting the hypothesis that barriers do not limit dispersal mediated by water flow. Downstream L. planeri populations in sympatry with L. fluviatilis displayed consistently higher genetic diversity. We conclude that genetic drift and slight downstream gene flow drive the genetic make-up of upstream L. planeri populations whereas gene flow between ecotypes maintains higher levels of genetic diversity in L. planeri populations sympatric with L. fluviatilis. We discuss the implications of these results for the design of conservation strategies of lamprey, and other freshwater organisms with several ecotypes, in fragmented dendritic river networks.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s42003-020-01377-3" target="_blank" style="color:black; text-decoration:none">Genomic and transcriptomic variation defines the chromosome-scale assembly of Haemonchus contortus , a model gastrointestinal worm</a></h1>
            <p>Stephen R. Doyle, Alan Tracey, Roz Laing, Nancy Holroyd, David Bartley, Wojtek Bazant, Helen Beasley, Robin Beech, Collette Britton, Karen Brooks, Umer Chaudhry, Kirsty Maitland, Axel Martinelli, Jennifer D. Noonan, Michael Paulini, Michael A. Quail, Elizabeth Redman, Faye H. Rodgers, Guillaume Sallé, Muhammad Zubair Shabbir, Geetha Sankaranarayanan, Janneke Wit, Kevin L. Howe, Neil Sargison, Eileen Devaney, Matthew Berriman, John S. Gilleard, James A. Cotton</p>
            <pre style="font-family:timesnewroman">Journal: Communications Biology    Published: 2020-11-09</pre>
            <h2>Abstract</h2>
            <font size="4">Haemonchus contortus is a globally distributed and economically important gastrointestinal pathogen of small ruminants and has become a key nematode model for studying anthelmintic resistance and other parasite-specific traits among a wider group of parasites including major human pathogens. Here, we report using PacBio long-read and OpGen and 10X Genomics long-molecule methods to generate a highly contiguous 283.4 Mbp chromosome-scale genome assembly including a resolved sex chromosome for the MHco3(ISE).N1 isolate. We show a remarkable pattern of conservation of chromosome content with Caenorhabditis elegans, but almost no conservation of gene order. Short and long-read transcriptome sequencing allowed us to define coordinated transcriptional regulation throughout the parasite’s life cycle and refine our understanding of cis- and trans-splicing. Finally, we provide a comprehensive picture of chromosome-wide genetic diversity both within a single isolate and globally. These data provide a high-quality comparison for understanding the evolution and genomics of Caenorhabditis and other nematodes and extend the experimental tractability of this model parasitic nematode in understanding helminth biology, drug discovery and vaccine development, as well as important adaptive traits such as drug resistance. Stephen Doyle et al. report the chromosome-scale genome assembly and transcriptome sequence data of Haemonchus contortus, a key parasitic nematode model. These data show nearly complete conservation of chromosome content with C. elegans and brings insight into transcriptional regulation and chromosome-wide genetic diversity in this important pathogen.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41536-020-00110-2" target="_blank" style="color:black; text-decoration:none">ILB ® resolves inflammatory scarring and promotes functional tissue repair</a></h1>
            <p>Lisa J. Hill, Hannah F. Botfield, Ghazala Begum, Omar Qureshi, Vasanthy Vigneswara, Imran Masood, Nicholas M. Barnes, Lars Bruce, Ann Logan</p>
            <pre style="font-family:timesnewroman">Journal: npj Regenerative Medicine    Published: 2021-01-07</pre>
            <h2>Abstract</h2>
            <font size="4">Fibrotic disease is a major cause of mortality worldwide, with fibrosis arising from prolonged inflammation and aberrant extracellular matrix dynamics. Compromised cellular and tissue repair processes following injury, infection, metabolic dysfunction, autoimmune conditions and vascular diseases leave tissues susceptible to unresolved inflammation, fibrogenesis, loss of function and scarring. There has been limited clinical success with therapies for inflammatory and fibrotic diseases such that there remains a large unmet therapeutic need to restore normal tissue homoeostasis without detrimental side effects. We investigated the effects of a newly formulated low molecular weight dextran sulfate (LMW-DS), termed ILB®, to resolve inflammation and activate matrix remodelling in rodent and human disease models. We demonstrated modulation of the expression of multiple pro-inflammatory cytokines and chemokines in vitro together with scar resolution and improved matrix remodelling in vivo. Of particular relevance, we demonstrated that ILB® acts, in part, by downregulating transforming growth factor (TGF)β signalling genes and by altering gene expression relating to extracellular matrix dynamics, leading to tissue remodelling, reduced fibrosis and functional tissue regeneration. These observations indicate the potential of ILB® to alleviate fibrotic diseases.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78630-6" target="_blank" style="color:black; text-decoration:none">DNA metabarcoding of fungal diversity in air and snow of Livingston Island, South Shetland Islands, Antarctica</a></h1>
            <p>Luiz Henrique Rosa, Otávio Henrique Bezerra Pinto, Tina Šantl-Temkiv, Peter Convey, Micheline Carvalho-Silva, Carlos Augusto Rosa, Paulo E. A. S. Câmara</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-11</pre>
            <h2>Abstract</h2>
            <font size="4">We assessed fungal diversity present in air and freshly deposited snow samples obtained from Livingston Island, Antarctica, using DNA metabarcoding through high throughput sequencing (HTS). A total of 740 m3 of air were pumped through a 0.22 µm membrane. Snow obtained shortly after deposition was kept at room temperature and yielded 3.760 L of water, which was filtered using Sterivex membranes of 0.22 µm mesh size. The total DNA present was extracted and sequenced. We detected 171 fungal amplicon sequence variants (ASVs), 70 from the air and 142 from the snow. They were dominated by the phyla Ascomycota, Basidiomycota, Mortierellomycota and Mucoromycota. Pseudogymnoascus, Cladosporium, Mortierella and Penicillium sp. were the most dominant ASVs detected in the air in rank order. In snow, Cladosporium, Pseudogymnoascus, Penicillium, Meyerozyma, Lecidea, Malassezia, Hanseniaspora, Austroplaca, Mortierella, Rhodotorula, Penicillium, Thelebolus, Aspergillus, Poaceicola, Glarea and Lecanora were the dominant ASVs present. In general, the two fungal assemblages displayed high diversity, richness, and dominance indices, with the assemblage found in snow having the highest diversity indices. Of the total fungal ASVs detected, 29 were only present in the air sample and 101 in the snow sample, with only 41 present in both samples; however, when only the dominant taxa from both samples were compared none occurred only in the air and, among the rare portion, 26 taxa occurred in both air and snow. Application of HTS revealed the presence of a more diverse fungal community in the air and snow of Livingston Island in comparison with studies using traditional isolation methods. The assemblages were dominated by cold-adapted and cosmopolitan fungal taxa, including members of the genera Pseudogymnoascus, Malassezia and Rhodotorula, which include some taxa reported as opportunistic. Our results support the hypothesis that the presence of microbiota in the airspora indicates the possibility of dispersal around Antarctica in the air column. However, further aeromycology studies are required to understand the dynamics of fungal dispersal within and beyond Antarctica.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78659-7" target="_blank" style="color:black; text-decoration:none">Effects of disturbances by forest elephants on diversity of trees and insects in tropical rainforests on Mount Cameroon</a></h1>
            <p>Vincent Maicher, Sylvain Delabye, Mercy Murkwe, Jiří Doležal, Jan Altman, Ishmeal N. Kobe, Julie Desmist, Eric B. Fokam, Tomasz Pyrcz, Robert Tropek</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-10</pre>
            <h2>Abstract</h2>
            <font size="4">Natural disturbances are essential for tropical forests biodiversity. In the Afrotropics, megaherbivores have played a key role before their recent decline. Contrastingly to savanna elephants, forest elephants’ impact on ecosystems remains poorly studied. Few decades ago, forests on Mount Cameroon were divided by lava flows, not being crossed by a local population of forest elephants until now. We assessed communities of trees, butterflies and two guilds of moths in the disturbed and undisturbed forests split by the longest lava flow. We surveyed 32 plots, recording 2025 trees of 97 species, and 7853 insects of 437 species. The disturbed forests differed in reduced tree density, height, and high canopy cover, and in increased DBH. Forest elephants’ selective browsing and foraging also decreased tree species richness and altered their composition. The elephant disturbance increased butterfly species richness and had various effects on species richness and composition of the insect groups. These changes were likely caused by disturbance-driven alterations of habitats and species composition of trees. Moreover, the abandonment of forests by elephants led to local declines of range-restricted butterflies. The recent declines of forest elephants across the Afrotropics probably caused similar changes in forest biodiversity and should be reflected by conservation actions.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78459-z" target="_blank" style="color:black; text-decoration:none">Comparative analysis of bacterioplankton assemblages from two subtropical karst reservoirs of southwestern China with contrasting trophic status</a></h1>
            <p>Qiang Li, Yadan Huang, Shenglin Xin, Zhongyi Li</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-18</pre>
            <h2>Abstract</h2>
            <font size="4">Although bacterioplankton play an important role in aquatic ecosystems, less is known about bacterioplankton assemblages from subtropical karst reservoirs of southwestern China with contrasting trophic status. Here, 16S rRNA gene next-generation sequencing coupled with water chemistry analysis was applied to compare the bacterioplankton communities from a light eutrophic reservoir, DL Reservoir, and a mesotrophic reservoir, WL Reservoir, in subtropical karst area of southwestern China. Our findings indicated that Proteobacteria, Firmicutes, Actinobacteria, Bacteroidetes, Cyanobacteria and Verrucomicrobia dominated bacterioplankton community with contrasting relative frequency in the two subtropical karst reservoirs. Proteobacteria and Bacteroidetes were the core communities, which played important roles in karst biogeochemical cycles. Though WT, TN and DOC play the decisive role in assembling karst aquatic bacterioplankton, trophic status exerted significantly negative direct effects on bacterioplankton community composition and alpha diversity. Due to contrasting trophic status in the two reservoirs, the dominant taxa such as Enterobacter, Clostridium sensu stricto, Candidatus Methylacidiphilum and Flavobacteriia, that harbor potential functions as valuable and natural indicators of karst water health status, differed in DL Reservoir and WL Reservoir.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s42003-020-01477-0" target="_blank" style="color:black; text-decoration:none">Differential longitudinal establishment of human fecal bacterial communities in germ-free porcine and murine models</a></h1>
            <p>Nirosh D. Aluthge, Wesley A. Tom, Alison C. Bartenslager, Thomas E. Burkey, Phillip S. Miller, Kelly D. Heath, Craig Kreikemeier-Bower, Hatem Kittana, Robert J. Schmaltz, Amanda E. Ramer-Tait, Samodha C. Fernando</p>
            <pre style="font-family:timesnewroman">Journal: Communications Biology    Published: 2020-12-11</pre>
            <h2>Abstract</h2>
            <font size="4">The majority of microbiome studies focused on understanding mechanistic relationships between the host and the microbiota have used mice and other rodents as the model of choice. However, the domestic pig is a relevant model that is currently underutilized for human microbiome investigations. In this study, we performed a direct comparison of the engraftment of fecal bacterial communities from human donors between human microbiota-associated (HMA) piglet and mouse models under identical dietary conditions. Analysis of 16S rRNA genes using amplicon sequence variants (ASVs) revealed that with the exception of early microbiota from infants, the more mature microbiotas tested established better in the HMA piglets compared to HMA mice. Of interest was the greater transplantation success of members belonging to phylum Firmicutes in the HMA piglets compared to the HMA mice. Together, these results provide evidence for the HMA piglet model potentially being more broadly applicable for donors with more mature microbiotas while the HMA mouse model might be more relevant for developing microbiotas such as those of infants. This study also emphasizes the necessity to exercise caution in extrapolating findings from HMA animals to humans, since up to 28% of taxa from some donors failed to colonize either model. Aluthge et al. evaluated the establishment of human fecal bacterial communities in human microbiota-associated (HMA) piglet and mouse models under identical dietary conditions. They find that human fecal bacterial communities from mature microbiotas establish more successfully in the HMA porcine model than mice, suggesting the broader applicability of the piglet model for human microbiome studies.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78673-9" target="_blank" style="color:black; text-decoration:none">Global emergence and evolutionary dynamics of bluetongue virus</a></h1>
            <p>Moh A. Alkhamis, Cecilia Aguilar-Vega, Nicholas M. Fountain-Jones, Kai Lin, Andres M. Perez, José M. Sánchez-Vizcaíno</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-10</pre>
            <h2>Abstract</h2>
            <font size="4">Bluetongue virus (BTV) epidemics are responsible for worldwide economic losses of up to US$ 3 billion. Understanding the global evolutionary epidemiology of BTV is critical in designing intervention programs. Here we employed phylodynamic models to quantify the evolutionary characteristics, spatiotemporal origins, and multi-host transmission dynamics of BTV across the globe. We inferred that goats are the ancestral hosts for BTV but are less likely to be important for cross-species transmission, sheep and cattle continue to be important for the transmission and maintenance of infection between other species. Our models pointed to China and India, countries with the highest population of goats, as the likely ancestral country for BTV emergence and dispersal worldwide over 1000 years ago. However, the increased diversification and dispersal of BTV coincided with the initiation of transcontinental livestock trade after the 1850s. Our analysis uncovered important epidemiological aspects of BTV that may guide future molecular surveillance of BTV.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78664-w" target="_blank" style="color:black; text-decoration:none">Defining intraspecific conservation units in the endemic Cuban Rock Iguanas ( Cyclura nubila nubila )</a></h1>
            <p>Kyle J. Shaney, L. Grisell Diaz-Ramirez, Sayra Espindola, Susette Castañeda-Rico, Vicente Berovides-Álvarez, Ella Vázquez-Domínguez</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-10</pre>
            <h2>Abstract</h2>
            <font size="4">Defining conservation units is an important step in species management and requires interpretation of the genetic diversity and ecological function of the taxon being considered. We used the endemic Cuban Rock Iguanas (Cyclura nubila nubila) as a model to highlight this challenge and examined patterns of its intraspecific genetic diversity across Cuba. We evaluated nuclear (microsatellite loci) and mitochondrial diversity across eight populations from the island and its off-shore cays, and applied the population genetics results for assignment of Management Unit (MU) status and Evolutionary Significant Units (ESUs) based on phylogeographic and time of divergence information. We identified at least six distinct Cuban Rock Iguana MUs, encompassing demographically isolated and genetically differentiated populations across Cuba, most with low effective population size, declining populations, and with high risk of inbreeding and genetic drift. Hence, each MU should be considered of urgent conservation priority. Given the key ecological seed dispersal role of C. n. nubila, the disappearance of any MU could trigger the loss of local ecological functional diversity and major negative impacts on their ecosystems. Two divergent ESUs were also identified, exhibiting an historical east–west geographic separation on Cuba. Based on a Caribbean phylogeographic assessment, our findings strengthen the conclusion that all geographically and evolutionarily differentiated Cyclura species and subspecies across the archipelago warrant ESU distinction.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41438-020-00441-7" target="_blank" style="color:black; text-decoration:none">Genomic consequences of apple improvement</a></h1>
            <p>Zoë Migicovsky, Kyle M. Gardner, Christopher Richards, C. Thomas Chao, Heidi R. Schwaninger, Gennaro Fazio, Gan-Yuan Zhong, Sean Myles</p>
            <pre style="font-family:timesnewroman">Journal: Horticulture Research    Published: 2021-01-01</pre>
            <h2>Abstract</h2>
            <font size="4">The apple (Malus domestica) is one of the world’s most commercially important perennial crops and its improvement has been the focus of human effort for thousands of years. Here, we genetically characterise over 1000 apple accessions from the United States Department of Agriculture (USDA) germplasm collection using over 30,000 single-nucleotide polymorphisms (SNPs). We confirm the close genetic relationship between modern apple cultivars and their primary progenitor species, Malus sieversii from Central Asia, and find that cider apples derive more of their ancestry from the European crabapple, Malus sylvestris, than do dessert apples. We determine that most of the USDA collection is a large complex pedigree: over half of the collection is interconnected by a series of first-degree relationships. In addition, 15% of the accessions have a first-degree relationship with one of the top 8 cultivars produced in the USA. With the exception of ‘Honeycrisp’, the top 8 cultivars are interconnected to each other via pedigree relationships. The cultivars ‘Golden Delicious’ and ‘Red Delicious’ were found to have over 60 first-degree relatives, consistent with their repeated use by apple breeders. We detected a signature of intense selection for red skin and provide evidence that breeders also selected for increased firmness. Our results suggest that Americans are eating apples largely from a single family tree and that the apple’s future improvement will benefit from increased exploitation of its tremendous natural genetic diversity.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77054-6" target="_blank" style="color:black; text-decoration:none">A core microbiota dominates a rich microbial diversity in the bovine udder and may indicate presence of dysbiosis</a></h1>
            <p>Davide Porcellato, Roger Meisal, Alberto Bombelli, Judith A. Narvhus</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-10</pre>
            <h2>Abstract</h2>
            <font size="4">The importance of the microbiome for bovine udder health is not well explored and most of the knowledge originates from research on mastitis. Better understanding of the microbial diversity inside the healthy udder of lactating cows might help to reduce mastitis, use of antibiotics and improve animal welfare. In this study, we investigated the microbial diversity of over 400 quarter milk samples from 60 cows sampled from two farms and on two different occasions during the same lactation period. Microbiota analysis was performed using amplicon sequencing of the 16S rRNA gene and over 1000 isolates were identified using MALDI-TOF MS. We detected a high abundance of two bacterial families, Corynebacteriaceae and Staphylococcaceae, which accounted for almost 50% of the udder microbiota of healthy cows and were detected in all the cow udders and in more than 98% of quarter milk samples. A strong negative correlation between these bacterial families was detected indicating a possible competition. The overall composition of the udder microbiota was highly diverse and significantly different between cows and between quarter milk samples from the same cow. Furthermore, we introduced a novel definition of a dysbiotic quarter at individual cow level, by analyzing the milk microbiota, and a high frequency of dysbiotic quarter samples were detected distributed among the farms and the samples. These results emphasize the importance of deepening the studies of the bovine udder microbiome to elucidate its role in udder health.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-79143-y" target="_blank" style="color:black; text-decoration:none">Microbial community composition in the rhizosphere of Larix decidua under different light regimes with additional focus on methane cycling microorganisms</a></h1>
            <p>Nadine Praeg, Paul Illmer</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-18</pre>
            <h2>Abstract</h2>
            <font size="4">Microbial community and diversity in the rhizosphere is strongly influenced by biotic and/or abiotic factors, like root exudates, nutrient availability, edaphon and climate. Here we report on the microbial diversity within the rhizosphere of Larix decidua, a dominant tree species in the Alps, as compared with the microbiome within the surrounding soil. We describe how increased light intensity influenced the rhizobiome and put emphasize on methane cycling microorganisms. Microbial taxa were classified into 26 bacterial, 4 archaeal and 6 fungal phyla revealing significant differences between bulk and rhizosphere soils. The dominant prokaryotic phyla were Proteobacteria, Acidobacteria, Actinobacteria (both, rhizosphere and bulk soil) and Bacteroidetes (rhizosphere soil only) and dominant fungal phyla in both fractions included Ascomycota and Basidiomycota. The rhizosphere community was indicated by Suillus sp., plant growth-promoting bacteria and Candidatus Saccharibacteria. Predicted genes in membrane transport and carbohydrate metabolism were significantly more abundant in rhizosphere soils while genes connected with energy metabolisms and cell motility increased in bulk soils. Dominant methanotrophic microorganisms were Upland Soil Cluster (USC) α methanotrophs, Methylogaea spp. and Methylosinus spp., while most methanogens belonged to Methanomassiliicoccales. The overall abundance of methanotrophs distinctly increased in the rhizosphere but to a very different species-specific extent. The increased light intensity only led to minor changes in the rhizobiome, nevertheless a couple of indicator species (e.g. Pseudomonas sp.) for intensified light conditions were established.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78988-7" target="_blank" style="color:black; text-decoration:none">Altered diversity and composition of gut microbiota in Wilson's disease</a></h1>
            <p>Xiangsheng Cai, Lin Deng, Xiaogui Ma, Yusheng Guo, Zhiting Feng, Minqi Liu, Yubin Guan, Yanting Huang, Jianxin Deng, Hongwei Li, Hong Sang, Fang Liu, Xiaorong Yang</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-11</pre>
            <h2>Abstract</h2>
            <font size="4">Wilson’s disease (WD) is an autosomal recessive inherited disorder of chronic copper toxicosis with high mortality and disability. Recent evidence suggests a correlation between dysbiosis in gut microbiome and multiple diseases such as genetic and metabolic disease. However, the impact of intestinal microbiota polymorphism in WD have not been fully elaborated and need to be explore for seeking some microbiota benefit for WD patients. In this study, the 16S rRNA sequencing was performed on fecal samples from 14 patients with WD and was compared to the results from 16 healthy individuals. The diversity and composition of the gut microbiome in the WD group were significantly lower than those in healthy individuals. The WD group presented unique richness of Gemellaceae, Pseudomonadaceae and Spirochaetaceae at family level, which were hardly detected in healthy controls. The WD group had a markedly lower abundance of Actinobacteria, Firmicutes and Verrucomicrobia, and a higher abundance of Bacteroidetes, Proteobacteria, Cyanobacteria and Fusobacteria than that in healthy individuals. The Firmicutes to Bacteroidetes ratio in the WD group was significantly lower than that of healthy control. In addition, the functional profile of the gut microbiome from WD patients showed a lower abundance of bacterial groups involved in the host immune and metabolism associated systems pathways such as transcription factors and ABC-type transporters, compared to healthy individuals. These results implied dysbiosis of gut microbiota may be influenced by the host metabolic disorders of WD, which may provide a new understanding of the pathogenesis and new possible therapeutic targets for WD.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-75947-0" target="_blank" style="color:black; text-decoration:none">Light source optimization for partially coherent holographic displays with consideration of speckle contrast, resolution, and depth of field</a></h1>
            <p>Seungjae Lee, Dongyeon Kim, Seung-Woo Nam, Byounghyo Lee, Jaebum Cho, Byoungho Lee</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-02</pre>
            <h2>Abstract</h2>
            <font size="4">Speckle reduction is an important topic in holographic displays as speckles not only reduce signal-to-noise ratio but also possess an eye-safety issue. Despite thorough exploration of speckle reduction methods using partially coherent light sources, the trade-off involved by the partial coherence has not been thoroughly discussed. Here, we introduce theoretical models that quantify the effects of partial coherence on the resolution and the speckle contrast. The theoretical models allow us to find an optimal light source that maximizes the speckle reduction while minimizing the decline of the other terms. We implement benchtop prototypes of partially coherent holographic displays using the optimal light source, and verify the theoretical models via simulation and experiment. We also present a criterion to evaluate the depth of field in partially coherent holographic displays. We conclude with a discussion about approximations and limitations inherent in the theoretical models.</font>
        </div>
        <br><hr>
    
    </body>
</html>
    
