
<!DOCTYPE HTML>
<html>
    <head>
        <meta charset="utf-8">
        <title>Nature, Keyword: diversity</title>
    </head>
    <body>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-20240-x" target="_blank" style="color:black; text-decoration:none">Longitudinal dynamics of gut bacteriome, mycobiome and virome after fecal microbiota transplantation in graft-versus-host disease</a></h1>
            <p>Fen Zhang, Tao Zuo, Yun Kit Yeoh, Frankie W. T. Cheng, Qin Liu, Whitney Tang, Kitty C. Y. Cheung, Keli Yang, Chun Pan Cheung, Chow Chung Mo, Mamie Hui, Francis K. L. Chan, Chi-Kong Li, Paul K. S. Chan, Siew C. Ng</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Fecal microbiota transplant (FMT) has emerged as a potential treatment for severe colitis associated with graft-versus-host disease (GvHD) following hematopoietic stem cell transplant. Bacterial engraftment from FMT donor to recipient has been reported, however the fate of fungi and viruses after FMT remains unclear. Here we report longitudinal dynamics of the gut bacteriome, mycobiome and virome in a teenager with GvHD after receiving four doses of FMT at weekly interval. After serial FMTs, the gut bacteriome, mycobiome and virome of the patient differ from compositions before FMT with variable temporal dynamics. Diversity of the gut bacterial community increases after each FMT. Gut fungal community initially shows expansion of several species followed by a decrease in diversity after multiple FMTs. In contrast, gut virome community varies substantially over time with a stable rise in diversity. The bacterium, Corynebacterium jeikeium, and Torque teno viruses, decrease after FMTs in parallel with an increase in the relative abundance of Caudovirales bacteriophages. Collectively, FMT may simultaneously impact on the various components of the gut microbiome with distinct effects. Fecal microbiota transplant (FMT) is emerging as a potential treatment for graft-versus-host disease (GvHD). Here, the authors examine temporal dynamics of the bacteriome, mycobiome, and virome of a patient with GvHD who received multiple FMTs.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-76917-2" target="_blank" style="color:black; text-decoration:none">Different response of the taxonomic, phylogenetic and functional diversity of birds to forest fragmentation</a></h1>
            <p>Michał Bełcik, Magdalena Lenda, Tatsuya Amano, Piotr Skórka</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-23</pre>
            <h2>Abstract</h2>
            <font size="4">Habitat fragmentation is considered as major threat to biodiversity worldwide. Biodiversity can be described as taxonomic, functional and phylogenetic diversity. However, the effect of forest fragmentation on taxonomic, phylogenetic and functional diversity is barely understood. We compare the response of taxonomic (species richness), phylogenetic and functional diversity of birds to forest fragmentation. We hypothesised that with increasing forest patch isolation and/or decreasing patch size the diversity of birds decreases but only if certain thresholds of fragmentation metrics are reached. Specifically, we hypothesized that out of the three diversity components the taxonomic diversity is the most sensitive to forest fragmentation, which means that it starts declining at larger patch size and higher connectivity values than phylogenetic and functional diversity do. We compared the three biodiversity metrics of central European bird species in a large set of forest patches located in an agricultural landscape. General additive modeling and segmented regression were used in analyses. Habitat fragmentation differentially affected studied biodiversity metrics. Bird taxonomic diversity was the most responsive towards changes in fragmentation. We observed an increase in taxonomic diversity with increasing patch area, which then stabilized after reaching certain patch size. Functional diversity turned out to be the least responsive to the fragmentation metrics and forest stand characteristics. It decreased linearly with the decreasing isolation of forest patches. Apart from the habitat fragmentation, bird taxonomic diversity but not phylogenetic diversity was positively associated with forest stand age. The lower share of dominant tree species, the highest taxonomic diversity was. While preserving a whole spectrum of forests (in terms of age, fragmentation and size) is important from the biodiversity perspective, forest bird species might need large, intact, old-growth forests. Since the large and intact forest becomes scarcer, our study underscore their importance for the preservation of forest specialist species.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-19228-4" target="_blank" style="color:black; text-decoration:none">Local community assembly mechanisms shape soil bacterial β diversity patterns along a latitudinal gradient</a></h1>
            <p>Xiao Zhang, Shirong Liu, Jingxin Wang, Yongtao Huang, Zachary Freedman, Shenglei Fu, Kuan Liu, Hui Wang, Xiangzhen Li, Minjie Yao, Xiaojing Liu, Jamie Schuler</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-10-27</pre>
            <h2>Abstract</h2>
            <font size="4">Biodiversity patterns across geographical gradients could result from regional species pool and local community assembly mechanisms. However, little has been done to separate the effects of local ecological mechanisms from variation in the regional species pools on bacterial diversity patterns. In this study, we compare assembly mechanisms of soil bacterial communities in 660 plots from 11 regions along a latitudinal gradient in eastern China with highly divergent species pools. Our results show that β diversity does not co-vary with γ diversity, and local community assembly mechanisms appear to explain variation in β diversity patterns after correcting for variation in regional species pools. The variation in environmental conditions along the latitudinal gradient accounts for the variation in β diversity through mediating the strength of heterogeneous selection. In conclusion, our study clearly illustrates the importance of local community assembly processes in shaping geographical patterns of soil bacterial β diversity. The relative importance of regional species pool and local assembly processes as drivers of beta diversity is unclear. Here the authors investigate soil bacterial diversity patterns along a 3700-km latitudinal gradient in Chinese forests, finding that community assembly processes differ based on environmental heterogeneity.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78736-x" target="_blank" style="color:black; text-decoration:none">Urban fragmentation leads to lower floral diversity, with knock-on impacts on bee biodiversity</a></h1>
            <p>Panagiotis Theodorou, Sarah-Christine Herbst, Belinda Kahnt, Patricia Landaverde-González, Lucie M. Baltz, Julia Osterman, Robert J. Paxton</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-10</pre>
            <h2>Abstract</h2>
            <font size="4">Bees and flowering plants are two closely interacting groups of organisms. Habitat loss and fragmentation associated with urbanisation are major threats to both partners. Yet how and why bee and floral richness and diversity co-vary within the urban landscape remain unclear. Here, we sampled bees and flowering plants in urban green spaces to investigate how bee and flowering plant species richness, their phylogenetic diversity and pollination-relevant functional trait diversity influence each other in response to urban fragmentation. As expected, bee abundance and richness were positively related to flowering plant richness, with bee body size (but not bee richness and diversity) increasing with nectar-holder depth of flowering plants. Causal modelling indicated that bottom-up effects dictated patterns of bee-flower relationships, with urban fragmentation diminishing flowering plants richness and thereby indirectly reducing bee species richness and abundance. The close relationship between bees and flowering plants highlights the risks of their parallel declines in response to land-use change within the urban landscape.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00855-0" target="_blank" style="color:black; text-decoration:none">Coinfection and infection duration shape how pathogens affect the African buffalo gut microbiota</a></h1>
            <p>Kate A. Sabey, Se Jin Song, Anna Jolles, Rob Knight, Vanessa O. Ezenwa</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2020-12-17</pre>
            <h2>Abstract</h2>
            <font size="4">Changes in the gut microbiota during pathogen infection are often predicted to influence disease outcomes. However, studies exploring whether pathogens induce microbiota shifts have yielded inconsistent results. This suggests that variation in infection, rather than the presence of infection alone, might shape pathogen–microbiota relationships. For example, most hosts are coinfected with multiple pathogens simultaneously, and hosts vary in how long they are infected, which may amplify or diminish microbial shifts expected in response to a focal pathogen. We used a longitudinal anthelmintic treatment study of free-ranging African buffalo (Syncerus caffer) to examine whether (i) coinfection with bovine tuberculosis (Mycobacterium bovis, TB) and gastrointestinal nematodes, and (ii) the duration of TB infection, modified effects of single pathogens on the gut microbiota. By accounting for the interaction between TB and nematodes, we found that coinfection affected changes in microbial abundance associated with single infections. Furthermore, the duration of TB infection predicted more microbiota variation than the presence of TB. Importantly, coinfection and infection duration had nearly as much influence on microbial patterns as demographic and environmental factors commonly examined in microbiota research. These findings demonstrate that acknowledging infection heterogeneities may be crucial to understanding relationships between pathogens and the gut microbiota.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00842-5" target="_blank" style="color:black; text-decoration:none">Distinct ecotypes within a natural haloarchaeal population enable adaptation to changing environmental conditions without causing population sweeps</a></h1>
            <p>Tomeu Viver, Roth E. Conrad, Luis H. Orellana, Mercedes Urdiain, José E. González-Pastor, Janet K. Hatt, Rudolf Amann, Josefa Antón, Konstantinos T. Konstantinidis, Ramon Rosselló-Móra</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2020-12-20</pre>
            <h2>Abstract</h2>
            <font size="4">Microbial communities thriving in hypersaline brines of solar salterns are highly resistant and resilient to environmental changes, and salinity is a major factor that deterministically influences community structure. Here, we demonstrate that this resilience occurs even after rapid osmotic shocks caused by a threefold change in salinity (a reduction from 34 to 12% salts) leading to massive amounts of archaeal cell lysis. Specifically, our temporal metagenomic datasets identified two co-occurring ecotypes within the most dominant archaeal population of the brines Haloquadratum walsbyi that exhibited different salt concentration preferences. The dominant ecotype was generally more abundant and occurred in high-salt conditions (34%); the low abundance ecotype always co-occurred but was enriched at salinities around 20% or lower and carried unique gene content related to solute transport and gene regulation. Despite their apparent distinct ecological preferences, the ecotypes did not outcompete each other presumably due to weak functional differentiation between them. Further, the osmotic shock selected for a temporal increase in taxonomic and functional diversity at both the Hqr. walsbyi population and whole-community levels supporting the specialization-disturbance hypothesis, that is, the expectation that disturbance favors generalists. Altogether, our results provide new insights into how intraspecies diversity is maintained in light of substantial gene-content differences and major environmental perturbations.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-76658-2" target="_blank" style="color:black; text-decoration:none">Diversity begets diversity in mammal species and human cultures</a></h1>
            <p>Marcus J. Hamilton, Robert S. Walker, Christopher P. Kempes</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-12</pre>
            <h2>Abstract</h2>
            <font size="4">Across the planet the biogeographic distribution of human cultural diversity tends to correlate positively with biodiversity. In this paper we focus on the biogeographic distribution of mammal species and human cultural diversity. We show that not only are these forms of diversity similarly distributed in space, but they both scale superlinearly with environmental production. We develop theory that explains that as environmental productivity increases the ecological kinetics of diversity increases faster than expected because more complex environments are also more interactive. Using biogeographic databases of the global distributions of mammal species and human cultures we test a series of hypotheses derived from this theory and find support for each. For both mammals and cultures, we show that (1) both forms of diversity increase exponentially with ecological kinetics; (2) the kinetics of diversity is faster than the kinetics of productivity; (3) diversity scales superlinearly with environmental productivity; and (4) the kinetics of diversity is faster in increasingly productive environments. This biogeographic convergence is particularly striking because while the dynamics of biological and cultural evolution may be similar in principle the underlying mechanisms and time scales are very different. However, a common currency underlying all forms of diversity is ecological kinetics; the temperature-dependent fluxes of energy and biotic interactions that sustain all forms of life at all levels of organization. Diversity begets diversity in mammal species and human cultures because ecological kinetics drives superlinear scaling with environmental productivity.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-19989-y" target="_blank" style="color:black; text-decoration:none">Using metacommunity ecology to understand environmental metabolomes</a></h1>
            <p>Robert E. Danczak, Rosalie K. Chu, Sarah J. Fansler, Amy E. Goldman, Emily B. Graham, Malak M. Tfaily, Jason Toyoda, James C. Stegen</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-12-11</pre>
            <h2>Abstract</h2>
            <font size="4">Environmental metabolomes are fundamentally coupled to microbially-linked biogeochemical processes within ecosystems. However, significant gaps exist in our understanding of their spatiotemporal organization, limiting our ability to uncover transferrable principles and predict ecosystem function. We propose that a theoretical paradigm, which integrates concepts from metacommunity ecology, is necessary to reveal underlying mechanisms governing metabolomes. We call this synthesis between ecology and metabolomics ‘meta-metabolome ecology’ and demonstrate its utility using a mass spectrometry dataset. We developed three relational metabolite dendrograms using molecular properties and putative biochemical transformations and performed ecological null modeling. Based upon null modeling results, we show that stochastic processes drove molecular properties while biochemical transformations were structured deterministically. We further suggest that potentially biochemically active metabolites were more deterministically assembled than less active metabolites. Understanding variation in the influences of stochasticity and determinism provides a way to focus attention on which meta-metabolomes and which parts of meta-metabolomes are most likely to be important to consider in mechanistic models. We propose that this paradigm will allow researchers to study the connections between ecological systems and their molecular processes in previously inaccessible detail. Despite growing interest in environmental metabolomics, we lack conceptual frameworks for considering how metabolites vary across space and time in ecological systems. Here, the authors apply (species) community assembly concepts to metabolomics data, offering a way forward in understanding the assembly of metabolite assemblages.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s42003-020-01551-7" target="_blank" style="color:black; text-decoration:none">Skin T cells maintain their diversity and functionality in the elderly</a></h1>
            <p>Hanako Koguchi-Yoshioka, Elena Hoffer, Stanley Cheuk, Yutaka Matsumura, Sa Vo, Petra Kjellman, Lucian Grema, Yosuke Ishitsuka, Yoshiyuki Nakamura, Naoko Okiyama, Yasuhiro Fujisawa, Manabu Fujimoto, Liv Eidsmo, Rachael A. Clark, Rei Watanabe</p>
            <pre style="font-family:timesnewroman">Journal: Communications Biology    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Recent studies have highlighted that human resident memory T cells (TRM) are functionally distinct from circulating T cells. Thus, it can be postulated that skin T cells age differently from blood-circulating T cells. We assessed T-cell density, diversity, and function in individuals of various ages to study the immunologic effects of aging on human skin from two different countries. No decline in the density of T cells was noted with advancing age, and the frequency of epidermal CD49a+ CD8 TRM was increased in elderly individuals regardless of ethnicity. T-cell diversity and antipathogen responses were maintained in the skin of elderly individuals but declined in the blood. Our findings demonstrate that in elderly individuals, skin T cells maintain their density, diversity, and protective cytokine production despite the reduced T-cell diversity and function in blood. Skin resident T cells may represent a long-lived, highly protective reservoir of immunity in elderly people. Koguchi-Yoshioka et al. demonstrate that functional activities and diversity of T cells are highly maintained in the skin of elderly people when compared with those in the blood regardless of the ethnicity of the participants or the experimental procedures. They also find the accumulation of epidermal T cells with maintained cytokine production in the elderly participants.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78300-7" target="_blank" style="color:black; text-decoration:none">Phytoplankton alpha diversity indices response the trophic state variation in hydrologically connected aquatic habitats in the Harbin Section of the Songhua River</a></h1>
            <p>Fanhua Meng, Zhenxiang Li, Lei Li, Feng Lu, Yan Liu, Xinxin Lu, Yawen Fan</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-07</pre>
            <h2>Abstract</h2>
            <font size="4">The relationship between biodiversity and ecological functioning is a central issue in freshwater ecology, but how this relationship is influenced by hydrological connectivity stress is still unknown. In this study we analyzed the dynamic of the phytoplankton alpha diversity indices and their relationships with trophic state in two hydrologically connected aquatic habitats (Jinhewan Wetland and Harbin Section of the Songhua River) in the Songhua River Basin in northeast China. We hypothesized that the phytoplankton alpha-diversity indices have the potential to provide a signal linking trophic state variation in hydrologically connected aquatic habitats. Our results showed the Cyanophyta and Bacillariophyta were abundant at most stations. T-test showed that phytoplankton alpha diversity indices varied significantly between rainy season and dry season. Trophic State Index recorded that a meso-trophic to eutrophic states of two connected habits during study period. Multivariate statistical analysis revealed that the dynamic of phytoplankton alpha diversity index was closely associated with trophic states change. Our result indicated that hydrological connectivity is a key factor influenced phytoplankton community assembly. In addition, it is beneficial to develop an integrated approach to appropriately describe and measure the trophic state variations of hydrologically connected aquatic habits in freshwater ecosystem.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-19793-8" target="_blank" style="color:black; text-decoration:none">Vitamin D metabolites and the gut microbiome in older men</a></h1>
            <p>Robert L. Thomas, Lingjing Jiang, John S. Adams, Zhenjiang Zech Xu, Jian Shen, Stefan Janssen, Gail Ackermann, Dirk Vanderschueren, Steven Pauwels, Rob Knight, Eric S. Orwoll, Deborah M. Kado</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-11-26</pre>
            <h2>Abstract</h2>
            <font size="4">The vitamin D receptor is highly expressed in the gastrointestinal tract where it transacts gene expression. With current limited understanding of the interactions between the gut microbiome and vitamin D, we conduct a cross-sectional analysis of 567 older men quantifying serum vitamin D metabolites using LC-MSMS and defining stool sub-Operational Taxonomic Units from16S ribosomal RNA gene sequencing data. Faith’s Phylogenetic Diversity and non-redundant covariate analyses reveal that the serum 1,25(OH)2D level explains 5% of variance in α-diversity. In β-diversity analyses using unweighted UniFrac, 1,25(OH)2D is the strongest factor assessed, explaining 2% of variance. Random forest analyses identify 12 taxa, 11 in the phylum Firmicutes, eight of which are positively associated with either 1,25(OH)2D and/or the hormone-to-prohormone [1,25(OH)2D/25(OH)D] “activation ratio.” Men with higher levels of 1,25(OH)2D and higher activation ratios, but not 25(OH)D itself, are more likely to possess butyrate producing bacteria that are associated with better gut microbial health. Here, the authors investigate associations of vitamin D metabolites with gut microbiome in a cross-sectional analysis of 567 elderly men enrolled in the Osteoporotic Fractures in Men (MrOS) Study and find larger alpha-diversity correlates with high 1,25(OH)2D and high 24,25(OH)2D and higher ratios of activation and catabolism.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-79130-3" target="_blank" style="color:black; text-decoration:none">Skin microbiome correlates with bioclimate and Batrachochytrium dendrobatidis infection intensity in Brazil’s Atlantic Forest treefrogs</a></h1>
            <p>Katharina Ruthsatz, Mariana L. Lyra, Carolina Lambertini, Anat M. Belasen, Thomas S. Jenkinson, Domingos da Silva Leite, C. Guilherme Becker, Célio F. B. Haddad, Timothy Y. James, Kelly R. Zamudio, Luís Felipe Toledo, Miguel Vences</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-18</pre>
            <h2>Abstract</h2>
            <font size="4">In Brazil’s Atlantic Forest (AF) biodiversity conservation is of key importance since the fungal pathogen Batrachochytrium dendrobatidis (Bd) has led to the rapid loss of amphibian populations here and worldwide. The impact of Bd on amphibians is determined by the host's immune system, of which the skin microbiome is a critical component. The richness and diversity of such cutaneous bacterial communities are known to be shaped by abiotic factors which thus may indirectly modulate host susceptibility to Bd. This study aimed to contribute to understanding the environment-host–pathogen interaction determining skin bacterial communities in 819 treefrogs (Anura: Hylidae and Phyllomedusidae) from 71 species sampled across the AF. We investigated whether abiotic factors influence the bacterial community richness and structure on the amphibian skin. We further tested for an association between skin bacterial community structure and Bd co-occurrence. Our data revealed that temperature, precipitation, and elevation consistently correlate with richness and diversity of the skin microbiome and also predict Bd infection status. Surprisingly, our data suggest a weak but significant positive correlation of Bd infection intensity and bacterial richness. We highlight the prospect of future experimental studies on the impact of changing environmental conditions associated with global change on environment-host–pathogen interactions in the AF.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78891-1" target="_blank" style="color:black; text-decoration:none">An altered microbiome in urban coyotes mediates relationships between anthropogenic diet and poor health</a></h1>
            <p>Scott Sugden, Dana Sanderson, Kyra Ford, Lisa Y. Stein, Colleen Cassady St. Clair</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-17</pre>
            <h2>Abstract</h2>
            <font size="4">Generalist species able to exploit anthropogenic food sources are becoming increasingly common in urban environments. Coyotes (Canis latrans) are one such urban generalist that now resides in cities across North America, where diseased or unhealthy coyotes are frequently reported in cases of human-wildlife conflict. Coyote health and fitness may be related to habitat use and diet via the gut microbiome, which has far-reaching effects on animal nutrition and physiology. In this study, we used stomach contents, stable isotope analysis, 16S rRNA gene amplicon sequencing, and measures of body condition to identify relationships among habitat use, diet, fecal microbiome composition, and health in urban and rural coyotes. Three distinct relationships emerged: (1) Urban coyotes consumed more anthropogenic food, which was associated with increased microbiome diversity, higher abundances of Streptococcus and Enterococcus, and poorer average body condition. (2) Conversely, rural coyotes harbored microbiomes rich in Fusobacteria, Sutterella, and Anaerobiospirillum, which were associated with protein-rich diets and improved body condition. (3) Diets rich in anthropogenic food were associated with increased abundances of Erysipelotrichiaceae, Lachnospiraceae, and Coriobacteriaceae, which correlated with larger spleens in urban coyotes. Urban coyotes also had an increased prevalence of the zoonotic parasite Echinococcus multilocularis, but there were no detectable connections between parasite infection and microbiome composition. Our results demonstrate how the consumption of carbohydrate-rich anthropogenic food by urban coyotes alters the microbiome to negatively affect body condition, with potential relationships to parasite susceptibility and conflict-prone behavior.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-79363-2" target="_blank" style="color:black; text-decoration:none">T-cell receptor repertoire of cytomegalovirus-specific cytotoxic T-cells after allogeneic stem cell transplantation</a></h1>
            <p>Takashi Toya, Ayumi Taguchi, Kazutaka Kitaura, Fumi Misumi, Yujiro Nakajima, Yuki Otsuka, Ryosuke Konuma, Hiroto Adachi, Atsushi Wada, Yuya Kishida, Tatsuya Konishi, Akihito Nagata, Yuta Yamada, Atsushi Marumo, Yuma Noguchi, Kota Yoshifuji, Junichi Mukae, Kyoko Inamoto, Aiko Igarashi, Yuho Najima, Takeshi Kobayashi, Kazuhiko Kakihana, Kazuteru Ohashi, Ryuji Suzuki, Takeshi Nagamatsu, Noriko Doki</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-17</pre>
            <h2>Abstract</h2>
            <font size="4">Cytomegalovirus (CMV) infection is a major complication during allogeneic stem cell transplantation (allo-SCT). However, mechanisms of adaptive immunity that drive this remain unclear. To define early immunological responses to CMV after transplantation, we using next-generation sequencing to examine the repertoire of T-cell receptors in CD8+/CMV pp65 tetramer+ cells (CMV-CTLs) in peripheral blood samples obtained from 16 allo-SCT recipients with HLA-A*24:02 at the time of CMV reactivation. In most patients, TCR beta repertoire of CMV-CTLs was highly skewed (median Inverse Simpson’s index: 1.595) and, 15 of 16 patients shared at least one TCR-beta clonotype with ≥ 2 patients. The shared TCRs were dominant in 12 patients and, two clonotypes were shared by about half of the patients. Similarity analysis showed that CDR3 sequences of shared TCRs were more similar than unshared TCRs. TCR beta repertoires of CMV-CTLs in 12 patients were also analyzed after 2–4 weeks to characterize the short-term dynamics of TCR repertoires. In ten patients, we observed persistence of prevailing clones. In the other two patients, TCR repertoires became more diverse, major clones declined, and new private clones subsequently emerged. These results provided the substantive clue to understand the immunological behavior against CMV reactivation after allo-SCT.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41586-020-2965-6" target="_blank" style="color:black; text-decoration:none">Crop asynchrony stabilizes food production</a></h1>
            <p>Lukas Egli, Matthias Schröter, Christoph Scherber, Teja Tscharntke, Ralf Seppelt</p>
            <pre style="font-family:timesnewroman">Journal: Nature    Published: 2020-12-09</pre>
            <h2>Abstract</h2>
            <font size="4"></font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00856-z" target="_blank" style="color:black; text-decoration:none">Morphological complexity affects the diversity of marine microbiomes</a></h1>
            <p>Matthew A. Lemay, Melissa Y. Chen, Florent Mazel, Katharine R. Hind, Samuel Starko, Patrick J. Keeling, Patrick T. Martone, Laura Wegener Parfrey</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2020-12-21</pre>
            <h2>Abstract</h2>
            <font size="4">Large eukaryotes support diverse communities of microbes on their surface—epibiota—that profoundly influence their biology. Alternate factors known to structure complex patterns of microbial diversity—host evolutionary history and ecology, environmental conditions and stochasticity—do not act independently and it is challenging to disentangle their relative effects. Here, we surveyed the epibiota from 38 sympatric seaweed species that span diverse clades and have convergent morphology, which strongly influences seaweed ecology. Host identity explains most of the variation in epibiont communities and deeper host phylogenetic relationships (e.g., genus level) explain a small but significant portion of epibiont community variation. Strikingly, epibiota community composition is significantly influenced by host morphology and epibiota richness increases with morphological complexity of the seaweed host. This effect is robust after controlling for phylogenetic non-independence and is strongest for crustose seaweeds. We experimentally validated the effect of host morphology by quantifying bacterial community assembly on latex sheets cut to resemble three seaweed morphologies. The patterns match those observed in our field survey. Thus, biodiversity increases with habitat complexity in host-associated microbial communities, mirroring patterns observed in animal communities. We suggest that host morphology and structural complexity are underexplored mechanisms structuring microbial communities.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78090-y" target="_blank" style="color:black; text-decoration:none">Linear infrastructure habitats increase landscape-scale diversity of plants but not of flower-visiting insects</a></h1>
            <p>Juliana Dániel-Ferreira, Riccardo Bommarco, Jörgen Wissman, Erik Öckinger</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-07</pre>
            <h2>Abstract</h2>
            <font size="4">Habitats along linear infrastructure, such as roads and electrical transmission lines, can have high local biodiversity. To determine whether these habitats also contribute to landscape-scale biodiversity, we estimated species richness, evenness and phylogenetic diversity of plant, butterfly and bumblebee communities in 32 km2 landscapes with or without power line corridors, and with contrasting areas of road verges. Landscapes with power line corridors had on average six more plant species than landscapes without power lines, but there was no such effect for butterflies and bumblebees. Plant communities displayed considerable evenness in species abundances both in landscapes with and without power lines and high and low road verge densities. We hypothesize that the higher number of plant species in landscapes with power line corridors is due to these landscapes having a higher extinction debt than the landscapes without power line corridors, such that plant diversity is declining slower in landscapes with power lines. This calls for targeted conservation actions in semi-natural grasslands within landscapes with power line corridors to maintain biodiversity and prevent imminent population extinctions.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41437-020-00387-5" target="_blank" style="color:black; text-decoration:none">Weak founder effects but significant spatial genetic imprint of recent contraction and expansion of European beech populations</a></h1>
            <p>Tonya A. Lander, Etienne K. Klein, Anne Roig, Sylvie Oddou-Muratorio</p>
            <pre style="font-family:timesnewroman">Journal: Heredity    Published: 2020-11-23</pre>
            <h2>Abstract</h2>
            <font size="4">Understanding the ecological and evolutionary processes occurring during species range shifts is important in the current context of global change. Here, we investigate the interplay between recent expansion, gene flow and genetic drift, and their consequences for genetic diversity and structure at landscape and local scales in European beech (Fagus sylvatica L.) On Mont Ventoux, South-Eastern France, we located beech forest refugia at the time of the most recent population minimum, ~150 years ago, and sampled 71 populations (2042 trees) in both refugia and expanding populations over an area of 15,000 ha. We inferred patterns of gene flow and genetic structure using 12 microsatellite markers. We identified six plots as originating from planting, rather than natural establishment, mostly from local genetic material. Comparing genetic diversity and structure in refugia versus recent populations did not support the existence of founder effects: heterozygosity (He = 0.667) and allelic richness (Ar = 4.298) were similar, and FST was low (0.031 overall). Still, significant spatial evidence of colonization was detected, with He increasing along the expansion front, while genetic differentiation from the entire pool (βWT) decreased. Isolation by distance was found in refugia but not in recently expanding populations. Our study indicates that beech capacities for colonization and gene flow were sufficient to preserve genetic diversity despite recent forest contraction and expansion. Because beech has long distance pollen and seed dispersal, these results illustrate a ‘best case scenario’ for the maintenance of high genetic diversity and adaptive potential under climate-change-related range change.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77382-7" target="_blank" style="color:black; text-decoration:none">Warming and leaf litter functional diversity, not litter quality, drive decomposition in a freshwater ecosystem</a></h1>
            <p>Gustavo H. Migliorini, Gustavo Q. Romero</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-23</pre>
            <h2>Abstract</h2>
            <font size="4">Environment, litter composition and decomposer community are known to be the main drivers of litter decomposition in aquatic ecosystems. However, it remains unclear whether litter quality or functional diversity prevails under warming conditions. Using tank bromeliad ecosystems, we evaluated the combined effects of warming, litter quality and litter functional diversity on the decomposition process. We also assessed the contribution of macroinvertebrates and microorganisms in explaining litter decomposition patterns using litter bags made with different mesh sizes. Our results showed that litter decomposition was driven by litter functional diversity and was increasingly higher under warming, in both mesh sizes. Decomposition was explained by increasing litter dissimilarities in C and N. Our results highlight the importance of considering different aspects of litter characteristics (e.g., quality and functional diversity) in order to predict the decomposition process in freshwater ecosystems. Considering the joint effect of warming and litter traits aspects allow a more refined understanding of the underlying mechanisms of climate change and biodiversity shifts effects on ecosystem functioning.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41591-020-1125-8" target="_blank" style="color:black; text-decoration:none">Single-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma</a></h1>
            <p>Ruiping Wang, Minghao Dang, Kazuto Harada, Guangchun Han, Fang Wang, Melissa Pool Pizzi, Meina Zhao, Ghia Tatlonghari, Shaojun Zhang, Dapeng Hao, Yang Lu, Shuangtao Zhao, Brian D. Badgwell, Mariela Blum Murphy, Namita Shanbhag, Jeannelyn S. Estrella, Sinchita Roy-Chowdhuri, Ahmed Adel Fouad Abdelhakeem, Yuanxin Wang, Guang Peng, Samir Hanash, George A. Calin, Xingzhi Song, Yanshuo Chu, Jianhua Zhang, Mingyao Li, Ken Chen, Alexander J. Lazar, Andrew Futreal, Shumei Song, Jaffer A. Ajani, Linghua Wang</p>
            <pre style="font-family:timesnewroman">Journal: Nature Medicine    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Intratumoral heterogeneity (ITH) is a fundamental property of cancer; however, the origins of ITH remain poorly understood. We performed single-cell transcriptome profiling of peritoneal carcinomatosis (PC) from 15 patients with gastric adenocarcinoma (GAC), constructed a map of 45,048 PC cells, profiled the transcriptome states of tumor cell populations, incisively explored ITH of malignant PC cells and identified significant correlates with patient survival. The links between tumor cell lineage/state compositions and ITH were illustrated at transcriptomic, genotypic, molecular and phenotypic levels. We uncovered the diversity in tumor cell lineage/state compositions in PC specimens and defined it as a key contributor to ITH. Single-cell analysis of ITH classified PC specimens into two subtypes that were prognostically independent of clinical variables, and a 12-gene prognostic signature was derived and validated in multiple large-scale GAC cohorts. The prognostic signature appears fundamental to GAC carcinogenesis and progression and could be practical for patient stratification. Single-cell analysis of gastric cancer samples tracks the cell of origin of metastatic lesions and identifies an independent prognostic signature of the clinical outcome.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-20337-3" target="_blank" style="color:black; text-decoration:none">The patterns of deleterious mutations during the domestication of soybean</a></h1>
            <p>Myung-Shin Kim, Roberto Lozano, Ji Hong Kim, Dong Nyuk Bae, Sang-Tae Kim, Jung-Ho Park, Man Soo Choi, Jaehyun Kim, Hyun-Choong Ok, Soo-Kwon Park, Michael A. Gore, Jung-Kyung Moon, Soon-Chun Jeong</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Globally, soybean is a major protein and oil crop. Enhancing our understanding of the soybean domestication and improvement process helps boost genomics-assisted breeding efforts. Here we present a genome-wide variation map of 10.6 million single-nucleotide polymorphisms and 1.4 million indels for 781 soybean individuals which includes 418 domesticated (Glycine max), 345 wild (Glycine soja), and 18 natural hybrid (G. max/G. soja) accessions. We describe the enhanced detection of 183 domestication-selective sweeps and the patterns of putative deleterious mutations during domestication and improvement. This predominantly selfing species shows 7.1% reduction of overall deleterious mutations in domesticated soybean relative to wild soybean and a further 1.4% reduction from landrace to improved accessions. The detected domestication-selective sweeps also show reduced levels of deleterious alleles. Importantly, genotype imputation with this resource increases the mapping resolution of genome-wide association studies for seed protein and oil traits in a soybean diversity panel. The accumulation of recombination events in selfing species may lead to a rapid fixation of both beneficial and deleterious mutations. Here, the authors resequence 781 soybean accessions, show purging of deleterious mutation during domestication, and report genome-wide associations for seed protein and oil traits.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41586-020-3003-4" target="_blank" style="color:black; text-decoration:none">Impacts of speciation and extinction measured by an evolutionary decay clock</a></h1>
            <p>Jennifer F. Hoyal Cuthill, Nicholas Guttenberg, Graham E. Budd</p>
            <pre style="font-family:timesnewroman">Journal: Nature    Published: 2020-12-09</pre>
            <h2>Abstract</h2>
            <font size="4">The hypothesis that destructive mass extinctions enable creative evolutionary radiations (creative destruction) is central to classic concepts of macroevolution1,2. However, the relative impacts of extinction and radiation on the co-occurrence of species have not been directly quantitatively compared across the Phanerozoic eon. Here we apply machine learning to generate a spatial embedding (multidimensional ordination) of the temporal co-occurrence structure of the Phanerozoic fossil record, covering 1,273,254 occurrences in the Paleobiology Database for 171,231 embedded species. This facilitates the simultaneous comparison of macroevolutionary disruptions, using measures independent of secular diversity trends. Among the 5% most significant periods of disruption, we identify the ‘big five’ mass extinction events2, seven additional mass extinctions, two combined mass extinction–radiation events and 15 mass radiations. In contrast to narratives that emphasize post-extinction radiations1,3, we find that the proportionally most comparable mass radiations and extinctions (such as the Cambrian explosion and the end-Permian mass extinction) are typically decoupled in time, refuting any direct causal relationship between them. Moreover, in addition to extinctions4, evolutionary radiations themselves cause evolutionary decay (modelled co-occurrence probability and shared fraction of species between times approaching zero), a concept that we describe as destructive creation. A direct test of the time to over-threshold macroevolutionary decay4 (shared fraction of species between two times ≤ 0.1), counted by the decay clock, reveals saw-toothed fluctuations around a Phanerozoic mean of 18.6 million years. As the Quaternary period began at a below-average decay-clock time of 11 million years, modern extinctions further increase life’s decay-clock debt. Analysis of data on species co-occurrence in the Paleobiology Database using a new machine learning algorithm reveals that mass extinctions and mass radiations are not coupled in evolutionary history.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41564-020-00811-w" target="_blank" style="color:black; text-decoration:none">Trace gas oxidizers are widespread and active members of soil microbial communities</a></h1>
            <p>Sean K. Bay, Xiyang Dong, James A. Bradley, Pok Man Leung, Rhys Grinter, Thanavit Jirapanjawat, Stefan K. Arndt, Perran L. M. Cook, Douglas E. LaRowe, Philipp A. Nauer, Eleonora Chiri, Chris Greening</p>
            <pre style="font-family:timesnewroman">Journal: Nature Microbiology    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Soil microorganisms globally are thought to be sustained primarily by organic carbon sources. Certain bacteria also consume inorganic energy sources such as trace gases, but they are presumed to be rare community members, except within some oligotrophic soils. Here we combined metagenomic, biogeochemical and modelling approaches to determine how soil microbial communities meet energy and carbon needs. Analysis of 40 metagenomes and 757 derived genomes indicated that over 70% of soil bacterial taxa encode enzymes to consume inorganic energy sources. Bacteria from 19 phyla encoded enzymes to use the trace gases hydrogen and carbon monoxide as supplemental electron donors for aerobic respiration. In addition, we identified a fourth phylum (Gemmatimonadota) potentially capable of aerobic methanotrophy. Consistent with the metagenomic profiling, communities within soil profiles from diverse habitats rapidly oxidized hydrogen, carbon monoxide and to a lesser extent methane below atmospheric concentrations. Thermodynamic modelling indicated that the power generated by oxidation of these three gases is sufficient to meet the maintenance needs of the bacterial cells capable of consuming them. Diverse bacteria also encode enzymes to use trace gases as electron donors to support carbon fixation. Altogether, these findings indicate that trace gas oxidation confers a major selective advantage in soil ecosystems, where availability of preferred organic substrates limits microbial growth. The observation that inorganic energy sources may sustain most soil bacteria also has broad implications for understanding atmospheric chemistry and microbial biodiversity in a changing world. A combination of metagenomic analyses, thermodynamic modelling and in situ measurements of gas fluxes shows that a large fraction of soil bacteria can use inorganic energy sources, such as the trace gases hydrogen and carbon monoxide, for growth and persistence.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78413-z" target="_blank" style="color:black; text-decoration:none">Metagenomic analysis reveals rapid development of soil biota on fresh volcanic ash</a></h1>
            <p>Hokyung Song, Dorsaf Kerfahi, Koichi Takahashi, Sophie L. Nixon, Binu M. Tripathi, Hyoki Kim, Ryunosuke Tateno, Jonathan Adams</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-08</pre>
            <h2>Abstract</h2>
            <font size="4">Little is known of the earliest stages of soil biota development of volcanic ash, and how rapidly it can proceed. We investigated the potential for soil biota development during the first 3 years, using outdoor mesocosms of sterile, freshly fallen volcanic ash from the Sakurajima volcano, Japan. Mesocosms were positioned in a range of climates across Japan and compared over 3 years, against the developed soils of surrounding natural ecosystems. DNA was extracted from mesocosms and community composition assessed using 16S rRNA gene sequences. Metagenome sequences were obtained using shotgun metagenome sequencing. While at 12 months there was insufficient DNA for sequencing, by 24 months and 36 months, the ash-soil metagenomes already showed a similar diversity of functional genes to the developed soils, with a similar range of functions. In a surprising contrast with our hypotheses, we found that the developing ash-soil community already showed a similar gene function diversity, phylum diversity and overall relative abundances of kingdoms of life when compared to developed forest soils. The ash mesocosms also did not show any increased relative abundance of genes associated with autotrophy (rbc, coxL), nor increased relative abundance of genes that are associated with acquisition of nutrients from abiotic sources (nifH). Although gene identities and taxonomic affinities in the developing ash-soils are to some extent distinct from the natural vegetation soils, it is surprising that so many of the key components of a soil community develop already by the 24-month stage. In this system, however, rapid development may be facilitated by the relatively moderate pH of the Sakurajima ash, proximity of our mesocosms to propagule sources, and the rapid establishment of a productive bryophyte and lichen layer on the surface. Ash from other volcanoes richer in acids or more distant from propagule sources could show a different pattern and slower soil biota development.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77256-y" target="_blank" style="color:black; text-decoration:none">Diet induces parallel changes to the gut microbiota and problem solving performance in a wild bird</a></h1>
            <p>Gabrielle L. Davidson, Niamh Wiley, Amy C. Cooke, Crystal N. Johnson, Fiona Fouhy, Michael S. Reichert, Iván de la Hera, Jodie M. S. Crane, Ipek G. Kulahci, R. Paul Ross, Catherine Stanton, John L. Quinn</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-27</pre>
            <h2>Abstract</h2>
            <font size="4">The microbial community in the gut is influenced by environmental factors, especially diet, which can moderate host behaviour through the microbiome-gut-brain axis. However, the ecological relevance of microbiome-mediated behavioural plasticity in wild animals is unknown. We presented wild-caught great tits (Parus major) with a problem-solving task and showed that performance was weakly associated with variation in the gut microbiome. We then manipulated the gut microbiome by feeding birds one of two diets that differed in their relative levels of fat, protein and fibre content: an insect diet (low content), or a seed diet (high content). Microbial communities were less diverse among individuals given the insect compared to those on the seed diet. Individuals were less likely to problem-solve after being given the insect diet, and the same microbiota metrics that were altered as a consequence of diet were also those that correlated with variation in problem solving performance. Although the effect on problem-solving behaviour could have been caused by motivational or nutritional differences between our treatments, our results nevertheless raise the possibility that dietary induced changes in the gut microbiota could be an important mechanism underlying individual behavioural plasticity in wild populations.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41522-020-00171-7" target="_blank" style="color:black; text-decoration:none">The human respiratory tract microbial community structures in healthy and cystic fibrosis infants</a></h1>
            <p>Marie-Madlen Pust, Lutz Wiehlmann, Colin Davenport, Isa Rudolf, Anna-Maria Dittrich, Burkhard Tümmler</p>
            <pre style="font-family:timesnewroman">Journal: npj Biofilms and Microbiomes    Published: 2020-12-15</pre>
            <h2>Abstract</h2>
            <font size="4">The metagenome development of the human respiratory tract was investigated by shotgun metagenome metagenomic sequencing of cough swabs from healthy children and children with cystic fibrosis (CF) between 3 weeks and 6 years of age. A healthy microbial community signature was associated with increased absolute abundances in terms of bacterial–human cell ratios of core and rare species across all age groups, with a higher diversity of rare species and a tightly interconnected species co-occurrence network, in which individual members were found in close proximity to each other and negative correlations were absent. Even without typical CF pathogens, the CF infant co-occurrence network was found to be less stable and prone to fragmentation due to fewer connections between species, a higher number of bridging species and the presence of negative species correlations. Detection of low-abundant DNA of the CF hallmark pathogen Pseudomonas aeruginosa was neither disease- nor age-associated in our cohort. Healthy and CF children come into contact with P. aeruginosa on a regular basis and from early on.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77480-6" target="_blank" style="color:black; text-decoration:none">Microsatellite and mitochondrial DNA analyses unveil the genetic structure of native sheep breeds from three major agro-ecological regions of India</a></h1>
            <p>Rekha Sharma, Sonika Ahlawat, Himani Sharma, Priyanka Sharma, Poonam Panchal, Reena Arora, M S Tantia</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-24</pre>
            <h2>Abstract</h2>
            <font size="4">Sheep farming has been fundamental to many civilizations in the world and is practiced in India since antiquity. Several thousand years of adaptation to local environmental conditions and selective breeding have evolved 44 sheep breeds in India. They are paramount in terms of economic, scientific, and cultural heritage. Genetic characterization information is imperative for sustainable utilization and conservation of ovine heritage. In this study, the genetic diversity, differentiation, and structure of 11 indigenous sheep breeds from three different agro-ecological zones of India were explored with genomic microsatellite loci and mitochondrial DNA (D loop). The estimated diversity parameters indicated that populations retained high levels of genetic diversity (Na = 8.27 ± 0.17; Ho = 0.65 ± 0.01), which provides an optimistic viewpoint for their survival. However, significant inbreeding was also observed in the nine populations. Moderate genetic differentiation existed among the groups (FST = 0.129 ± 0.012), and most likely clusters existing in the dataset are seven. Phylogenetic clustering was in line with the geographical locations of sheep populations. Mitochondrial sequences revealed high haplotype diversity with the existence of maternal haplogroups A, B, and C, and signals of population expansion. Decreased genetic diversity and unique maternal lineage (C) in endangered Tibetan and Bonpala sheep breed, warrant their immediate scientific management. Overall, the quantitative data reported here on the extant variability, and genetic relationships among the Indian sheep breeds, provide critically important inputs that will be valuable for the decision-making process on their management, both for the conservation of endangered breeds, and formulation of breeding programs to check genetic erosion.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77313-6" target="_blank" style="color:black; text-decoration:none">Feed additives for the control of post-weaning Streptococcus suis disease and the effect on the faecal and nasal microbiota</a></h1>
            <p>Florencia Correa-Fiz, Carlos Neila-Ibáñez, Sergio López-Soria, Sebastian Napp, Blanca Martinez, Laia Sobrevia, Simon Tibble, Virginia Aragon, Lourdes Migura-Garcia</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-23</pre>
            <h2>Abstract</h2>
            <font size="4">Medicated feed is a common strategy to control the occurrence of Streptococcus suis disease in swine production, but feed additives may constitute an alternative to metaphylaxis. In a farm with post-weaning S. suis disease, the following additives were tested: lysozyme (Lys), medium chain fatty acids plus lysozyme (FA + Lys), FA plus a natural anti-inflammatory (FA + antiinf) and amoxicillin (Amox). During the course of the study, FA + antiinf and Amox groups showed lower prevalence of clinical signs compatible with S. suis disease than the rest of the groups. Piglets from the FA + antiinf group showed high diversity and richness in their nasal and faecal microbiota. Diet supplements did not have major effects on the faecal microbiota, where the genus Mitsuokella was the only differentially present in the FA + Lys group. In the nasal microbiota, piglets from FA + antiinf presented higher differential abundance of a sequence variant from Ruminococcaceae and lower abundance of an unclassified genus from Weeksellaceae. In general, we detected more significant changes in the nasal than in the feacal microbiota, and found that parity of the dams affected the microbiota composition of their offspring, with piglets born to gilts exhibiting lower richness and diversity. Our results suggest that additives could be useful to control post-weaning disease when removing antimicrobials in farms.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77878-2" target="_blank" style="color:black; text-decoration:none">Recent land use and management changes decouple the adaptation of livestock diversity to the environment</a></h1>
            <p>Elena Velado-Alonso, Ignacio Morales-Castilla, Antonio Gómez-Sal</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-03</pre>
            <h2>Abstract</h2>
            <font size="4">Native livestock breeds, i.e. those autochthonous to a specific region, are locally adapted domesticated animals that conserve genetic resources, guaranty food security and provide agroecosystem services. Native breeds are largely threatened worldwide by agricultural intensification and rural areas abandonment processes related to recent changes in production schemes and planning. Yet, our gap of knowledge regarding livestock breed-environment relationships may prevent the design of successful conservation measures. In this work, we analyse the links between livestock diversity -i.e. richness of native breeds- and a selection of environmental factors that express at broad scales, with a temporal perspective. We compare native breeds distributional patterns before and after the agricultural intensification, in the context of land-use change in mainland Spain. Our results confirm the existence of strong associations between the distribution of native livestock breeds and environmental factors. These links, however, weaken for contemporary distributions. In fact, changes in breed distribution reflect a shift towards more productive environments. Finally, we found that the areas having higher breed richness are undergoing land abandonment processes. Succeeding in the conservation of threatened native breeds will require going beyond merely genetic and production-oriented views. Ecological and sociocultural perspectives should also be accounted for as global change processes are determinant for livestock agrobiodiversity.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41893-020-00637-8" target="_blank" style="color:black; text-decoration:none">Impact of local and landscape complexity on the stability of field-level pest control</a></h1>
            <p>Ashley E. Larsen, Frederik Noack</p>
            <pre style="font-family:timesnewroman">Journal: Nature Sustainability    Published: 2020-11-09</pre>
            <h2>Abstract</h2>
            <font size="4">Agricultural production has increased dramatically in the past 50 years, supported, in part, by the simplification of agricultural landscapes. While the benefits of increased food production are difficult to dispute, simplification, at both the local and landscape level, has fuelled declines in biodiversity and ecosystem services. In addition to the concerns that this loss of complexity necessitates higher levels of pesticide use in general, local and landscape simplification may also increase pest outbreaks and, consequently, infrequent but particularly high pesticide use with potentially damaging consequences for the environment and human health. We find that increasing cropland in the landscape—and larger fields generally—increase the level and variability of pesticide use while crop diversity has the opposite effect, as predicted by ecological theory. In all cases, accounting for non-random planting decisions and farmer-specific behaviour strongly influences the magnitude of the estimated statistical relationships. This suggests that, while complexity increases stability and reduces high deviations in insecticide use, accounting for crop and farmer-specific characteristics is crucial for statistical inference and sound scientific understanding. Agricultural landscapes have become more simplified as crop production has risen. This study finds that pesticide use rises with crop area and field size but falls with crop diversity.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-80444-5" target="_blank" style="color:black; text-decoration:none">A glimpse of antimicrobial resistance gene diversity in kefir and yoghurt</a></h1>
            <p>Adrienn Gréta Tóth, István Csabai, Gergely Maróti, Ákos Jerzsele, Attila Dubecz, Árpád V. Patai, Maura Fiona Judge, Sára Ágnes Nagy, László Makrai, Krisztián Bányai, Géza Szita, Norbert Solymosi</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-31</pre>
            <h2>Abstract</h2>
            <font size="4">Antimicrobial resistance (AMR) is a global threat gaining more and more practical significance every year. The main determinants of AMR are the antimicrobial resistance genes (ARGs). Since bacteria can share genetic components via horizontal gene transfer, even non-pathogenic bacteria may provide ARG to any pathogens which they become physically close to (e.g. in the human gut). In addition, fermented food naturally contains bacteria in high amounts. In this study, we examined the diversity of ARG content in various kefir and yoghurt samples (products, grains, bacterial strains) using a unified metagenomic approach. We found numerous ARGs of commonly used fermenting bacteria. Even with the strictest filter restrictions, we identified ARGs undermining the efficacy of aminocoumarins, aminoglycosides, carbapenems, cephalosporins, cephamycins, diaminopyrimidines, elfamycins, fluoroquinolones, fosfomycins, glycylcyclines, lincosamides, macrolides, monobactams, nitrofurans, nitroimidazoles, penams, penems, peptides, phenicols, rifamycins, tetracyclines and triclosan. In the case of gene lmrD, we detected genetic environment providing mobility of this ARG. Our findings support the theory that during the fermentation process, the ARG content of foods can grow due to bacterial multiplication. The results presented suggest that the starting culture strains of fermented foods should be monitored and selected in order to decrease the intake of ARGs via foods.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41522-020-00173-5" target="_blank" style="color:black; text-decoration:none">The upper respiratory tract microbiome of indigenous Orang Asli in north-eastern Peninsular Malaysia</a></h1>
            <p>David W. Cleary, Denise E. Morris, Rebecca A. Anderson, Jessica Jones, Ahmed Ghazi Alattraqchi, Nor Iza A. Rahman, Salwani Ismail, Mohd Sayuti Razali, Rahmah Mohd Amin, Aniza Abd Aziz, Nor Kamaruzaman Esa, Salman Amiruddin, Ching Hoong Chew, Hafis Simin, Ramle Abdullah, Chew Chieng Yeo, Stuart C. Clarke</p>
            <pre style="font-family:timesnewroman">Journal: npj Biofilms and Microbiomes    Published: 2021-01-05</pre>
            <h2>Abstract</h2>
            <font size="4">Much microbiome research has focused on populations that are predominantly of European descent, and from narrow demographics that do not capture the socio-economic and lifestyle differences which impact human health. Here we examined the airway microbiomes of the Orang Asli, the indigenous peoples of Malaysia. A total of 130 participants were recruited from two sites in the north-eastern state of Terengganu in Peninsular Malaysia. Using 16S rRNA sequencing, the nasal microbiome was significantly more diverse in those aged 5–17 years compared to 50+ years (p = 0.023) and clustered by age (PERMANOVA analysis of the Bray–Curtis distance, p = 0.001). Hierarchical clustering of Bray–Curtis dissimilarity scores revealed six microbiome clusters. The largest cluster (n = 28; 35.4%) had a marked abundance of Corynebacterium. In the oral microbiomes Streptococcus, Neisseria and Haemophilus were dominant. Using conventional microbiology, high levels of Staphylococcus aureus carriage were observed, particularly in the 18–65 age group (n = 17/36; 47.2% 95% CI: 30.9–63.5). The highest carriage of pneumococci was in the <5 and 5 to 17 year olds, with 57.1% (4/7) and 49.2% (30/61), respectively. Sixteen pneumococcal serotypes were identified, the most common being the nonvaccine-type 23A (14.6%) and the vaccine-type 6B (9.8%). The prevalence of pneumococcal serotypes covered by pneumococcal conjugate vaccines support introduction into a Malaysian national immunisation schedule. In addition, the dominance of Corynebacterium in the airway microbiomes is intriguing given their role as a potentially protective commensal with respect to acute infection and respiratory health.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41590-020-00830-z" target="_blank" style="color:black; text-decoration:none">Broadly effective metabolic and immune recovery with C5 inhibition in CHAPLE disease</a></h1>
            <p>Ahmet Ozen, Nurhan Kasap, Ivan Vujkovic-Cvijin, Richard Apps, Foo Cheung, Elif Karakoc-Aydiner, Bilge Akkelle, Sinan Sari, Engin Tutar, Figen Ozcay, Dilara Kocacik Uygun, Ali Islek, Gamze Akgun, Merve Selcuk, Oya Balci Sezer, Yu Zhang, Gunsel Kutluk, Erdem Topal, Ersin Sayar, Cigdem Celikel, Roderick H. J. Houwen, Aysen Bingol, Ismail Ogulur, Sevgi Bilgic Eltan, Andrew L. Snow, Camille Lake, Giovanna Fantoni, Camille Alba, Brian Sellers, Samuel D. Chauvin, Clifton L. Dalgard, Olivier Harari, Yan G. Ni, Ming-Dauh Wang, Kishor Devalaraja-Narashimha, Poorani Subramanian, Rabia Ergelen, Reha Artan, Sukru Nail Guner, Buket Dalgic, John Tsang, Yasmine Belkaid, Deniz Ertem, Safa Baris, Michael J. Lenardo</p>
            <pre style="font-family:timesnewroman">Journal: Nature Immunology    Published: 2021-01-04</pre>
            <h2>Abstract</h2>
            <font size="4">Complement hyperactivation, angiopathic thrombosis and protein-losing enteropathy (CHAPLE disease) is a lethal disease caused by genetic loss of the complement regulatory protein CD55, leading to overactivation of complement and innate immunity together with immunodeficiency due to immunoglobulin wasting in the intestine. We report in vivo human data accumulated using the complement C5 inhibitor eculizumab for the medical treatment of patients with CHAPLE disease. We observed cessation of gastrointestinal pathology together with restoration of normal immunity and metabolism. We found that patients rapidly renormalized immunoglobulin concentrations and other serum proteins as revealed by aptamer profiling, re-established a healthy gut microbiome, discontinued immunoglobulin replacement and other treatments and exhibited catch-up growth. Thus, we show that blockade of C5 by eculizumab effectively re-establishes regulation of the innate immune complement system to substantially reduce the pathophysiological manifestations of CD55 deficiency in humans. CHAPLE disease is a lethal syndrome caused by genetic loss of the complement regulatory protein CD55. Lenardo, Ozen and their colleagues report that blockade of C5 complement activation in a small cohort of pediatric patients with CHAPLE disease reduced gastrointestinal pathology and restored their immunity and growth.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41390-020-01287-4" target="_blank" style="color:black; text-decoration:none">Valuing and achieving diversity in academic medicine</a></h1>
            <p>Marva Moxey-Mims, Robin Steinhorn, Elena Fuentes-Afflick</p>
            <pre style="font-family:timesnewroman">Journal: Pediatric Research    Published: 2020-11-19</pre>
            <h2>Abstract</h2>
            <font size="4"></font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41396-020-00755-3" target="_blank" style="color:black; text-decoration:none">Evolution of diversity explains the impact of pre-adaptation of a focal species on the structure of a natural microbial community</a></h1>
            <p>Daniel Padfield, Alex Vujakovic, Steve Paterson, Rob Griffiths, Angus Buckling, Elze Hesse</p>
            <pre style="font-family:timesnewroman">Journal: The ISME Journal    Published: 2020-09-03</pre>
            <h2>Abstract</h2>
            <font size="4">Rapid within-species evolution can alter community structure, yet the mechanisms underpinning this effect remain unknown. Populations that rapidly evolve large amounts of phenotypic diversity are likely to interact with more species and have the largest impact on community structure. However, the evolution of phenotypic diversity is, in turn, influenced by the presence of other species. Here, we investigate how microbial community structure changes as a consequence of rapidly evolved within-species diversity using Pseudomonas fluorescens as a focal species. Evolved P. fluorescens populations showed substantial phenotypic diversification in resource-use (and correlated genomic change) irrespective of whether they were pre-adapted in isolation or in a community context. Manipulating diversity revealed that more diverse P. fluorescens populations had the greatest impact on community structure, by suppressing some bacterial taxa, but facilitating others. These findings suggest that conditions that promote the evolution of high within-population diversity should result in a larger impact on community structure.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78934-7" target="_blank" style="color:black; text-decoration:none">DNA metabarcoding uncovers fungal diversity in soils of protected and non-protected areas on Deception Island, Antarctica</a></h1>
            <p>Luiz Henrique Rosa, Thamar Holanda da Silva, Mayara Baptistucci Ogaki, Otávio Henrique Bezerra Pinto, Michael Stech, Peter Convey, Micheline Carvalho-Silva, Carlos Augusto Rosa, Paulo E. A. S. Câmara</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-15</pre>
            <h2>Abstract</h2>
            <font size="4">We assessed soil fungal diversity at two sites on Deception Island, South Shetland Islands, Antarctica using DNA metabarcoding analysis. The first site was a relatively undisturbed area, and the second was much more heavily impacted by research and tourism. We detected 346 fungal amplicon sequence variants dominated by the phyla Ascomycota, Basidiomycota, Mortierellomycota and Chytridiomycota. We also detected taxa belonging to the rare phyla Mucoromycota and Rozellomycota, which have been difficult to detect in Antarctica by traditional isolation methods. Cladosporium sp., Pseudogymnoascus roseus, Leotiomycetes sp. 2, Penicillium sp., Mortierella sp. 1, Mortierella sp. 2, Pseudogymnoascus appendiculatus and Pseudogymnoascus sp. were the most dominant fungi. In addition, 440,153 of the total of 1,214,875 reads detected could be classified only at the level of Fungi. In both sampling areas the DNA of opportunistic, phytopathogenic and symbiotic fungi were detected, which might have been introduced by human activities, transported by birds or wind, and/or represent resident fungi not previously reported from Antarctica. Further long-term studies are required to elucidate how biological colonization in the island may be affected by climatic changes and/or other anthropogenic influences.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78642-2" target="_blank" style="color:black; text-decoration:none">Impact of outdoor nature-related activities on gut microbiota, fecal serotonin, and perceived stress in preschool children: the Play&Grow randomized controlled trial</a></h1>
            <p>Tanja Sobko, Suisha Liang, Will H. G. Cheng, Hein M. Tun</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-15</pre>
            <h2>Abstract</h2>
            <font size="4">Due to rapid urbanization, children today have fewer opportunities to interact with nature and this may result in a greater risk for developing stress and depression. Outdoor nature-related activities can enhance general well-being. However, the underlying mechanisms are not fully delineated. Here we recruited 54 preschool children to participate in a 10-week structured nature-related “Play&Grow” program. Following the intervention, children were assessed for connectedness to nature and perceived stress levels using validated questionnaires. Moreover, fecal serotonin level and gut microbiota profiles were measured by ELISA and 16S rDNA amplicon sequencing, respectively. Children were significantly more connected to nature after the intervention. Their gut microbiota altered, especially by modulating the abundance of Roseburia and the fecal-serotonin level. Moreover, we also observed a reduction in the overall perceived stress, particularly in the frequency of anger among these children. This study is the first to demonstrate the impact of nature-related activities on gut microbiota, fecal serotonin and psychosocial behaviour of preschool children. However, further mechanistic studies are needed to confirm the functional role of gut microbiota in the association between connectedness to nature and improved psychosocial behavior.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77673-z" target="_blank" style="color:black; text-decoration:none">Emotional well-being and gut microbiome profiles by enterotype</a></h1>
            <p>Sung-Ha Lee, Seok-Hwan Yoon, Yeonjae Jung, Namil Kim, Uigi Min, Jongsik Chun, Incheol Choi</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-26</pre>
            <h2>Abstract</h2>
            <font size="4">With increasing attention being paid to improving emotional well-being, recent evidence points to gut microbiota as a key player in regulating mental and physical health via bidirectional communication between the brain and gut. Here, we examine the association between emotional well-being and gut microbiome profiles (i.e., gut microbiome composition, diversity, and the moderating role of the enterotypes) among healthy Korean adults (n = 83, mean age = 48.9, SD = 13.2). The research was performed using high-throughput 16S rRNA gene sequencing to obtain gut microbiome profiles, as well as a self-report survey that included the Positive Affect Negative Affect Schedule (PANAS). The cluster-based analysis identified two enterotypes dominated by the genera Bacteroides (n = 49) and Prevotella (n = 34). Generalized linear regression analysis reveals significant associations between positive emotion and gut microbiome diversity (Shannon Index) among participants in the Prevotella dominant group, whereas no such relationship emerged among participants in the Bacteroides group. Moreover, a novel genus from the family Lachnospiraceae is associated with emotional well-being scores, both positive and negative. Together, the current findings highlight the enterotype-specific links between the gut microbiota community and emotion in healthy adults and suggest the possible roles of the gut microbiome in promoting mental health.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-19929-w" target="_blank" style="color:black; text-decoration:none">A collection of bacterial isolates from the pig intestine reveals functional and taxonomic diversity</a></h1>
            <p>David Wylensek, Thomas C. A. Hitch, Thomas Riedel, Afrizal Afrizal, Neeraj Kumar, Esther Wortmann, Tianzhe Liu, Saravanan Devendran, Till R. Lesker, Sara B. Hernández, Viktoria Heine, Eva M. Buhl, Paul M. D’Agostino, Fabio Cumbo, Thomas Fischöder, Marzena Wyschkon, Torey Looft, Valeria R. Parreira, Birte Abt, Heidi L. Doden, Lindsey Ly, João M. P. Alves, Markus Reichlin, Krzysztof Flisikowski, Laura Navarro Suarez, Anthony P. Neumann, Garret Suen, Tomas de Wouters, Sascha Rohn, Ilias Lagkouvardos, Emma Allen-Vercoe, Cathrin Spröer, Boyke Bunk, Anja J. Taverne-Thiele, Marcel Giesbers, Jerry M. Wells, Klaus Neuhaus, Angelika Schnieke, Felipe Cava, Nicola Segata, Lothar Elling, Till Strowig, Jason M. Ridlon, Tobias A. M. Gulder, Jörg Overmann, Thomas Clavel</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-12-15</pre>
            <h2>Abstract</h2>
            <font size="4">Our knowledge about the gut microbiota of pigs is still scarce, despite the importance of these animals for biomedical research and agriculture. Here, we present a collection of cultured bacteria from the pig gut, including 110 species across 40 families and nine phyla. We provide taxonomic descriptions for 22 novel species and 16 genera. Meta-analysis of 16S rRNA amplicon sequence data and metagenome-assembled genomes reveal prevalent and pig-specific species within Lactobacillus, Streptococcus, Clostridium, Desulfovibrio, Enterococcus, Fusobacterium, and several new genera described in this study. Potentially interesting functions discovered in these organisms include a fucosyltransferase encoded in the genome of the novel species Clostridium porci, and prevalent gene clusters for biosynthesis of sactipeptide-like peptides. Many strains deconjugate primary bile acids in in vitro assays, and a Clostridium scindens strain produces secondary bile acids via dehydroxylation. In addition, cells of the novel species Bullifex porci are coccoidal or spherical under the culture conditions tested, in contrast with the usual helical shape of other members of the family Spirochaetaceae. The strain collection, called ‘Pig intestinal bacterial collection’ (PiBAC), is publicly available at 
                www.dsmz.de/pibac
                
               and opens new avenues for functional studies of the pig gut microbiota. The authors present a public collection of 117 bacterial isolates from the pig gut, including the description of 38 novel taxa. Interesting functions discovered in these organisms include a new fucosyltransferease and sactipeptide-like molecules encoded by biosynthetic gene clusters.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78923-w" target="_blank" style="color:black; text-decoration:none">Population genetics of the Manila clam ( Ruditapes philippinarum) in East Asia</a></h1>
            <p>Yue Tan, Lei Fang, Ming Qiu, Zhongming Huo, Xiwu Yan</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-14</pre>
            <h2>Abstract</h2>
            <font size="4">The Manila clam Ruditapes philippinarum is the world’s second most important bivalve mollusk commercially farmed, whose indigenous populations are mainly distributed in the coastal areas of East Asia. However, with the development of commercialization, mixture of populations and loss of local germplasm have become prominent problems. Here, genetic differentiation of seven Manila clam populations from East Asia was investigated through analyzing the polymorphism of the mitochondrial cytochrome C oxidase subunit I (COI) gene as well as 20 simple sequence repeat (SSR) molecular loci. In total, 40 haplotypes were identified, among which 31 were unique. Moreover, two main haplotypes were detected with several radiating derived haplotypes. Populations in Japan-North Korea shared haplotype Hap_31, and populations in China shared haplotype Hap_7, suggesting that the natural geographical isolation of the Yangtze River and the Yalu River might have divided the East Asian indigenous populations into three groups, which were located in South China, North China, and Japan-North Korea, respectively. The Aquaculture breeding activities from South to North in China might have promoted gene exchange among Manila clam populations. Population in Laizhou had the highest genetic diversity and therefore could be an excellent germplasm source.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-18871-1" target="_blank" style="color:black; text-decoration:none">Health and disease markers correlate with gut microbiome composition across thousands of people</a></h1>
            <p>Ohad Manor, Chengzhen L. Dai, Sergey A. Kornilov, Brett Smith, Nathan D. Price, Jennifer C. Lovejoy, Sean M. Gibbons, Andrew T. Magis</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-10-15</pre>
            <h2>Abstract</h2>
            <font size="4">Variation in the human gut microbiome can reflect host lifestyle and behaviors and influence disease biomarker levels in the blood. Understanding the relationships between gut microbes and host phenotypes are critical for understanding wellness and disease. Here, we examine associations between the gut microbiota and ~150 host phenotypic features across ~3,400 individuals. We identify major axes of taxonomic variance in the gut and a putative diversity maximum along the Firmicutes-to-Bacteroidetes axis. Our analyses reveal both known and unknown associations between microbiome composition and host clinical markers and lifestyle factors, including host-microbe associations that are composition-specific. These results suggest potential opportunities for targeted interventions that alter the composition of the microbiome to improve host health. By uncovering the interrelationships between host diet and lifestyle factors, clinical blood markers, and the human gut microbiome at the population-scale, our results serve as a roadmap for future studies on host-microbe interactions and interventions. Variation in the gut microbiome can reflect host lifestyle, behaviour, and influence blood-based biomarkers. Here the authors examine associations between the microbiota and 150 host phenotypic features in a large cohort of >3,000 individuals.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78940-9" target="_blank" style="color:black; text-decoration:none">The largest known cowrie and the iterative evolution of giant cypraeid gastropods</a></h1>
            <p>Stefano Dominici, Mariagabriella Fornasiero, Luca Giusberti</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-14</pre>
            <h2>Abstract</h2>
            <font size="4">Based on the fossil record, we explore the macroevolutionary relationship between species richness and gigantism in cowries (Cypraeidae), the best-studied family of gastropods, with a global diversity distribution that parallels that of tropical corals, mangroves and seagrasses. We introduce Vicetia bizzottoi sp. nov. based on a Priabonian fossil found in northeastern Italy, the largest documented cowrie found so far and the youngest of a lineage of Eocene Gisortiinae species. The Gisortiinae stratigraphic record in western Europe indicates that species selection favoured large size and armouring of the shell. Palaeoecology and per-stage species richness suggest that gigantism occurred in peripheral habitats with respect to diversity hotspots, where smaller species were favoured. The Eocene–Oligocene boundary was marked by a turnover and the Chattian global warming favoured small-sized species of derived clades. Species selection leading to gigantism is further documented in Miocene lineages of Zoila and Umbilia, in the southern hemisphere, two extant genera distributed at the periphery of modern diversity hotspots, suggesting that the negative relationship between size and diversity is a recurring pattern in the evolutionary history of cowries. This palaeontological evidence is projected onto the existing hypotheses that explain analogous biogeographic patterns in various other taxa. Likewise, body size-species richness negative relationship was possibly driven in cowries by physiological, ecological and life history constraints.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41559-020-01329-4" target="_blank" style="color:black; text-decoration:none">Remote spectral detection of biodiversity effects on forest biomass</a></h1>
            <p>Laura J. Williams, Jeannine Cavender-Bares, Philip A. Townsend, John J. Couture, Zhihui Wang, Artur Stefanski, Christian Messier, Peter B. Reich</p>
            <pre style="font-family:timesnewroman">Journal: Nature Ecology & Evolution    Published: 2020-11-02</pre>
            <h2>Abstract</h2>
            <font size="4">Quantifying how biodiversity affects ecosystem functions through time over large spatial extents is needed for meeting global biodiversity goals yet is infeasible with field-based approaches alone. Imaging spectroscopy is a tool with potential to help address this challenge. Here, we demonstrate a spectral approach to assess biodiversity effects in young forests that provides insight into its underlying drivers. Using airborne imaging of a tree-diversity experiment, spectral differences among stands enabled us to quantify net biodiversity effects on stem biomass and canopy nitrogen. By subsequently partitioning these effects, we reveal how distinct processes contribute to diversity-induced differences in stand-level spectra, chemistry and biomass. Across stands, biomass overyielding was best explained by species with greater leaf nitrogen dominating upper canopies in mixtures, rather than intraspecific shifts in canopy structure or chemistry. Remote imaging spectroscopy may help to detect the form and drivers of biodiversity–ecosystem function relationships across space and time, advancing the capacity to monitor and manage Earth’s ecosystems. Based on a tree-diversity experiment, the authors introduce and test an approach to quantify net biodiversity effects on stem biomass in young forests using airborne spectroscopic imaging.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-77919-w" target="_blank" style="color:black; text-decoration:none">Glioma and temozolomide induced alterations in gut microbiome</a></h1>
            <p>Anthony Patrizz, Antonio Dono, Soheil Zorofchian, Gabriella Hines, Takeshi Takayasu, Nuruddin Husein, Yoshihiro Otani, Octavio Arevalo, H. Alex Choi, Jude Savarraj, Nitin Tandon, Bhanu P. Ganesh, Balveen Kaur, Louise D. McCullough, Leomar Y. Ballester, Yoshua Esquenazi</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-03</pre>
            <h2>Abstract</h2>
            <font size="4">The gut microbiome is fundamental in neurogenesis processes. Alterations in microbial constituents promote inflammation and immunosuppression. Recently, in immune-oncology, specific microbial taxa have been described to enhance the effects of therapeutic modalities. However, the effects of microbial dysbiosis on glioma are still unknown. The aim of this study was to explore the effects of glioma development and Temozolomide (TMZ) on fecal microbiome in mice and humans. C57BL/6 mice were implanted with GL261/Sham and given TMZ/Saline. Fecal samples were collected longitudinally and analyzed by 16S rRNA sequencing. Fecal samples were collected from healthy controls as well as glioma patients at diagnosis, before and after chemoradiation. Compared to healthy controls, mice and glioma patients demonstrated significant differences in beta diversity, Firmicutes/Bacteroides (F/B) ratio, and increase of Verrucomicrobia phylum and Akkermansia genus. These changes were not observed following TMZ in mice. TMZ treatment in the non-tumor bearing mouse-model diminished the F/B ratio, increase Muribaculaceae family and decrease Ruminococcaceae family. Nevertheless, there were no changes in Verrucomicrobia/Akkermansia. Glioma development leads to gut dysbiosis in a mouse-model, which was not observed in the setting of TMZ. These findings seem translational to humans and warrant further study.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41438-020-00441-7" target="_blank" style="color:black; text-decoration:none">Genomic consequences of apple improvement</a></h1>
            <p>Zoë Migicovsky, Kyle M. Gardner, Christopher Richards, C. Thomas Chao, Heidi R. Schwaninger, Gennaro Fazio, Gan-Yuan Zhong, Sean Myles</p>
            <pre style="font-family:timesnewroman">Journal: Horticulture Research    Published: 2021-01-01</pre>
            <h2>Abstract</h2>
            <font size="4">The apple (Malus domestica) is one of the world’s most commercially important perennial crops and its improvement has been the focus of human effort for thousands of years. Here, we genetically characterise over 1000 apple accessions from the United States Department of Agriculture (USDA) germplasm collection using over 30,000 single-nucleotide polymorphisms (SNPs). We confirm the close genetic relationship between modern apple cultivars and their primary progenitor species, Malus sieversii from Central Asia, and find that cider apples derive more of their ancestry from the European crabapple, Malus sylvestris, than do dessert apples. We determine that most of the USDA collection is a large complex pedigree: over half of the collection is interconnected by a series of first-degree relationships. In addition, 15% of the accessions have a first-degree relationship with one of the top 8 cultivars produced in the USA. With the exception of ‘Honeycrisp’, the top 8 cultivars are interconnected to each other via pedigree relationships. The cultivars ‘Golden Delicious’ and ‘Red Delicious’ were found to have over 60 first-degree relatives, consistent with their repeated use by apple breeders. We detected a signature of intense selection for red skin and provide evidence that breeders also selected for increased firmness. Our results suggest that Americans are eating apples largely from a single family tree and that the apple’s future improvement will benefit from increased exploitation of its tremendous natural genetic diversity.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-76656-4" target="_blank" style="color:black; text-decoration:none">Effects of two measures of riparian plant biodiversity on litter decomposition and associated processes in stream microcosms</a></h1>
            <p>Naiara López-Rojo, Javier Pérez, Ana Basaguren, Jesús Pozo, Juan Rubio-Ríos, J. Jesús Casas, Luz Boyero</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-11-12</pre>
            <h2>Abstract</h2>
            <font size="4">Plant litter decomposition is a key ecosystem process that can be altered by global changes such as biodiversity loss. These effects can be particularly important in detritus-based ecosystems, such as headwater streams, which are mainly fuelled by allochthonous plant litter inputs. However, experiments examining effects of plant diversity on litter decomposition in streams have not reached consensus about which measures of biodiversity are more relevant. We explored the influence of two of these measures, plant species richness (SR; monocultures vs. 3-species mixtures) and phylogenetic distance (PD; species belonging to the same family vs. different families), on leaf litter decomposition and associated processes and variables (nutrient dynamics, fungal biomass and detritivore growth), in a stream microcosm experiment using litter from 9 tree species belonging to 3 families. We found a negative effect of SR on decomposition (which contradicted the results of previous experiments) but a positive effect on fungal biomass. While PD did not affect decomposition, both SR and PD altered nutrient dynamics: there was greater litter and detritivore N loss in low-PD mixtures, and greater litter P loss and detritivore P gain in monocultures. This suggested that the number of species in mixtures and the similarity of their traits both modulated nutrient availability and utilization by detritivores. Moreover, the greater fungal biomass with higher SR could imply positive effects on detritivores in the longer term. Our results provide new insights of the functional repercussions of biodiversity loss by going beyond the often-explored relationship between SR and decomposition, and reveal an influence of plant species phylogenetic relatedness on nutrient cycling that merits further investigation.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78057-z" target="_blank" style="color:black; text-decoration:none">Bacterial diversity and functional analysis of severe early childhood caries and recurrence in India</a></h1>
            <p>Balakrishnan Kalpana, Puniethaa Prabhu, Ashaq Hussain Bhat, Arunsaikiran Senthilkumar, Raj Pranap Arun, Sharath Asokan, Sachin S. Gunthe, Rama S. Verma</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-04</pre>
            <h2>Abstract</h2>
            <font size="4">Dental caries is the most prevalent oral disease affecting nearly 70% of children in India and elsewhere. Micro-ecological niche based acidification due to dysbiosis in oral microbiome are crucial for caries onset and progression. Here we report the tooth bacteriome diversity compared in Indian children with caries free (CF), severe early childhood caries (SC) and recurrent caries (RC). High quality V3–V4 amplicon sequencing revealed that SC exhibited high bacterial diversity with unique combination and interrelationship. Gracillibacteria_GN02 and TM7 were unique in CF and SC respectively, while Bacteroidetes, Fusobacteria were significantly high in RC. Interestingly, we found Streptococcus oralis subsp. tigurinus clade 071 in all groups with significant abundance in SC and RC. Positive correlation between low and high abundant bacteria as well as with TCS, PTS and ABC transporters were seen from co-occurrence network analysis. This could lead to persistence of SC niche resulting in RC. Comparative in vitro assessment of biofilm formation showed that the standard culture of S. oralis and its phylogenetically similar clinical isolates showed profound biofilm formation and augmented the growth and enhanced biofilm formation in S. mutans in both dual and multispecies cultures.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78630-6" target="_blank" style="color:black; text-decoration:none">DNA metabarcoding of fungal diversity in air and snow of Livingston Island, South Shetland Islands, Antarctica</a></h1>
            <p>Luiz Henrique Rosa, Otávio Henrique Bezerra Pinto, Tina Šantl-Temkiv, Peter Convey, Micheline Carvalho-Silva, Carlos Augusto Rosa, Paulo E. A. S. Câmara</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-11</pre>
            <h2>Abstract</h2>
            <font size="4">We assessed fungal diversity present in air and freshly deposited snow samples obtained from Livingston Island, Antarctica, using DNA metabarcoding through high throughput sequencing (HTS). A total of 740 m3 of air were pumped through a 0.22 µm membrane. Snow obtained shortly after deposition was kept at room temperature and yielded 3.760 L of water, which was filtered using Sterivex membranes of 0.22 µm mesh size. The total DNA present was extracted and sequenced. We detected 171 fungal amplicon sequence variants (ASVs), 70 from the air and 142 from the snow. They were dominated by the phyla Ascomycota, Basidiomycota, Mortierellomycota and Mucoromycota. Pseudogymnoascus, Cladosporium, Mortierella and Penicillium sp. were the most dominant ASVs detected in the air in rank order. In snow, Cladosporium, Pseudogymnoascus, Penicillium, Meyerozyma, Lecidea, Malassezia, Hanseniaspora, Austroplaca, Mortierella, Rhodotorula, Penicillium, Thelebolus, Aspergillus, Poaceicola, Glarea and Lecanora were the dominant ASVs present. In general, the two fungal assemblages displayed high diversity, richness, and dominance indices, with the assemblage found in snow having the highest diversity indices. Of the total fungal ASVs detected, 29 were only present in the air sample and 101 in the snow sample, with only 41 present in both samples; however, when only the dominant taxa from both samples were compared none occurred only in the air and, among the rare portion, 26 taxa occurred in both air and snow. Application of HTS revealed the presence of a more diverse fungal community in the air and snow of Livingston Island in comparison with studies using traditional isolation methods. The assemblages were dominated by cold-adapted and cosmopolitan fungal taxa, including members of the genera Pseudogymnoascus, Malassezia and Rhodotorula, which include some taxa reported as opportunistic. Our results support the hypothesis that the presence of microbiota in the airspora indicates the possibility of dispersal around Antarctica in the air column. However, further aeromycology studies are required to understand the dynamics of fungal dispersal within and beyond Antarctica.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41598-020-78659-7" target="_blank" style="color:black; text-decoration:none">Effects of disturbances by forest elephants on diversity of trees and insects in tropical rainforests on Mount Cameroon</a></h1>
            <p>Vincent Maicher, Sylvain Delabye, Mercy Murkwe, Jiří Doležal, Jan Altman, Ishmeal N. Kobe, Julie Desmist, Eric B. Fokam, Tomasz Pyrcz, Robert Tropek</p>
            <pre style="font-family:timesnewroman">Journal: Scientific Reports    Published: 2020-12-10</pre>
            <h2>Abstract</h2>
            <font size="4">Natural disturbances are essential for tropical forests biodiversity. In the Afrotropics, megaherbivores have played a key role before their recent decline. Contrastingly to savanna elephants, forest elephants’ impact on ecosystems remains poorly studied. Few decades ago, forests on Mount Cameroon were divided by lava flows, not being crossed by a local population of forest elephants until now. We assessed communities of trees, butterflies and two guilds of moths in the disturbed and undisturbed forests split by the longest lava flow. We surveyed 32 plots, recording 2025 trees of 97 species, and 7853 insects of 437 species. The disturbed forests differed in reduced tree density, height, and high canopy cover, and in increased DBH. Forest elephants’ selective browsing and foraging also decreased tree species richness and altered their composition. The elephant disturbance increased butterfly species richness and had various effects on species richness and composition of the insect groups. These changes were likely caused by disturbance-driven alterations of habitats and species composition of trees. Moreover, the abandonment of forests by elephants led to local declines of range-restricted butterflies. The recent declines of forest elephants across the Afrotropics probably caused similar changes in forest biodiversity and should be reflected by conservation actions.</font>
        </div>
        <br><hr>
    
        <div>
            <h1><a href="https://www.nature.com/articles/s41467-020-19252-4" target="_blank" style="color:black; text-decoration:none">General destabilizing effects of eutrophication on grassland productivity at multiple spatial scales</a></h1>
            <p>Yann Hautier, Pengfei Zhang, Michel Loreau, Kevin R. Wilcox, Eric W. Seabloom, Elizabeth T. Borer, Jarrett E. K. Byrnes, Sally E. Koerner, Kimberly J. Komatsu, Jonathan S. Lefcheck, Andy Hector, Peter B. Adler, Juan Alberti, Carlos A. Arnillas, Jonathan D. Bakker, Lars A. Brudvig, Miguel N. Bugalho, Marc Cadotte, Maria C. Caldeira, Oliver Carroll, Mick Crawley, Scott L. Collins, Pedro Daleo, Laura E. Dee, Nico Eisenhauer, Anu Eskelinen, Philip A. Fay, Benjamin Gilbert, Amandine Hansar, Forest Isbell, Johannes M. H. Knops, Andrew S. MacDougall, Rebecca L. McCulley, Joslin L. Moore, John W. Morgan, Akira S. Mori, Pablo L. Peri, Edwin T. Pos, Sally A. Power, Jodi N. Price, Peter B. Reich, Anita C. Risch, Christiane Roscher, Mahesh Sankaran, Martin Schütz, Melinda Smith, Carly Stevens, Pedro M. Tognetti, Risto Virtanen, Glenda M. Wardle, Peter A. Wilfahrt, Shaopeng Wang</p>
            <pre style="font-family:timesnewroman">Journal: Nature Communications    Published: 2020-10-23</pre>
            <h2>Abstract</h2>
            <font size="4">Eutrophication is a widespread environmental change that usually reduces the stabilizing effect of plant diversity on productivity in local communities. Whether this effect is scale dependent remains to be elucidated. Here, we determine the relationship between plant diversity and temporal stability of productivity for 243 plant communities from 42 grasslands across the globe and quantify the effect of chronic fertilization on these relationships. Unfertilized local communities with more plant species exhibit greater asynchronous dynamics among species in response to natural environmental fluctuations, resulting in greater local stability (alpha stability). Moreover, neighborhood communities that have greater spatial variation in plant species composition within sites (higher beta diversity) have greater spatial asynchrony of productivity among communities, resulting in greater stability at the larger scale (gamma stability). Importantly, fertilization consistently weakens the contribution of plant diversity to both of these stabilizing mechanisms, thus diminishing the positive effect of biodiversity on stability at differing spatial scales. Our findings suggest that preserving grassland functional stability requires conservation of plant diversity within and among ecological communities. Eutrophication has been shown to weaken diversity-stability relationships in grasslands, but it is unclear whether the effect depends on scale. Analysing a globally distributed network of grassland sites, the authors show a positive role of beta diversity and spatial asynchrony as drivers of stability but find that nitrogen enrichment weakens the diversity-stability relationships at different spatial scales.</font>
        </div>
        <br><hr>
    
    </body>
</html>
    
